{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Import modules and dependencies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn import preprocessing\n",
    "import matplotlib.pyplot as plt\n",
    "import glob\n",
    "import os\n",
    "from matplotlib import pyplot\n",
    "import matplotlib.pyplot as plt\n",
    "from pathlib import Path\n",
    "from os.path import splitext\n",
    "from functools import reduce\n",
    "from pandas.api.types import is_numeric_dtype\n",
    "import warnings\n",
    "from pandas.core.common import SettingWithCopyWarning\n",
    "from matplotlib.font_manager import FontProperties \n",
    "from matplotlib import cm\n",
    "import itertools\n",
    "from matplotlib import container\n",
    "from matplotlib import gridspec\n",
    "from matplotlib.patches import Patch\n",
    "from matplotlib.lines import Line2D\n",
    "import matplotlib.lines as mlines\n",
    "import matplotlib as mpl\n",
    "from pandas.plotting import table\n",
    "import random\n",
    "import matplotlib.patches as mpatches\n",
    "from collections import Counter\n",
    "import functools\n",
    "import operator\n",
    "from itertools import chain\n",
    "import pandas as pd\n",
    "import glob, os\n",
    "import shutil as sh\n",
    "import numpy as np\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn import preprocessing\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib import pyplot\n",
    "import matplotlib.pyplot as plt\n",
    "from pathlib import Path\n",
    "from os.path import splitext\n",
    "from functools import reduce\n",
    "from pandas.api.types import is_numeric_dtype\n",
    "import warnings\n",
    "from pandas.core.common import SettingWithCopyWarning\n",
    "from matplotlib.font_manager import FontProperties \n",
    "from matplotlib import cm\n",
    "import itertools\n",
    "from matplotlib import container\n",
    "from matplotlib import gridspec\n",
    "from matplotlib.patches import Patch\n",
    "from matplotlib.lines import Line2D\n",
    "import matplotlib.lines as mlines\n",
    "import matplotlib as mpl\n",
    "from pandas.plotting import table\n",
    "from statistics import mean\n",
    "import statistics\n",
    "from scipy import stats\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_dataframes(path):\n",
    "    filelist = [os.path.splitext(file)[0] for file in os.listdir(path) if file.endswith('.vcf')]\n",
    "    list_of_DFs = []\n",
    "    for file in filelist: \n",
    "#         print(file)\n",
    "        DF = path + '/' + file + '.vcf'\n",
    "        file = pd.read_csv(DF, header=0, sep='\\t', low_memory=False)\n",
    "        file['sample'] = DF\n",
    "        list_of_DFs.append(file)\n",
    "    print(filelist)\n",
    "    return(list_of_DFs)\n",
    "\n",
    "def load_cleaned_dataframes(path):\n",
    "    filelist = [os.path.splitext(file)[0] for file in os.listdir(path) if file.endswith('.csv')]\n",
    "    list_of_DFs = []\n",
    "    for file in filelist: \n",
    "#         print(file)\n",
    "        DF = path + '/' + file + '.csv'\n",
    "        file = pd.read_csv(DF, header=0, sep='\\t', low_memory=False)\n",
    "        file['sample'] = DF\n",
    "        list_of_DFs.append(file)\n",
    "    print(filelist)\n",
    "    return(list_of_DFs)\n",
    "\n",
    "# clean up the VCF files and save as CSV files \n",
    "\n",
    "def clean_dataframes(path, list_of_DFs):\n",
    "    if not os.path.exists(path + '/cleaned'):\n",
    "            os.makedirs(path + '/cleaned')\n",
    "    for DF in list_of_DFs:\n",
    "        DF = DF[['sample', 'Position', 'Ref', 'Cons']]\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('data_raw/VCFs/'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.rstrip('.fasta.vcf'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('SA_'))\n",
    "        sample = DF['sample'].values[0]\n",
    "        DF['Position'] = DF['Position'].apply(str)\n",
    "        DF['SNP'] = DF['Position'] + '_' + DF['Ref'] + '>' + DF['Cons']\n",
    "        DF.to_csv(path + 'cleaned/' + sample + '-cleaned.csv', sep='\\t')\n",
    "        \n",
    "def make_snp_list(path, list_of_DFs):\n",
    "    list_of_lists = []\n",
    "    for DF in list_of_DFs:\n",
    "        DF = DF['SNP'].tolist()\n",
    "        list_of_lists.append(DF)\n",
    "    return(list_of_lists)\n",
    "        \n",
    "\n",
    "def make_dictionary(path, list_of_DFs):\n",
    "    names_and_snpies = {}\n",
    "    for DF in list_of_DFs:\n",
    "        key = DF['sample'].values[0]\n",
    "        key = key.strip('VCFs/cleaned//')\n",
    "        key = key.strip('-cleaned.csv')\n",
    "        value = DF['SNP'].tolist()\n",
    "        names_and_snpies[key] = value\n",
    "    return(names_and_snpies)\n",
    "\n",
    "def get_unique_snps(path, list_of_DFs):\n",
    "    all_samples = pd.concat(list_of_DFs)\n",
    "    all_samples = all_samples.drop(axis=1, columns=['sample','Position','Ref','Cons','Unnamed: 0'])\n",
    "    all_samples = all_samples.drop_duplicates(keep='first')\n",
    "    all_samples = all_samples.reset_index()\n",
    "    all_samples = all_samples.drop(axis=1, columns='index')\n",
    "    return (all_samples)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "4886 sequences were dropped during filtering\n",
      "\t4886 of these were dropped because of 'campus!=UW-Madison'\n",
      "278 sequences have been written out to data/UWMadisonDorm.fasta\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.\n",
      "To initialize your shell, run\n",
      "\n",
      "    $ conda init <SHELL_NAME>\n",
      "\n",
      "Currently supported shells are:\n",
      "  - bash\n",
      "  - fish\n",
      "  - tcsh\n",
      "  - xonsh\n",
      "  - zsh\n",
      "  - powershell\n",
      "\n",
      "See 'conda init --help' for more information and options.\n",
      "\n",
      "IMPORTANT: You may need to close and restart your shell after running 'conda init'.\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "conda activate nextstrain\n",
    "augur filter --sequences data/sequences.fasta --metadata data/metadata.tsv --group-by month year submitting_lab --exclude-where 'campus!=UW-Madison' --output data/UWMadisonDorm.fasta\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "splitfasta UWMadisonDorm.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_1.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_10.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_100.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_101.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_102.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_103.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_104.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_105.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_106.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_107.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_108.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_109.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_11.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_110.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_111.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_112.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_113.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_114.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_115.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_116.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_117.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_118.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_119.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_12.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_120.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_121.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_122.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_123.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_124.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_125.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_126.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_127.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_128.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_129.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_13.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_130.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_131.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_132.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_133.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_134.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_135.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_136.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_137.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_138.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_139.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_14.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_140.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_141.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_142.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_143.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_144.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_145.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_146.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_147.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_148.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_149.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_15.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_150.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_151.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_152.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_153.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_154.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_155.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_156.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_157.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_158.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_159.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_16.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_160.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_161.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_162.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_163.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_164.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_165.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_166.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_167.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_168.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_169.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_17.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_170.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_171.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_172.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_173.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_174.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_175.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_176.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_177.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_178.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_179.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_18.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_180.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_181.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_182.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_183.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_184.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_185.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_186.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_187.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_188.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_189.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_19.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_190.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_191.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_192.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_193.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_194.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_195.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_196.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_197.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_198.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_199.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_2.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_20.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_200.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_201.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_202.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_203.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_204.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_205.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_206.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_207.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_208.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_209.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_21.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_210.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_211.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_212.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_213.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_214.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_215.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_216.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_217.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_218.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_219.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_22.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_220.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_221.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_222.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_223.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_224.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_225.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_226.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_227.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_228.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_229.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_23.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_230.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_231.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_232.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_233.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_234.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_235.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_236.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_237.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_238.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_239.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_24.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_240.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_241.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_242.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_243.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_244.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_245.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_246.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_247.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_248.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_249.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_25.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_250.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_251.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_252.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_253.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_254.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_255.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_256.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_257.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_258.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_259.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_26.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_260.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_261.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_262.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_263.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_264.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_265.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_266.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_267.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_268.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_269.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_27.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_270.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_271.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_272.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_273.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_274.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_275.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_276.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_277.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_278.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_28.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_29.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_3.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_30.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_31.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_32.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_33.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_34.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_35.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_36.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_37.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_38.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_39.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_4.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_40.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_41.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_42.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_43.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_44.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_45.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_46.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_47.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_48.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_49.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_5.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_50.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_51.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_52.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_53.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_54.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_55.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_56.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_57.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_58.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_59.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_6.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_60.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_61.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_62.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_63.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_64.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_65.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_66.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_67.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_68.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_69.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_7.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_70.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_71.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_72.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_73.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_74.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_75.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_76.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_77.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_78.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_79.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_8.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_80.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_81.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_82.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_83.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_84.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_85.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_86.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_87.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_88.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_89.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_9.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_90.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_91.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_92.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_93.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_94.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_95.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_96.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_97.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_98.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/bin/faidx\", line 10, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 195, in main\n",
      "    write_sequence(args)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/cli.py\", line 19, in write_sequence\n",
      "    fasta = Fasta(args.fasta, default_seq=args.default_seq, key_function=eval(args.header_function), strict_bounds=not args.lazy, split_char=args.delimiter, filt_function=filt_function, read_long_names=args.long_names, rebuild=not args.no_rebuild)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 1011, in __init__\n",
      "    build_index=build_index)\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 434, in __init__\n",
      "    self.build_index()\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 605, in build_index\n",
      "    raise e\n",
      "  File \"/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/pyfaidx/__init__.py\", line 580, in build_index\n",
      "    \"Check that sequence definition lines start with '>'.\" % self.filename)\n",
      "pyfaidx.FastaIndexingError: The FASTA file UWMadisonDorm_split_files/UWMadisonDorm_99.fasta.fai does not contain a valid sequence. Check that sequence definition lines start with '>'.\n"
     ]
    },
    {
     "ename": "CalledProcessError",
     "evalue": "Command 'b'\\nfor f in UWMadisonDorm_split_files/*; do \\n    faidx -x $f\\ndone\\n'' returned non-zero exit status 1.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mCalledProcessError\u001b[0m                        Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-91a1986a7299>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun_cell_magic\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'bash'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'\\nfor f in UWMadisonDorm_split_files/*; do \\n    faidx -x $f\\ndone\\n'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/opt/anaconda3/lib/python3.7/site-packages/IPython/core/interactiveshell.py\u001b[0m in \u001b[0;36mrun_cell_magic\u001b[0;34m(self, magic_name, line, cell)\u001b[0m\n\u001b[1;32m   2357\u001b[0m             \u001b[0;32mwith\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbuiltin_trap\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2358\u001b[0m                 \u001b[0margs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mmagic_arg_s\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 2359\u001b[0;31m                 \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mfn\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   2360\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2361\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/opt/anaconda3/lib/python3.7/site-packages/IPython/core/magics/script.py\u001b[0m in \u001b[0;36mnamed_script_magic\u001b[0;34m(line, cell)\u001b[0m\n\u001b[1;32m    140\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    141\u001b[0m                 \u001b[0mline\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mscript\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 142\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mshebang\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    143\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    144\u001b[0m         \u001b[0;31m# write a basic docstring:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m</Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/decorator.py:decorator-gen-110>\u001b[0m in \u001b[0;36mshebang\u001b[0;34m(self, line, cell)\u001b[0m\n",
      "\u001b[0;32m~/opt/anaconda3/lib/python3.7/site-packages/IPython/core/magic.py\u001b[0m in \u001b[0;36m<lambda>\u001b[0;34m(f, *a, **k)\u001b[0m\n\u001b[1;32m    185\u001b[0m     \u001b[0;31m# but it's overkill for just that one bit of state.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    186\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mmagic_deco\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 187\u001b[0;31m         \u001b[0mcall\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mlambda\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mk\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mk\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    188\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    189\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mcallable\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/opt/anaconda3/lib/python3.7/site-packages/IPython/core/magics/script.py\u001b[0m in \u001b[0;36mshebang\u001b[0;34m(self, line, cell)\u001b[0m\n\u001b[1;32m    243\u001b[0m             \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstderr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    244\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0margs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mraise_error\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreturncode\u001b[0m\u001b[0;34m!=\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 245\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mCalledProcessError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreturncode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moutput\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0merr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    246\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    247\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_run_script\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mto_close\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mCalledProcessError\u001b[0m: Command 'b'\\nfor f in UWMadisonDorm_split_files/*; do \\n    faidx -x $f\\ndone\\n'' returned non-zero exit status 1."
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f in UWMadisonDorm_split_files/*; do \n",
    "    faidx -x $f\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "mv USA*.fasta dormfiles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14182020.fasta out=dormfiles/USAWI-UW-14182020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14182020.fasta, out=dormfiles/USAWI-UW-14182020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tDeleting contents of ref/genome/1 because reference is specified and overwrite=true\n",
      "NOTE:\tDeleting contents of ref/index/1 because reference is specified and overwrite=true\n",
      "Writing reference.\n",
      "Executing dna.FastaToChromArrays2 [ref.fa, 1, writeinthread=false, genscaffoldinfo=true, retain, waitforwriting=false, gz=true, maxlen=536670912, writechroms=true, minscaf=1, midpad=300, startpad=8000, stoppad=8000, nodisk=false]\n",
      "\n",
      "Set genScaffoldInfo=true\n",
      "Writing chunk 1\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.005 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "No index available; generating from reference genome: /Volumes/GoogleDrive/My Drive/SARS-CoV-2_Sequencing/CDCMadison/ref/index/1/chr1_index_k13_c15_b1.block\n",
      "Indexing threads started for block 0-1\n",
      "Indexing threads finished for block 0-1\n",
      "Generated Index:\t1.359 seconds.\n",
      "Analyzed Index:   \t2.713 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.118 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.758 seconds.\n",
      "Reads/sec:       \t79.18\n",
      "kBases/sec:      \t39.30\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.243 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14182020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14192020.fasta out=dormfiles/USAWI-UW-14192020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14192020.fasta, out=dormfiles/USAWI-UW-14192020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.601 seconds.\n",
      "Analyzed Index:   \t2.555 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.750 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.325 seconds.\n",
      "Reads/sec:       \t184.51\n",
      "kBases/sec:      \t91.59\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.471 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14192020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14202020.fasta out=dormfiles/USAWI-UW-14202020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14202020.fasta, out=dormfiles/USAWI-UW-14202020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.710 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.121 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.301 seconds.\n",
      "Reads/sec:       \t199.42\n",
      "kBases/sec:      \t98.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.960 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14202020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14212020.fasta out=dormfiles/USAWI-UW-14212020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14212020.fasta, out=dormfiles/USAWI-UW-14212020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.588 seconds.\n",
      "Analyzed Index:   \t2.740 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.432 seconds.\n",
      "Reads/sec:       \t138.97\n",
      "kBases/sec:      \t68.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.082 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14212020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14222020.fasta out=dormfiles/USAWI-UW-14222020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14222020.fasta, out=dormfiles/USAWI-UW-14222020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.537 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.557 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.259 seconds.\n",
      "Reads/sec:       \t232.06\n",
      "kBases/sec:      \t115.19\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.185 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14222020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14232020.fasta out=dormfiles/USAWI-UW-14232020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14232020.fasta, out=dormfiles/USAWI-UW-14232020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.604 seconds.\n",
      "Analyzed Index:   \t2.515 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.400 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.277 seconds.\n",
      "Reads/sec:       \t216.96\n",
      "kBases/sec:      \t107.69\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.028 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14232020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14242020.fasta out=dormfiles/USAWI-UW-14242020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14242020.fasta, out=dormfiles/USAWI-UW-14242020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.604 seconds.\n",
      "Analyzed Index:   \t2.857 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.249 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.415 seconds.\n",
      "Reads/sec:       \t144.44\n",
      "kBases/sec:      \t71.29\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.5402% \t       19111\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.5402% \t       19111\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1760% \t       29367\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0304% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0304% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 25.0000% \t       15 \t  0.7936% \t         235\n",
      "\n",
      "Total time:     \t4.370 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14242020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14252020.fasta out=dormfiles/USAWI-UW-14252020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14252020.fasta, out=dormfiles/USAWI-UW-14252020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.605 seconds.\n",
      "Analyzed Index:   \t3.082 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.366 seconds.\n",
      "Reads/sec:       \t164.03\n",
      "kBases/sec:      \t81.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1841% \t       29539\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 15.0000% \t        9 \t  0.7588% \t         226\n",
      "\n",
      "Total time:     \t4.380 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14252020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14262020.fasta out=dormfiles/USAWI-UW-14262020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14262020.fasta, out=dormfiles/USAWI-UW-14262020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.648 seconds.\n",
      "Analyzed Index:   \t2.724 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.347 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.329 seconds.\n",
      "Reads/sec:       \t182.15\n",
      "kBases/sec:      \t90.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.298 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14262020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14272020.fasta out=dormfiles/USAWI-UW-14272020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14272020.fasta, out=dormfiles/USAWI-UW-14272020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.650 seconds.\n",
      "Analyzed Index:   \t2.655 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.216 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.374 seconds.\n",
      "Reads/sec:       \t160.60\n",
      "kBases/sec:      \t79.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.146 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14272020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14282020.fasta out=dormfiles/USAWI-UW-14282020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14282020.fasta, out=dormfiles/USAWI-UW-14282020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.569 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.164 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.60\n",
      "kBases/sec:      \t108.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.855 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14282020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14292020.fasta out=dormfiles/USAWI-UW-14292020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14292020.fasta, out=dormfiles/USAWI-UW-14292020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.616 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.460 seconds.\n",
      "Reads/sec:       \t130.57\n",
      "kBases/sec:      \t64.81\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.978 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14292020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14302020.fasta out=dormfiles/USAWI-UW-14302020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14302020.fasta, out=dormfiles/USAWI-UW-14302020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.619 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.716 seconds.\n",
      "Reads/sec:       \t83.76\n",
      "kBases/sec:      \t41.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.239 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14302020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14312020.fasta out=dormfiles/USAWI-UW-14312020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14312020.fasta, out=dormfiles/USAWI-UW-14312020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.593 seconds.\n",
      "Analyzed Index:   \t2.586 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.372 seconds.\n",
      "Reads/sec:       \t161.24\n",
      "kBases/sec:      \t80.03\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.858 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14312020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-14322020.fasta out=dormfiles/USAWI-UW-14322020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-14322020.fasta, out=dormfiles/USAWI-UW-14322020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.886 seconds.\n",
      "Analyzed Index:   \t2.720 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.899 seconds.\n",
      "Reads/sec:       \t66.71\n",
      "kBases/sec:      \t33.11\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.829 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-14322020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16302020.fasta out=dormfiles/USAWI-UW-16302020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16302020.fasta, out=dormfiles/USAWI-UW-16302020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.607 seconds.\n",
      "Analyzed Index:   \t2.777 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.367 seconds.\n",
      "Reads/sec:       \t163.56\n",
      "kBases/sec:      \t81.19\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "unambiguous:     \t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  3.3333% \t        2 \t  3.3577% \t        1000\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 97.1240% \t       26983\n",
      "Error Rate:      \t 17.2414% \t       10 \t  0.0360% \t          10\n",
      "Sub Rate:        \t 17.2414% \t       10 \t  0.0360% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 17.2414% \t       10 \t  2.8400% \t         789\n",
      "\n",
      "Total time:     \t4.077 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16302020.fasta.pileup\n",
      "27782 bases in pileup file\n",
      "27782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16312020.fasta out=dormfiles/USAWI-UW-16312020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16312020.fasta, out=dormfiles/USAWI-UW-16312020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.571 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.605 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.678 seconds.\n",
      "Reads/sec:       \t88.56\n",
      "kBases/sec:      \t43.96\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.678 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16312020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16322020.fasta out=dormfiles/USAWI-UW-16322020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16322020.fasta, out=dormfiles/USAWI-UW-16322020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.617 seconds.\n",
      "Analyzed Index:   \t3.189 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.271 seconds.\n",
      "Reads/sec:       \t221.60\n",
      "kBases/sec:      \t109.99\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.414 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16322020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16332020.fasta out=dormfiles/USAWI-UW-16332020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16332020.fasta, out=dormfiles/USAWI-UW-16332020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.614 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.314 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.334 seconds.\n",
      "Reads/sec:       \t179.63\n",
      "kBases/sec:      \t89.16\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.086 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16332020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16342020.fasta out=dormfiles/USAWI-UW-16342020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16342020.fasta, out=dormfiles/USAWI-UW-16342020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.575 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.174 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.554 seconds.\n",
      "Reads/sec:       \t108.31\n",
      "kBases/sec:      \t53.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.151 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16342020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16352020.fasta out=dormfiles/USAWI-UW-16352020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16352020.fasta, out=dormfiles/USAWI-UW-16352020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.589 seconds.\n",
      "Analyzed Index:   \t2.582 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.442 seconds.\n",
      "Reads/sec:       \t135.72\n",
      "kBases/sec:      \t67.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0672% \t          20\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0672% \t          20\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.933 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16352020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "20 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16362020.fasta out=dormfiles/USAWI-UW-16362020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16362020.fasta, out=dormfiles/USAWI-UW-16362020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.605 seconds.\n",
      "Analyzed Index:   \t2.607 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.589 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.343 seconds.\n",
      "Reads/sec:       \t174.72\n",
      "kBases/sec:      \t86.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.387 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16362020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16372020.fasta out=dormfiles/USAWI-UW-16372020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16372020.fasta, out=dormfiles/USAWI-UW-16372020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.549 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.109 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t231.11\n",
      "kBases/sec:      \t114.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.745 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16372020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16382020.fasta out=dormfiles/USAWI-UW-16382020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16382020.fasta, out=dormfiles/USAWI-UW-16382020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.594 seconds.\n",
      "Analyzed Index:   \t2.564 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.446 seconds.\n",
      "Reads/sec:       \t134.67\n",
      "kBases/sec:      \t66.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "\n",
      "Total time:     \t3.905 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16382020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16392020.fasta out=dormfiles/USAWI-UW-16392020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16392020.fasta, out=dormfiles/USAWI-UW-16392020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.726 seconds.\n",
      "Analyzed Index:   \t2.489 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.257 seconds.\n",
      "Reads/sec:       \t233.46\n",
      "kBases/sec:      \t115.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.768 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16392020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16402020.fasta out=dormfiles/USAWI-UW-16402020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16402020.fasta, out=dormfiles/USAWI-UW-16402020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.547 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.481 seconds.\n",
      "Reads/sec:       \t124.71\n",
      "kBases/sec:      \t61.90\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9295% \t       29761\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.922 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16402020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16412020.fasta out=dormfiles/USAWI-UW-16412020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16412020.fasta, out=dormfiles/USAWI-UW-16412020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.630 seconds.\n",
      "Analyzed Index:   \t2.504 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.475 seconds.\n",
      "Reads/sec:       \t126.21\n",
      "kBases/sec:      \t62.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.918 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16412020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16422020.fasta out=dormfiles/USAWI-UW-16422020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16422020.fasta, out=dormfiles/USAWI-UW-16422020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.600 seconds.\n",
      "Analyzed Index:   \t2.655 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.671 seconds.\n",
      "Reads/sec:       \t89.47\n",
      "kBases/sec:      \t44.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.234 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16422020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16432020.fasta out=dormfiles/USAWI-UW-16432020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16432020.fasta, out=dormfiles/USAWI-UW-16432020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.230 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.632 seconds.\n",
      "Analyzed Index:   \t3.080 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.121 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.21\n",
      "kBases/sec:      \t105.83\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.774 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16432020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16442020.fasta out=dormfiles/USAWI-UW-16442020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16442020.fasta, out=dormfiles/USAWI-UW-16442020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.615 seconds.\n",
      "Analyzed Index:   \t2.679 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.351 seconds.\n",
      "Reads/sec:       \t170.74\n",
      "kBases/sec:      \t84.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.083 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16442020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16452020.fasta out=dormfiles/USAWI-UW-16452020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16452020.fasta, out=dormfiles/USAWI-UW-16452020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.615 seconds.\n",
      "Analyzed Index:   \t2.586 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.224 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.396 seconds.\n",
      "Reads/sec:       \t151.43\n",
      "kBases/sec:      \t75.17\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.059 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16452020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16462020.fasta out=dormfiles/USAWI-UW-16462020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16462020.fasta, out=dormfiles/USAWI-UW-16462020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.600 seconds.\n",
      "Analyzed Index:   \t2.584 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.435 seconds.\n",
      "Reads/sec:       \t137.82\n",
      "kBases/sec:      \t68.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.932 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16462020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16472020.fasta out=dormfiles/USAWI-UW-16472020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16472020.fasta, out=dormfiles/USAWI-UW-16472020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.772 seconds.\n",
      "Analyzed Index:   \t2.555 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.398 seconds.\n",
      "Reads/sec:       \t150.73\n",
      "kBases/sec:      \t74.82\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9295% \t       29761\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.055 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16472020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16482020.fasta out=dormfiles/USAWI-UW-16482020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16482020.fasta, out=dormfiles/USAWI-UW-16482020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.764 seconds.\n",
      "Analyzed Index:   \t2.559 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.293 seconds.\n",
      "Reads/sec:       \t205.08\n",
      "kBases/sec:      \t101.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.954 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16482020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16492020.fasta out=dormfiles/USAWI-UW-16492020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16492020.fasta, out=dormfiles/USAWI-UW-16492020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.567 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.317 seconds.\n",
      "Reads/sec:       \t189.36\n",
      "kBases/sec:      \t93.99\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t3.792 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16492020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16502020.fasta out=dormfiles/USAWI-UW-16502020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16502020.fasta, out=dormfiles/USAWI-UW-16502020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.780 seconds.\n",
      "Analyzed Index:   \t2.695 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.423 seconds.\n",
      "Reads/sec:       \t141.86\n",
      "kBases/sec:      \t70.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.349 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16502020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16512020.fasta out=dormfiles/USAWI-UW-16512020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16512020.fasta, out=dormfiles/USAWI-UW-16512020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.592 seconds.\n",
      "Analyzed Index:   \t2.610 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t222.64\n",
      "kBases/sec:      \t110.51\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.784 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16512020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16522020.fasta out=dormfiles/USAWI-UW-16522020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16522020.fasta, out=dormfiles/USAWI-UW-16522020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.626 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.362 seconds.\n",
      "Reads/sec:       \t165.97\n",
      "kBases/sec:      \t82.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9127% \t       29756\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.894 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16522020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16532020.fasta out=dormfiles/USAWI-UW-16532020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16532020.fasta, out=dormfiles/USAWI-UW-16532020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.688 seconds.\n",
      "Analyzed Index:   \t2.734 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.447 seconds.\n",
      "Reads/sec:       \t134.13\n",
      "kBases/sec:      \t66.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.194 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16532020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16542020.fasta out=dormfiles/USAWI-UW-16542020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16542020.fasta, out=dormfiles/USAWI-UW-16542020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.962 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.349 seconds.\n",
      "Reads/sec:       \t171.88\n",
      "kBases/sec:      \t85.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.261 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16542020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16552020.fasta out=dormfiles/USAWI-UW-16552020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16552020.fasta, out=dormfiles/USAWI-UW-16552020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.670 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.279 seconds.\n",
      "Reads/sec:       \t214.82\n",
      "kBases/sec:      \t106.63\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.862 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16552020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16562020.fasta out=dormfiles/USAWI-UW-16562020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16562020.fasta, out=dormfiles/USAWI-UW-16562020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.624 seconds.\n",
      "Analyzed Index:   \t2.793 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.297 seconds.\n",
      "Reads/sec:       \t201.88\n",
      "kBases/sec:      \t100.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.030 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16562020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16572020.fasta out=dormfiles/USAWI-UW-16572020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16572020.fasta, out=dormfiles/USAWI-UW-16572020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.755 seconds.\n",
      "Analyzed Index:   \t2.816 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.357 seconds.\n",
      "Reads/sec:       \t167.94\n",
      "kBases/sec:      \t83.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.255 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16572020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16582020.fasta out=dormfiles/USAWI-UW-16582020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16582020.fasta, out=dormfiles/USAWI-UW-16582020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.718 seconds.\n",
      "Analyzed Index:   \t2.676 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.315 seconds.\n",
      "Reads/sec:       \t190.41\n",
      "kBases/sec:      \t94.51\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.230 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16582020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16592020.fasta out=dormfiles/USAWI-UW-16592020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16592020.fasta, out=dormfiles/USAWI-UW-16592020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.005 seconds.\n",
      "Analyzed Index:   \t2.741 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.312 seconds.\n",
      "Reads/sec:       \t192.04\n",
      "kBases/sec:      \t95.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.367 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16592020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16602020.fasta out=dormfiles/USAWI-UW-16602020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16602020.fasta, out=dormfiles/USAWI-UW-16602020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.765 seconds.\n",
      "Analyzed Index:   \t2.897 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.449 seconds.\n",
      "Reads/sec:       \t133.62\n",
      "kBases/sec:      \t66.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0672% \t          20\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0672% \t          20\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.493 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16602020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "20 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16612020.fasta out=dormfiles/USAWI-UW-16612020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16612020.fasta, out=dormfiles/USAWI-UW-16612020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.626 seconds.\n",
      "Analyzed Index:   \t2.702 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.294 seconds.\n",
      "Reads/sec:       \t204.23\n",
      "kBases/sec:      \t101.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.964 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16612020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16622020.fasta out=dormfiles/USAWI-UW-16622020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16622020.fasta, out=dormfiles/USAWI-UW-16622020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.775 seconds.\n",
      "Analyzed Index:   \t2.639 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.139 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.358 seconds.\n",
      "Reads/sec:       \t167.43\n",
      "kBases/sec:      \t83.11\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 61.6667% \t       37 \t 61.3861% \t       18282\n",
      "semiperfect site:\t 61.6667% \t       37 \t 61.3861% \t       18282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1841% \t       29539\n",
      "Error Rate:      \t 33.3333% \t       20 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 33.3333% \t       20 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.151 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16622020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16632020.fasta out=dormfiles/USAWI-UW-16632020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16632020.fasta, out=dormfiles/USAWI-UW-16632020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.611 seconds.\n",
      "Analyzed Index:   \t2.883 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.293 seconds.\n",
      "Reads/sec:       \t204.50\n",
      "kBases/sec:      \t101.51\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "\n",
      "Total time:     \t4.283 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16632020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16642020.fasta out=dormfiles/USAWI-UW-16642020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16642020.fasta, out=dormfiles/USAWI-UW-16642020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.606 seconds.\n",
      "Analyzed Index:   \t2.757 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.128 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.292 seconds.\n",
      "Reads/sec:       \t205.19\n",
      "kBases/sec:      \t101.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.019 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16642020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16652020.fasta out=dormfiles/USAWI-UW-16652020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16652020.fasta, out=dormfiles/USAWI-UW-16652020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.626 seconds.\n",
      "Analyzed Index:   \t2.846 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.329 seconds.\n",
      "Reads/sec:       \t182.52\n",
      "kBases/sec:      \t90.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.128 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16652020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16662020.fasta out=dormfiles/USAWI-UW-16662020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16662020.fasta, out=dormfiles/USAWI-UW-16662020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.607 seconds.\n",
      "Analyzed Index:   \t2.687 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t222.82\n",
      "kBases/sec:      \t110.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.865 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16662020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16672020.fasta out=dormfiles/USAWI-UW-16672020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16672020.fasta, out=dormfiles/USAWI-UW-16672020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.628 seconds.\n",
      "Analyzed Index:   \t2.814 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t222.86\n",
      "kBases/sec:      \t110.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16672020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16682020.fasta out=dormfiles/USAWI-UW-16682020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16682020.fasta, out=dormfiles/USAWI-UW-16682020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.853 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.275 seconds.\n",
      "Reads/sec:       \t218.56\n",
      "kBases/sec:      \t108.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 18.3333% \t       11 \t  0.7756% \t         231\n",
      "\n",
      "Total time:     \t4.062 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16682020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16692020.fasta out=dormfiles/USAWI-UW-16692020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16692020.fasta, out=dormfiles/USAWI-UW-16692020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.660 seconds.\n",
      "Analyzed Index:   \t2.858 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.364 seconds.\n",
      "Reads/sec:       \t164.62\n",
      "kBases/sec:      \t81.71\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.536 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16692020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16702020.fasta out=dormfiles/USAWI-UW-16702020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16702020.fasta, out=dormfiles/USAWI-UW-16702020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.621 seconds.\n",
      "Analyzed Index:   \t2.734 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.847 seconds.\n",
      "Reads/sec:       \t70.82\n",
      "kBases/sec:      \t35.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.531 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16702020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16712020.fasta out=dormfiles/USAWI-UW-16712020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16712020.fasta, out=dormfiles/USAWI-UW-16712020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.211 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.619 seconds.\n",
      "Analyzed Index:   \t2.637 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.10\n",
      "kBases/sec:      \t107.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.060 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16712020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16722020.fasta out=dormfiles/USAWI-UW-16722020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16722020.fasta, out=dormfiles/USAWI-UW-16722020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.627 seconds.\n",
      "Analyzed Index:   \t2.800 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.54\n",
      "kBases/sec:      \t106.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.085 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16722020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16732020.fasta out=dormfiles/USAWI-UW-16732020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16732020.fasta, out=dormfiles/USAWI-UW-16732020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.605 seconds.\n",
      "Analyzed Index:   \t2.670 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.270 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.412 seconds.\n",
      "Reads/sec:       \t145.61\n",
      "kBases/sec:      \t72.28\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.200 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16732020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16742020.fasta out=dormfiles/USAWI-UW-16742020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16742020.fasta, out=dormfiles/USAWI-UW-16742020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.616 seconds.\n",
      "Analyzed Index:   \t2.631 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.307 seconds.\n",
      "Reads/sec:       \t195.58\n",
      "kBases/sec:      \t97.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "\n",
      "Total time:     \t3.868 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16742020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16752020.fasta out=dormfiles/USAWI-UW-16752020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16752020.fasta, out=dormfiles/USAWI-UW-16752020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.602 seconds.\n",
      "Analyzed Index:   \t2.970 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.408 seconds.\n",
      "Reads/sec:       \t147.07\n",
      "kBases/sec:      \t73.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.298 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16752020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16762020.fasta out=dormfiles/USAWI-UW-16762020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16762020.fasta, out=dormfiles/USAWI-UW-16762020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t2.530 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.263 seconds.\n",
      "Reads/sec:       \t228.46\n",
      "kBases/sec:      \t113.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 79.8536% \t       23782\n",
      "semiperfect site:\t 80.0000% \t       48 \t 79.8536% \t       23782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 13.3333% \t        8 \t  0.7588% \t         226\n",
      "\n",
      "Total time:     \t3.806 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16762020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16772020.fasta out=dormfiles/USAWI-UW-16772020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16772020.fasta, out=dormfiles/USAWI-UW-16772020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.584 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.15\n",
      "kBases/sec:      \t112.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.743 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16772020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16782020.fasta out=dormfiles/USAWI-UW-16782020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16782020.fasta, out=dormfiles/USAWI-UW-16782020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.593 seconds.\n",
      "Analyzed Index:   \t2.573 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29781 bases)\n",
      "\n",
      "Mapping:          \t0.615 seconds.\n",
      "Reads/sec:       \t97.55\n",
      "kBases/sec:      \t48.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29281\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29281\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6789% \t         500\n",
      "\n",
      "perfect best site:\t 56.6667% \t       34 \t 57.0834% \t       17000\n",
      "semiperfect site:\t 56.6667% \t       34 \t 57.0834% \t       17000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 97.0289% \t       28412\n",
      "Error Rate:      \t 15.2542% \t        9 \t  0.0307% \t           9\n",
      "Sub Rate:        \t 13.5593% \t        8 \t  0.0273% \t           8\n",
      "Del Rate:        \t  1.6949% \t        1 \t  0.0034% \t           1\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 30.5085% \t       18 \t  2.9404% \t         861\n",
      "\n",
      "Total time:     \t4.082 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16782020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16792020.fasta out=dormfiles/USAWI-UW-16792020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16792020.fasta, out=dormfiles/USAWI-UW-16792020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.592 seconds.\n",
      "Analyzed Index:   \t2.573 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.299 seconds.\n",
      "Reads/sec:       \t200.58\n",
      "kBases/sec:      \t99.56\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t3.766 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16792020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16802020.fasta out=dormfiles/USAWI-UW-16802020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16802020.fasta, out=dormfiles/USAWI-UW-16802020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.101 seconds.\n",
      "Analyzed Index:   \t2.551 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.720 seconds.\n",
      "Reads/sec:       \t83.34\n",
      "kBases/sec:      \t41.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "unambiguous:     \t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  5.0000% \t        3 \t  5.0366% \t        1500\n",
      "\n",
      "perfect best site:\t 60.0000% \t       36 \t 60.4392% \t       18000\n",
      "semiperfect site:\t 60.0000% \t       36 \t 60.4392% \t       18000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 97.0592% \t       26965\n",
      "Error Rate:      \t 10.5263% \t        6 \t  0.0216% \t           6\n",
      "Sub Rate:        \t 10.5263% \t        6 \t  0.0216% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 26.3158% \t       15 \t  2.9192% \t         811\n",
      "\n",
      "Total time:     \t4.670 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16802020.fasta.pileup\n",
      "27782 bases in pileup file\n",
      "27782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16812020.fasta out=dormfiles/USAWI-UW-16812020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16812020.fasta, out=dormfiles/USAWI-UW-16812020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.811 seconds.\n",
      "Analyzed Index:   \t2.938 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.465 seconds.\n",
      "Reads/sec:       \t128.90\n",
      "kBases/sec:      \t63.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.528 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16812020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16822020.fasta out=dormfiles/USAWI-UW-16822020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16822020.fasta, out=dormfiles/USAWI-UW-16822020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.661 seconds.\n",
      "Analyzed Index:   \t2.703 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.623 seconds.\n",
      "Reads/sec:       \t96.29\n",
      "kBases/sec:      \t47.80\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1874% \t       29540\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.348 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16822020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16832020.fasta out=dormfiles/USAWI-UW-16832020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16832020.fasta, out=dormfiles/USAWI-UW-16832020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.700 seconds.\n",
      "Analyzed Index:   \t2.629 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.429 seconds.\n",
      "Reads/sec:       \t139.94\n",
      "kBases/sec:      \t69.46\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9522% \t       29268\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.130 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16832020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16842020.fasta out=dormfiles/USAWI-UW-16842020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16842020.fasta, out=dormfiles/USAWI-UW-16842020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.044 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.931 seconds.\n",
      "Analyzed Index:   \t3.254 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t211.21\n",
      "kBases/sec:      \t104.84\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  8.3333% \t        5 \t  0.7454% \t         222\n",
      "\n",
      "Total time:     \t5.127 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16842020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16852020.fasta out=dormfiles/USAWI-UW-16852020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16852020.fasta, out=dormfiles/USAWI-UW-16852020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.671 seconds.\n",
      "Analyzed Index:   \t2.827 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.495 seconds.\n",
      "Reads/sec:       \t121.31\n",
      "kBases/sec:      \t60.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.503 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16852020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16862020.fasta out=dormfiles/USAWI-UW-16862020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16862020.fasta, out=dormfiles/USAWI-UW-16862020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.609 seconds.\n",
      "Analyzed Index:   \t2.625 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.600 seconds.\n",
      "Reads/sec:       \t99.96\n",
      "kBases/sec:      \t49.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1874% \t       29540\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.146 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16862020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16872020.fasta out=dormfiles/USAWI-UW-16872020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16872020.fasta, out=dormfiles/USAWI-UW-16872020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.611 seconds.\n",
      "Analyzed Index:   \t2.590 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.596 seconds.\n",
      "Reads/sec:       \t100.60\n",
      "kBases/sec:      \t49.93\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.102 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16872020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16882020.fasta out=dormfiles/USAWI-UW-16882020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16882020.fasta, out=dormfiles/USAWI-UW-16882020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.594 seconds.\n",
      "Analyzed Index:   \t2.523 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.374 seconds.\n",
      "Reads/sec:       \t160.23\n",
      "kBases/sec:      \t79.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.818 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16882020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16892020.fasta out=dormfiles/USAWI-UW-16892020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16892020.fasta, out=dormfiles/USAWI-UW-16892020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.584 seconds.\n",
      "Analyzed Index:   \t2.481 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t223.16\n",
      "kBases/sec:      \t110.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7454% \t         222\n",
      "\n",
      "Total time:     \t3.650 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16892020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16902020.fasta out=dormfiles/USAWI-UW-16902020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16902020.fasta, out=dormfiles/USAWI-UW-16902020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.607 seconds.\n",
      "Analyzed Index:   \t2.637 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.249 seconds.\n",
      "Reads/sec:       \t240.50\n",
      "kBases/sec:      \t119.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t3.825 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16902020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16912020.fasta out=dormfiles/USAWI-UW-16912020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16912020.fasta, out=dormfiles/USAWI-UW-16912020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.619 seconds.\n",
      "Analyzed Index:   \t2.618 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.152 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.344 seconds.\n",
      "Reads/sec:       \t174.30\n",
      "kBases/sec:      \t86.52\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1841% \t       29539\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t3.970 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16912020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16922020.fasta out=dormfiles/USAWI-UW-16922020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16922020.fasta, out=dormfiles/USAWI-UW-16922020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.640 seconds.\n",
      "Analyzed Index:   \t2.847 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.119 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.407 seconds.\n",
      "Reads/sec:       \t147.49\n",
      "kBases/sec:      \t73.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.554 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16922020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16932020.fasta out=dormfiles/USAWI-UW-16932020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16932020.fasta, out=dormfiles/USAWI-UW-16932020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.617 seconds.\n",
      "Analyzed Index:   \t2.963 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.462 seconds.\n",
      "Reads/sec:       \t129.74\n",
      "kBases/sec:      \t64.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.367 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16932020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16942020.fasta out=dormfiles/USAWI-UW-16942020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16942020.fasta, out=dormfiles/USAWI-UW-16942020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.621 seconds.\n",
      "Analyzed Index:   \t2.784 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.277 seconds.\n",
      "Reads/sec:       \t216.78\n",
      "kBases/sec:      \t107.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.185 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16942020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16952020.fasta out=dormfiles/USAWI-UW-16952020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16952020.fasta, out=dormfiles/USAWI-UW-16952020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.470 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.14\n",
      "kBases/sec:      \t107.78\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.641 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16952020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16962020.fasta out=dormfiles/USAWI-UW-16962020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16962020.fasta, out=dormfiles/USAWI-UW-16962020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.679 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.474 seconds.\n",
      "Reads/sec:       \t126.58\n",
      "kBases/sec:      \t62.83\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16962020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16972020.fasta out=dormfiles/USAWI-UW-16972020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16972020.fasta, out=dormfiles/USAWI-UW-16972020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.621 seconds.\n",
      "Analyzed Index:   \t2.834 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.304 seconds.\n",
      "Reads/sec:       \t197.33\n",
      "kBases/sec:      \t97.95\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.103 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16972020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16982020.fasta out=dormfiles/USAWI-UW-16982020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16982020.fasta, out=dormfiles/USAWI-UW-16982020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.617 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.154 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.287 seconds.\n",
      "Reads/sec:       \t209.01\n",
      "kBases/sec:      \t103.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.895 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16982020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-16992020.fasta out=dormfiles/USAWI-UW-16992020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-16992020.fasta, out=dormfiles/USAWI-UW-16992020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.744 seconds.\n",
      "Analyzed Index:   \t2.602 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.288 seconds.\n",
      "Reads/sec:       \t208.29\n",
      "kBases/sec:      \t103.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.305 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-16992020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17002020.fasta out=dormfiles/USAWI-UW-17002020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17002020.fasta, out=dormfiles/USAWI-UW-17002020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.606 seconds.\n",
      "Analyzed Index:   \t2.567 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.105 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.261 seconds.\n",
      "Reads/sec:       \t229.82\n",
      "kBases/sec:      \t114.07\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.779 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17002020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17012020.fasta out=dormfiles/USAWI-UW-17012020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17012020.fasta, out=dormfiles/USAWI-UW-17012020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.968 seconds.\n",
      "Analyzed Index:   \t2.573 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.267 seconds.\n",
      "Reads/sec:       \t224.60\n",
      "kBases/sec:      \t111.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.115 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17012020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17022020.fasta out=dormfiles/USAWI-UW-17022020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17022020.fasta, out=dormfiles/USAWI-UW-17022020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.587 seconds.\n",
      "Analyzed Index:   \t2.533 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.381 seconds.\n",
      "Reads/sec:       \t157.30\n",
      "kBases/sec:      \t78.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.805 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17022020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17032020.fasta out=dormfiles/USAWI-UW-17032020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17032020.fasta, out=dormfiles/USAWI-UW-17032020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.171 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.544 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t226.90\n",
      "kBases/sec:      \t112.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9488% \t       29267\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.864 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17032020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17042020.fasta out=dormfiles/USAWI-UW-17042020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17042020.fasta, out=dormfiles/USAWI-UW-17042020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.609 seconds.\n",
      "Analyzed Index:   \t2.601 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.268 seconds.\n",
      "Reads/sec:       \t223.72\n",
      "kBases/sec:      \t111.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.788 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17042020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17052020.fasta out=dormfiles/USAWI-UW-17052020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17052020.fasta, out=dormfiles/USAWI-UW-17052020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.631 seconds.\n",
      "Analyzed Index:   \t2.599 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t222.77\n",
      "kBases/sec:      \t110.57\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.798 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17052020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17062020.fasta out=dormfiles/USAWI-UW-17062020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17062020.fasta, out=dormfiles/USAWI-UW-17062020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.609 seconds.\n",
      "Analyzed Index:   \t2.792 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.314 seconds.\n",
      "Reads/sec:       \t190.93\n",
      "kBases/sec:      \t94.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.025 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17062020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17072020.fasta out=dormfiles/USAWI-UW-17072020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17072020.fasta, out=dormfiles/USAWI-UW-17072020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.670 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.323 seconds.\n",
      "Reads/sec:       \t185.80\n",
      "kBases/sec:      \t92.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.895 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17072020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17082020.fasta out=dormfiles/USAWI-UW-17082020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17082020.fasta, out=dormfiles/USAWI-UW-17082020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.511 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.272 seconds.\n",
      "Reads/sec:       \t220.54\n",
      "kBases/sec:      \t109.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2176% \t       29549\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.683 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17082020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17092020.fasta out=dormfiles/USAWI-UW-17092020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17092020.fasta, out=dormfiles/USAWI-UW-17092020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.669 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.54\n",
      "kBases/sec:      \t106.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.860 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17092020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17102020.fasta out=dormfiles/USAWI-UW-17102020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17102020.fasta, out=dormfiles/USAWI-UW-17102020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.620 seconds.\n",
      "Analyzed Index:   \t3.113 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.428 seconds.\n",
      "Reads/sec:       \t140.28\n",
      "kBases/sec:      \t69.63\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.468 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17102020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17112020.fasta out=dormfiles/USAWI-UW-17112020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17112020.fasta, out=dormfiles/USAWI-UW-17112020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.866 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.306 seconds.\n",
      "Reads/sec:       \t196.05\n",
      "kBases/sec:      \t97.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.7314% \t       29702\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.2115% \t          63\n",
      "\n",
      "Total time:     \t4.060 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17112020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17122020.fasta out=dormfiles/USAWI-UW-17122020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17122020.fasta, out=dormfiles/USAWI-UW-17122020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.587 seconds.\n",
      "Analyzed Index:   \t2.453 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t228.98\n",
      "kBases/sec:      \t113.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.0472% \t       29003\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0444% \t          13\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0444% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 11.6667% \t        7 \t  0.9084% \t         266\n",
      "\n",
      "Total time:     \t3.605 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17122020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17132020.fasta out=dormfiles/USAWI-UW-17132020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17132020.fasta, out=dormfiles/USAWI-UW-17132020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.589 seconds.\n",
      "Analyzed Index:   \t2.506 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.258 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.261 seconds.\n",
      "Reads/sec:       \t230.15\n",
      "kBases/sec:      \t114.24\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.851 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17132020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17142020.fasta out=dormfiles/USAWI-UW-17142020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17142020.fasta, out=dormfiles/USAWI-UW-17142020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.516 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.345 seconds.\n",
      "Reads/sec:       \t173.80\n",
      "kBases/sec:      \t86.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.749 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17142020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17152020.fasta out=dormfiles/USAWI-UW-17152020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17152020.fasta, out=dormfiles/USAWI-UW-17152020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.589 seconds.\n",
      "Analyzed Index:   \t2.507 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.447 seconds.\n",
      "Reads/sec:       \t134.30\n",
      "kBases/sec:      \t66.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.843 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17152020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17162020.fasta out=dormfiles/USAWI-UW-17162020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17162020.fasta, out=dormfiles/USAWI-UW-17162020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.610 seconds.\n",
      "Analyzed Index:   \t2.804 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.122 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.612 seconds.\n",
      "Reads/sec:       \t98.08\n",
      "kBases/sec:      \t48.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.389 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17162020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17172020.fasta out=dormfiles/USAWI-UW-17172020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17172020.fasta, out=dormfiles/USAWI-UW-17172020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.934 seconds.\n",
      "Analyzed Index:   \t3.461 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.143 seconds.\n",
      "Cleared Memory:    \t0.135 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.540 seconds.\n",
      "Reads/sec:       \t111.07\n",
      "kBases/sec:      \t55.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t5.482 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17172020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17182020.fasta out=dormfiles/USAWI-UW-17182020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17182020.fasta, out=dormfiles/USAWI-UW-17182020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.035 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.862 seconds.\n",
      "Analyzed Index:   \t3.555 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.119 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.446 seconds.\n",
      "Reads/sec:       \t134.42\n",
      "kBases/sec:      \t66.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9454% \t       29266\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0478% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0068% \t           2\n",
      "\n",
      "Total time:     \t5.276 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17182020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17192020.fasta out=dormfiles/USAWI-UW-17192020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17192020.fasta, out=dormfiles/USAWI-UW-17192020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.710 seconds.\n",
      "Analyzed Index:   \t2.785 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.671 seconds.\n",
      "Reads/sec:       \t89.38\n",
      "kBases/sec:      \t44.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.495 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17192020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17202020.fasta out=dormfiles/USAWI-UW-17202020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17202020.fasta, out=dormfiles/USAWI-UW-17202020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.637 seconds.\n",
      "Analyzed Index:   \t2.729 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.598 seconds.\n",
      "Reads/sec:       \t100.27\n",
      "kBases/sec:      \t49.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.270 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17202020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17212020.fasta out=dormfiles/USAWI-UW-17212020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17212020.fasta, out=dormfiles/USAWI-UW-17212020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.764 seconds.\n",
      "Analyzed Index:   \t2.686 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.371 seconds.\n",
      "Reads/sec:       \t161.59\n",
      "kBases/sec:      \t80.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.309 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17212020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17222020.fasta out=dormfiles/USAWI-UW-17222020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17222020.fasta, out=dormfiles/USAWI-UW-17222020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.749 seconds.\n",
      "Analyzed Index:   \t2.729 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.418 seconds.\n",
      "Reads/sec:       \t143.49\n",
      "kBases/sec:      \t71.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.269 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17222020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17232020.fasta out=dormfiles/USAWI-UW-17232020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17232020.fasta, out=dormfiles/USAWI-UW-17232020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.623 seconds.\n",
      "Analyzed Index:   \t2.627 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.476 seconds.\n",
      "Reads/sec:       \t126.06\n",
      "kBases/sec:      \t62.57\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.035 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17232020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17242020.fasta out=dormfiles/USAWI-UW-17242020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17242020.fasta, out=dormfiles/USAWI-UW-17242020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.592 seconds.\n",
      "Analyzed Index:   \t2.528 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29733 bases)\n",
      "\n",
      "Mapping:          \t0.340 seconds.\n",
      "Reads/sec:       \t176.46\n",
      "kBases/sec:      \t87.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 91.6667% \t       55 \t 91.5918% \t       27233\n",
      "unambiguous:     \t 91.6667% \t       55 \t 91.5918% \t       27233\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  5.0000% \t        3 \t  5.0449% \t        1500\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4122% \t       21233\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4122% \t       21233\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 98.0795% \t       26710\n",
      "Error Rate:      \t 10.5263% \t        6 \t  0.0220% \t           6\n",
      "Sub Rate:        \t 10.5263% \t        6 \t  0.0220% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 12.2807% \t        7 \t  1.8984% \t         517\n",
      "\n",
      "Total time:     \t3.765 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17242020.fasta.pileup\n",
      "27233 bases in pileup file\n",
      "27233 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17252020.fasta out=dormfiles/USAWI-UW-17252020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17252020.fasta, out=dormfiles/USAWI-UW-17252020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.691 seconds.\n",
      "Analyzed Index:   \t2.554 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.537 seconds.\n",
      "Reads/sec:       \t111.78\n",
      "kBases/sec:      \t55.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.272 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17252020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17262020.fasta out=dormfiles/USAWI-UW-17262020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17262020.fasta, out=dormfiles/USAWI-UW-17262020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.211 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.647 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.493 seconds.\n",
      "Reads/sec:       \t121.79\n",
      "kBases/sec:      \t60.45\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.249 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17262020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17272020.fasta out=dormfiles/USAWI-UW-17272020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17272020.fasta, out=dormfiles/USAWI-UW-17272020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.871 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.620 seconds.\n",
      "Reads/sec:       \t96.74\n",
      "kBases/sec:      \t48.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 13.3333% \t        8 \t  0.7555% \t         225\n",
      "\n",
      "Total time:     \t4.434 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17272020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17282020.fasta out=dormfiles/USAWI-UW-17282020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17282020.fasta, out=dormfiles/USAWI-UW-17282020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.740 seconds.\n",
      "Analyzed Index:   \t2.884 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.512 seconds.\n",
      "Reads/sec:       \t117.10\n",
      "kBases/sec:      \t58.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.453 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17282020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17292020.fasta out=dormfiles/USAWI-UW-17292020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17292020.fasta, out=dormfiles/USAWI-UW-17292020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.643 seconds.\n",
      "Analyzed Index:   \t2.574 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.376 seconds.\n",
      "Reads/sec:       \t159.64\n",
      "kBases/sec:      \t79.24\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.310 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17292020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17302020.fasta out=dormfiles/USAWI-UW-17302020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17302020.fasta, out=dormfiles/USAWI-UW-17302020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.632 seconds.\n",
      "Analyzed Index:   \t2.732 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.478 seconds.\n",
      "Reads/sec:       \t125.62\n",
      "kBases/sec:      \t62.35\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.224 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17302020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17312020.fasta out=dormfiles/USAWI-UW-17312020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17312020.fasta, out=dormfiles/USAWI-UW-17312020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.695 seconds.\n",
      "Analyzed Index:   \t2.776 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.329 seconds.\n",
      "Reads/sec:       \t182.64\n",
      "kBases/sec:      \t90.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.159 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17312020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17322020.fasta out=dormfiles/USAWI-UW-17322020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17322020.fasta, out=dormfiles/USAWI-UW-17322020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.625 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.223 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.275 seconds.\n",
      "Reads/sec:       \t218.12\n",
      "kBases/sec:      \t108.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.977 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17322020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17332020.fasta out=dormfiles/USAWI-UW-17332020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17332020.fasta, out=dormfiles/USAWI-UW-17332020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.610 seconds.\n",
      "Analyzed Index:   \t2.719 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.231 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.608 seconds.\n",
      "Reads/sec:       \t98.72\n",
      "kBases/sec:      \t49.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.405 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17332020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17342020.fasta out=dormfiles/USAWI-UW-17342020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17342020.fasta, out=dormfiles/USAWI-UW-17342020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.656 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.288 seconds.\n",
      "Reads/sec:       \t208.38\n",
      "kBases/sec:      \t103.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.923 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17342020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17352020.fasta out=dormfiles/USAWI-UW-17352020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17352020.fasta, out=dormfiles/USAWI-UW-17352020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.611 seconds.\n",
      "Analyzed Index:   \t2.518 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.387 seconds.\n",
      "Reads/sec:       \t155.17\n",
      "kBases/sec:      \t77.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.821 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17352020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17362020.fasta out=dormfiles/USAWI-UW-17362020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17362020.fasta, out=dormfiles/USAWI-UW-17362020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.730 seconds.\n",
      "Analyzed Index:   \t2.637 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.457 seconds.\n",
      "Reads/sec:       \t131.29\n",
      "kBases/sec:      \t65.17\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.133 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17362020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17372020.fasta out=dormfiles/USAWI-UW-17372020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17372020.fasta, out=dormfiles/USAWI-UW-17372020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.685 seconds.\n",
      "Analyzed Index:   \t2.910 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.453 seconds.\n",
      "Reads/sec:       \t132.57\n",
      "kBases/sec:      \t65.81\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.431 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17372020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17382020.fasta out=dormfiles/USAWI-UW-17382020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17382020.fasta, out=dormfiles/USAWI-UW-17382020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.621 seconds.\n",
      "Analyzed Index:   \t2.630 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.419 seconds.\n",
      "Reads/sec:       \t143.35\n",
      "kBases/sec:      \t71.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1841% \t       29539\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.997 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17382020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17392020.fasta out=dormfiles/USAWI-UW-17392020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17392020.fasta, out=dormfiles/USAWI-UW-17392020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.741 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.476 seconds.\n",
      "Reads/sec:       \t126.01\n",
      "kBases/sec:      \t62.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.309 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17392020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17402020.fasta out=dormfiles/USAWI-UW-17402020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17402020.fasta, out=dormfiles/USAWI-UW-17402020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.503 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.300 seconds.\n",
      "Reads/sec:       \t200.24\n",
      "kBases/sec:      \t99.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.720 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17402020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17412020.fasta out=dormfiles/USAWI-UW-17412020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17412020.fasta, out=dormfiles/USAWI-UW-17412020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.767 seconds.\n",
      "Analyzed Index:   \t2.801 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.103 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.303 seconds.\n",
      "Reads/sec:       \t197.76\n",
      "kBases/sec:      \t98.16\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.219 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17412020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17422020.fasta out=dormfiles/USAWI-UW-17422020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17422020.fasta, out=dormfiles/USAWI-UW-17422020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.599 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.200 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.258 seconds.\n",
      "Reads/sec:       \t232.26\n",
      "kBases/sec:      \t115.29\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.891 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17422020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17432020.fasta out=dormfiles/USAWI-UW-17432020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17432020.fasta, out=dormfiles/USAWI-UW-17432020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.885 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.347 seconds.\n",
      "Reads/sec:       \t172.77\n",
      "kBases/sec:      \t85.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.148 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17432020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17442020.fasta out=dormfiles/USAWI-UW-17442020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17442020.fasta, out=dormfiles/USAWI-UW-17442020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.614 seconds.\n",
      "Analyzed Index:   \t2.642 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.243 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.379 seconds.\n",
      "Reads/sec:       \t158.29\n",
      "kBases/sec:      \t78.57\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.119 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17442020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17452020.fasta out=dormfiles/USAWI-UW-17452020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17452020.fasta, out=dormfiles/USAWI-UW-17452020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.645 seconds.\n",
      "Analyzed Index:   \t2.593 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.259 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t228.64\n",
      "kBases/sec:      \t113.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.997 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17452020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17462020.fasta out=dormfiles/USAWI-UW-17462020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17462020.fasta, out=dormfiles/USAWI-UW-17462020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.605 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.449 seconds.\n",
      "Reads/sec:       \t133.77\n",
      "kBases/sec:      \t66.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 10.0000% \t        6 \t  0.7488% \t         223\n",
      "\n",
      "Total time:     \t3.956 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17462020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17472020.fasta out=dormfiles/USAWI-UW-17472020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17472020.fasta, out=dormfiles/USAWI-UW-17472020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.601 seconds.\n",
      "Analyzed Index:   \t2.617 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.524 seconds.\n",
      "Reads/sec:       \t114.41\n",
      "kBases/sec:      \t56.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.172 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17472020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17482020.fasta out=dormfiles/USAWI-UW-17482020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17482020.fasta, out=dormfiles/USAWI-UW-17482020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.742 seconds.\n",
      "Analyzed Index:   \t2.682 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.437 seconds.\n",
      "Reads/sec:       \t137.25\n",
      "kBases/sec:      \t68.12\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.265 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17482020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17492020.fasta out=dormfiles/USAWI-UW-17492020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17492020.fasta, out=dormfiles/USAWI-UW-17492020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.600 seconds.\n",
      "Analyzed Index:   \t2.482 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.288 seconds.\n",
      "Reads/sec:       \t208.57\n",
      "kBases/sec:      \t103.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t3.675 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17492020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17502020.fasta out=dormfiles/USAWI-UW-17502020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17502020.fasta, out=dormfiles/USAWI-UW-17502020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.583 seconds.\n",
      "Analyzed Index:   \t2.431 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t212.63\n",
      "kBases/sec:      \t105.54\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2111% \t       29051\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0171% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0171% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 15.0000% \t        9 \t  0.7718% \t         226\n",
      "\n",
      "Total time:     \t3.597 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17502020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17512020.fasta out=dormfiles/USAWI-UW-17512020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17512020.fasta, out=dormfiles/USAWI-UW-17512020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.594 seconds.\n",
      "Analyzed Index:   \t3.312 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.100 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.526 seconds.\n",
      "Reads/sec:       \t114.05\n",
      "kBases/sec:      \t56.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.875 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17512020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17522020.fasta out=dormfiles/USAWI-UW-17522020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17522020.fasta, out=dormfiles/USAWI-UW-17522020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.034 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.930 seconds.\n",
      "Analyzed Index:   \t4.246 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.342 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.347 seconds.\n",
      "Reads/sec:       \t172.75\n",
      "kBases/sec:      \t85.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.208 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17522020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17532020.fasta out=dormfiles/USAWI-UW-17532020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17532020.fasta, out=dormfiles/USAWI-UW-17532020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.692 seconds.\n",
      "Analyzed Index:   \t2.575 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.309 seconds.\n",
      "Reads/sec:       \t193.99\n",
      "kBases/sec:      \t96.29\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9530% \t       29768\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.912 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17532020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17542020.fasta out=dormfiles/USAWI-UW-17542020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17542020.fasta, out=dormfiles/USAWI-UW-17542020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.513 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.366 seconds.\n",
      "Reads/sec:       \t164.04\n",
      "kBases/sec:      \t81.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.791 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17542020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17552020.fasta out=dormfiles/USAWI-UW-17552020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17552020.fasta, out=dormfiles/USAWI-UW-17552020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.654 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.191 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.349 seconds.\n",
      "Reads/sec:       \t171.86\n",
      "kBases/sec:      \t85.30\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.028 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17552020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17562020.fasta out=dormfiles/USAWI-UW-17562020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17562020.fasta, out=dormfiles/USAWI-UW-17562020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.605 seconds.\n",
      "Analyzed Index:   \t2.607 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.273 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.301 seconds.\n",
      "Reads/sec:       \t199.40\n",
      "kBases/sec:      \t98.97\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.018 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17562020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17572020.fasta out=dormfiles/USAWI-UW-17572020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17572020.fasta, out=dormfiles/USAWI-UW-17572020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.635 seconds.\n",
      "Analyzed Index:   \t2.674 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.301 seconds.\n",
      "Reads/sec:       \t199.47\n",
      "kBases/sec:      \t99.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t3.927 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17572020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17582020.fasta out=dormfiles/USAWI-UW-17582020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17582020.fasta, out=dormfiles/USAWI-UW-17582020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.613 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.675 seconds.\n",
      "Reads/sec:       \t88.91\n",
      "kBases/sec:      \t44.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.186 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17582020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17592020.fasta out=dormfiles/USAWI-UW-17592020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17592020.fasta, out=dormfiles/USAWI-UW-17592020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.766 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.525 seconds.\n",
      "Reads/sec:       \t114.35\n",
      "kBases/sec:      \t56.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9295% \t       29761\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.206 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17592020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17602020.fasta out=dormfiles/USAWI-UW-17602020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17602020.fasta, out=dormfiles/USAWI-UW-17602020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.634 seconds.\n",
      "Analyzed Index:   \t2.845 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.308 seconds.\n",
      "Reads/sec:       \t195.00\n",
      "kBases/sec:      \t96.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.116 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17602020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17612020.fasta out=dormfiles/USAWI-UW-17612020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17612020.fasta, out=dormfiles/USAWI-UW-17612020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.611 seconds.\n",
      "Analyzed Index:   \t2.909 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.432 seconds.\n",
      "Reads/sec:       \t138.79\n",
      "kBases/sec:      \t68.89\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.289 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17612020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17622020.fasta out=dormfiles/USAWI-UW-17622020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17622020.fasta, out=dormfiles/USAWI-UW-17622020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.709 seconds.\n",
      "Analyzed Index:   \t2.728 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.580 seconds.\n",
      "Reads/sec:       \t103.49\n",
      "kBases/sec:      \t51.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.335 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17622020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17632020.fasta out=dormfiles/USAWI-UW-17632020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17632020.fasta, out=dormfiles/USAWI-UW-17632020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.588 seconds.\n",
      "Analyzed Index:   \t2.603 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.422 seconds.\n",
      "Reads/sec:       \t142.02\n",
      "kBases/sec:      \t70.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.926 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17632020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17642020.fasta out=dormfiles/USAWI-UW-17642020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17642020.fasta, out=dormfiles/USAWI-UW-17642020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.865 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.183 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.676 seconds.\n",
      "Reads/sec:       \t88.80\n",
      "kBases/sec:      \t44.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9295% \t       29761\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.560 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17642020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17652020.fasta out=dormfiles/USAWI-UW-17652020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17652020.fasta, out=dormfiles/USAWI-UW-17652020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.648 seconds.\n",
      "Analyzed Index:   \t2.591 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.128 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.462 seconds.\n",
      "Reads/sec:       \t129.77\n",
      "kBases/sec:      \t64.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0672% \t          20\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0672% \t          20\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.109 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17652020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "20 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17662020.fasta out=dormfiles/USAWI-UW-17662020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17662020.fasta, out=dormfiles/USAWI-UW-17662020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.715 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.417 seconds.\n",
      "Reads/sec:       \t143.71\n",
      "kBases/sec:      \t71.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9488% \t       29267\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0410% \t          12\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0410% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0102% \t           3\n",
      "\n",
      "Total time:     \t4.077 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17662020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17672020.fasta out=dormfiles/USAWI-UW-17672020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17672020.fasta, out=dormfiles/USAWI-UW-17672020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.809 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.214 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.316 seconds.\n",
      "Reads/sec:       \t190.11\n",
      "kBases/sec:      \t94.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 96.6667% \t       58 \t 96.6423% \t       28782\n",
      "unambiguous:     \t 96.6667% \t       58 \t 96.6423% \t       28782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  3.3333% \t        2 \t  3.3577% \t        1000\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 98.1099% \t       28238\n",
      "Error Rate:      \t 24.1379% \t       14 \t  0.0625% \t          18\n",
      "Sub Rate:        \t 24.1379% \t       14 \t  0.0625% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  8.6207% \t        5 \t  1.8275% \t         526\n",
      "\n",
      "Total time:     \t4.180 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17672020.fasta.pileup\n",
      "28782 bases in pileup file\n",
      "28782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17682020.fasta out=dormfiles/USAWI-UW-17682020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17682020.fasta, out=dormfiles/USAWI-UW-17682020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.036 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.729 seconds.\n",
      "Analyzed Index:   \t3.198 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29783 bases)\n",
      "\n",
      "Mapping:          \t0.531 seconds.\n",
      "Reads/sec:       \t112.98\n",
      "kBases/sec:      \t56.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29783\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29783\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1524% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1524% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29758\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.854 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17682020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17692020.fasta out=dormfiles/USAWI-UW-17692020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17692020.fasta, out=dormfiles/USAWI-UW-17692020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.600 seconds.\n",
      "Analyzed Index:   \t2.740 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.308 seconds.\n",
      "Reads/sec:       \t194.52\n",
      "kBases/sec:      \t96.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.979 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17692020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17702020.fasta out=dormfiles/USAWI-UW-17702020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17702020.fasta, out=dormfiles/USAWI-UW-17702020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.645 seconds.\n",
      "Analyzed Index:   \t2.557 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.286 seconds.\n",
      "Reads/sec:       \t209.96\n",
      "kBases/sec:      \t104.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.797 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17702020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17712020.fasta out=dormfiles/USAWI-UW-17712020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17712020.fasta, out=dormfiles/USAWI-UW-17712020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.628 seconds.\n",
      "Analyzed Index:   \t2.719 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.110 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.405 seconds.\n",
      "Reads/sec:       \t148.30\n",
      "kBases/sec:      \t73.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.126 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17712020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17722020.fasta out=dormfiles/USAWI-UW-17722020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17722020.fasta, out=dormfiles/USAWI-UW-17722020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.629 seconds.\n",
      "Analyzed Index:   \t2.665 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.257 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.311 seconds.\n",
      "Reads/sec:       \t192.81\n",
      "kBases/sec:      \t95.70\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2277% \t       29552\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.104 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17722020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17732020.fasta out=dormfiles/USAWI-UW-17732020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17732020.fasta, out=dormfiles/USAWI-UW-17732020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.606 seconds.\n",
      "Analyzed Index:   \t2.904 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.190 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.322 seconds.\n",
      "Reads/sec:       \t186.22\n",
      "kBases/sec:      \t92.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 15.0000% \t        9 \t  0.7588% \t         226\n",
      "\n",
      "Total time:     \t4.263 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17732020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17742020.fasta out=dormfiles/USAWI-UW-17742020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17742020.fasta, out=dormfiles/USAWI-UW-17742020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.586 seconds.\n",
      "Analyzed Index:   \t2.866 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.164 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.464 seconds.\n",
      "Reads/sec:       \t129.20\n",
      "kBases/sec:      \t64.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.324 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17742020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17752020.fasta out=dormfiles/USAWI-UW-17752020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17752020.fasta, out=dormfiles/USAWI-UW-17752020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.632 seconds.\n",
      "Analyzed Index:   \t2.604 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.04\n",
      "kBases/sec:      \t107.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.841 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17752020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17762020.fasta out=dormfiles/USAWI-UW-17762020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17762020.fasta, out=dormfiles/USAWI-UW-17762020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.709 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.467 seconds.\n",
      "Reads/sec:       \t128.57\n",
      "kBases/sec:      \t63.82\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.083 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17762020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17772020.fasta out=dormfiles/USAWI-UW-17772020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17772020.fasta, out=dormfiles/USAWI-UW-17772020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.630 seconds.\n",
      "Analyzed Index:   \t2.636 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.351 seconds.\n",
      "Reads/sec:       \t170.70\n",
      "kBases/sec:      \t84.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.948 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17772020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17782020.fasta out=dormfiles/USAWI-UW-17782020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17782020.fasta, out=dormfiles/USAWI-UW-17782020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.601 seconds.\n",
      "Analyzed Index:   \t2.972 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.625 seconds.\n",
      "Reads/sec:       \t96.01\n",
      "kBases/sec:      \t47.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 58.3333% \t       35 \t 58.7603% \t       17500\n",
      "semiperfect site:\t 58.3333% \t       35 \t 58.7603% \t       17500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1706% \t       29535\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 11.6667% \t        7 \t  0.7521% \t         224\n",
      "\n",
      "Total time:     \t4.530 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17782020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17792020.fasta out=dormfiles/USAWI-UW-17792020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17792020.fasta, out=dormfiles/USAWI-UW-17792020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.805 seconds.\n",
      "Analyzed Index:   \t2.854 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.414 seconds.\n",
      "Reads/sec:       \t144.87\n",
      "kBases/sec:      \t71.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.436 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17792020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17802020.fasta out=dormfiles/USAWI-UW-17802020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17802020.fasta, out=dormfiles/USAWI-UW-17802020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.610 seconds.\n",
      "Analyzed Index:   \t2.545 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.43\n",
      "kBases/sec:      \t106.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9362% \t       29763\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.757 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17802020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17812020.fasta out=dormfiles/USAWI-UW-17812020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17812020.fasta, out=dormfiles/USAWI-UW-17812020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.629 seconds.\n",
      "Analyzed Index:   \t3.817 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.123 seconds.\n",
      "Cleared Memory:    \t0.117 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.517 seconds.\n",
      "Reads/sec:       \t116.09\n",
      "kBases/sec:      \t57.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.333 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17812020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17822020.fasta out=dormfiles/USAWI-UW-17822020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17822020.fasta, out=dormfiles/USAWI-UW-17822020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.039 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.195 seconds.\n",
      "Analyzed Index:   \t3.078 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.337 seconds.\n",
      "Reads/sec:       \t177.93\n",
      "kBases/sec:      \t88.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.041 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17822020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17832020.fasta out=dormfiles/USAWI-UW-17832020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17832020.fasta, out=dormfiles/USAWI-UW-17832020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.715 seconds.\n",
      "Analyzed Index:   \t2.958 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.517 seconds.\n",
      "Reads/sec:       \t115.97\n",
      "kBases/sec:      \t57.56\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.682 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17832020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17842020.fasta out=dormfiles/USAWI-UW-17842020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17842020.fasta, out=dormfiles/USAWI-UW-17842020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.610 seconds.\n",
      "Analyzed Index:   \t2.860 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.408 seconds.\n",
      "Reads/sec:       \t146.97\n",
      "kBases/sec:      \t72.95\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.192 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17842020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17852020.fasta out=dormfiles/USAWI-UW-17852020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17852020.fasta, out=dormfiles/USAWI-UW-17852020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.739 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.404 seconds.\n",
      "Reads/sec:       \t148.43\n",
      "kBases/sec:      \t73.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1874% \t       29540\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.065 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17852020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17862020.fasta out=dormfiles/USAWI-UW-17862020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17862020.fasta, out=dormfiles/USAWI-UW-17862020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.628 seconds.\n",
      "Analyzed Index:   \t2.532 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t228.62\n",
      "kBases/sec:      \t113.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.0506% \t       29004\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0478% \t          14\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0478% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 11.6667% \t        7 \t  0.9016% \t         264\n",
      "\n",
      "Total time:     \t3.773 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17862020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17872020.fasta out=dormfiles/USAWI-UW-17872020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17872020.fasta, out=dormfiles/USAWI-UW-17872020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.765 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.567 seconds.\n",
      "Reads/sec:       \t105.91\n",
      "kBases/sec:      \t52.57\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.245 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17872020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17882020.fasta out=dormfiles/USAWI-UW-17882020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17882020.fasta, out=dormfiles/USAWI-UW-17882020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.735 seconds.\n",
      "Analyzed Index:   \t2.901 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.617 seconds.\n",
      "Reads/sec:       \t97.21\n",
      "kBases/sec:      \t48.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.652 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17882020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17892020.fasta out=dormfiles/USAWI-UW-17892020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17892020.fasta, out=dormfiles/USAWI-UW-17892020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.825 seconds.\n",
      "Analyzed Index:   \t2.910 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.255 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.416 seconds.\n",
      "Reads/sec:       \t144.37\n",
      "kBases/sec:      \t71.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.815 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17892020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17902020.fasta out=dormfiles/USAWI-UW-17902020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17902020.fasta, out=dormfiles/USAWI-UW-17902020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.678 seconds.\n",
      "Analyzed Index:   \t2.621 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.418 seconds.\n",
      "Reads/sec:       \t143.69\n",
      "kBases/sec:      \t71.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17902020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17912020.fasta out=dormfiles/USAWI-UW-17912020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17912020.fasta, out=dormfiles/USAWI-UW-17912020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.670 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.189 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.273 seconds.\n",
      "Reads/sec:       \t220.06\n",
      "kBases/sec:      \t109.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t3.972 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17912020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17922020.fasta out=dormfiles/USAWI-UW-17922020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17922020.fasta, out=dormfiles/USAWI-UW-17922020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.764 seconds.\n",
      "Analyzed Index:   \t2.851 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.448 seconds.\n",
      "Reads/sec:       \t134.01\n",
      "kBases/sec:      \t66.52\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.671 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17922020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17932020.fasta out=dormfiles/USAWI-UW-17932020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17932020.fasta, out=dormfiles/USAWI-UW-17932020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.650 seconds.\n",
      "Analyzed Index:   \t2.524 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.292 seconds.\n",
      "Reads/sec:       \t205.63\n",
      "kBases/sec:      \t102.07\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7454% \t         222\n",
      "\n",
      "Total time:     \t3.804 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17932020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17942020.fasta out=dormfiles/USAWI-UW-17942020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17942020.fasta, out=dormfiles/USAWI-UW-17942020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.601 seconds.\n",
      "Analyzed Index:   \t2.564 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.065 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.353 seconds.\n",
      "Reads/sec:       \t170.12\n",
      "kBases/sec:      \t84.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.834 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17942020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17952020.fasta out=dormfiles/USAWI-UW-17952020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17952020.fasta, out=dormfiles/USAWI-UW-17952020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.481 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.305 seconds.\n",
      "Reads/sec:       \t196.50\n",
      "kBases/sec:      \t97.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.703 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17952020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17962020.fasta out=dormfiles/USAWI-UW-17962020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17962020.fasta, out=dormfiles/USAWI-UW-17962020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.587 seconds.\n",
      "Analyzed Index:   \t2.511 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.441 seconds.\n",
      "Reads/sec:       \t136.16\n",
      "kBases/sec:      \t67.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.851 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17962020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17972020.fasta out=dormfiles/USAWI-UW-17972020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17972020.fasta, out=dormfiles/USAWI-UW-17972020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.678 seconds.\n",
      "Analyzed Index:   \t2.548 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.480 seconds.\n",
      "Reads/sec:       \t125.07\n",
      "kBases/sec:      \t62.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.075 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17972020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17982020.fasta out=dormfiles/USAWI-UW-17982020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17982020.fasta, out=dormfiles/USAWI-UW-17982020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.593 seconds.\n",
      "Analyzed Index:   \t2.508 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.341 seconds.\n",
      "Reads/sec:       \t175.99\n",
      "kBases/sec:      \t87.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.747 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17982020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-17992020.fasta out=dormfiles/USAWI-UW-17992020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-17992020.fasta, out=dormfiles/USAWI-UW-17992020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.649 seconds.\n",
      "Analyzed Index:   \t2.733 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.379 seconds.\n",
      "Reads/sec:       \t158.37\n",
      "kBases/sec:      \t78.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.555 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-17992020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18002020.fasta out=dormfiles/USAWI-UW-18002020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18002020.fasta, out=dormfiles/USAWI-UW-18002020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.215 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.772 seconds.\n",
      "Analyzed Index:   \t2.810 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.359 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.538 seconds.\n",
      "Reads/sec:       \t111.59\n",
      "kBases/sec:      \t55.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0672% \t          20\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0672% \t          20\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.923 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18002020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "20 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18012020.fasta out=dormfiles/USAWI-UW-18012020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18012020.fasta, out=dormfiles/USAWI-UW-18012020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.604 seconds.\n",
      "Analyzed Index:   \t2.586 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.439 seconds.\n",
      "Reads/sec:       \t136.70\n",
      "kBases/sec:      \t67.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.969 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18012020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18022020.fasta out=dormfiles/USAWI-UW-18022020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18022020.fasta, out=dormfiles/USAWI-UW-18022020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.612 seconds.\n",
      "Analyzed Index:   \t2.473 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.303 seconds.\n",
      "Reads/sec:       \t198.05\n",
      "kBases/sec:      \t98.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.695 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18022020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18032020.fasta out=dormfiles/USAWI-UW-18032020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18032020.fasta, out=dormfiles/USAWI-UW-18032020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.645 seconds.\n",
      "Analyzed Index:   \t2.399 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.280 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.370 seconds.\n",
      "Reads/sec:       \t162.02\n",
      "kBases/sec:      \t80.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.925 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18032020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18042020.fasta out=dormfiles/USAWI-UW-18042020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18042020.fasta, out=dormfiles/USAWI-UW-18042020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.619 seconds.\n",
      "Analyzed Index:   \t2.535 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.406 seconds.\n",
      "Reads/sec:       \t147.62\n",
      "kBases/sec:      \t73.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.874 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18042020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18052020.fasta out=dormfiles/USAWI-UW-18052020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18052020.fasta, out=dormfiles/USAWI-UW-18052020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.027 seconds.\n",
      "Analyzed Index:   \t2.497 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.314 seconds.\n",
      "Reads/sec:       \t191.29\n",
      "kBases/sec:      \t94.95\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.175 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18052020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18062020.fasta out=dormfiles/USAWI-UW-18062020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18062020.fasta, out=dormfiles/USAWI-UW-18062020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.748 seconds.\n",
      "Analyzed Index:   \t2.583 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.064 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.278 seconds.\n",
      "Reads/sec:       \t215.94\n",
      "kBases/sec:      \t107.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.930 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18062020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18072020.fasta out=dormfiles/USAWI-UW-18072020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18072020.fasta, out=dormfiles/USAWI-UW-18072020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.611 seconds.\n",
      "Analyzed Index:   \t2.625 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.293 seconds.\n",
      "Reads/sec:       \t204.89\n",
      "kBases/sec:      \t101.70\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9454% \t       29266\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0546% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0546% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.860 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18072020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18082020.fasta out=dormfiles/USAWI-UW-18082020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18082020.fasta, out=dormfiles/USAWI-UW-18082020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.668 seconds.\n",
      "Analyzed Index:   \t2.940 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.458 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.417 seconds.\n",
      "Reads/sec:       \t143.81\n",
      "kBases/sec:      \t71.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.753 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18082020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18092020.fasta out=dormfiles/USAWI-UW-18092020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18092020.fasta, out=dormfiles/USAWI-UW-18092020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.509 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t213.10\n",
      "kBases/sec:      \t105.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.705 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18092020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18102020.fasta out=dormfiles/USAWI-UW-18102020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18102020.fasta, out=dormfiles/USAWI-UW-18102020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.523 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.392 seconds.\n",
      "Reads/sec:       \t153.24\n",
      "kBases/sec:      \t76.07\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.806 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18102020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18112020.fasta out=dormfiles/USAWI-UW-18112020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18112020.fasta, out=dormfiles/USAWI-UW-18112020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.563 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.286 seconds.\n",
      "Reads/sec:       \t209.99\n",
      "kBases/sec:      \t104.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.740 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18112020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18122020.fasta out=dormfiles/USAWI-UW-18122020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18122020.fasta, out=dormfiles/USAWI-UW-18122020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.749 seconds.\n",
      "Analyzed Index:   \t2.509 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.248 seconds.\n",
      "Reads/sec:       \t242.01\n",
      "kBases/sec:      \t120.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.809 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18122020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18132020.fasta out=dormfiles/USAWI-UW-18132020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18132020.fasta, out=dormfiles/USAWI-UW-18132020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.587 seconds.\n",
      "Analyzed Index:   \t2.588 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.361 seconds.\n",
      "Reads/sec:       \t166.04\n",
      "kBases/sec:      \t82.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.845 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18132020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18142020.fasta out=dormfiles/USAWI-UW-18142020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18142020.fasta, out=dormfiles/USAWI-UW-18142020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.758 seconds.\n",
      "Analyzed Index:   \t2.678 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.150 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.376 seconds.\n",
      "Reads/sec:       \t159.69\n",
      "kBases/sec:      \t79.26\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 96.6667% \t       58 \t 96.6423% \t       28782\n",
      "unambiguous:     \t 96.6667% \t       58 \t 96.6423% \t       28782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6789% \t         500\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1870% \t       28548\n",
      "Error Rate:      \t 22.0339% \t       13 \t  0.0486% \t          14\n",
      "Sub Rate:        \t 22.0339% \t       13 \t  0.0486% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0847% \t        3 \t  0.7644% \t         220\n",
      "\n",
      "Total time:     \t4.197 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18142020.fasta.pileup\n",
      "28782 bases in pileup file\n",
      "28782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18152020.fasta out=dormfiles/USAWI-UW-18152020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18152020.fasta, out=dormfiles/USAWI-UW-18152020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.640 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.135 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.261 seconds.\n",
      "Reads/sec:       \t229.71\n",
      "kBases/sec:      \t114.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.874 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18152020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18162020.fasta out=dormfiles/USAWI-UW-18162020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18162020.fasta, out=dormfiles/USAWI-UW-18162020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.748 seconds.\n",
      "Analyzed Index:   \t2.598 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.363 seconds.\n",
      "Reads/sec:       \t165.23\n",
      "kBases/sec:      \t82.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2176% \t       29549\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.211 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18162020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18172020.fasta out=dormfiles/USAWI-UW-18172020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18172020.fasta, out=dormfiles/USAWI-UW-18172020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.602 seconds.\n",
      "Analyzed Index:   \t2.583 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.160 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.274 seconds.\n",
      "Reads/sec:       \t218.94\n",
      "kBases/sec:      \t108.67\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.861 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18172020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18182020.fasta out=dormfiles/USAWI-UW-18182020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18182020.fasta, out=dormfiles/USAWI-UW-18182020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.602 seconds.\n",
      "Analyzed Index:   \t2.616 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.289 seconds.\n",
      "Reads/sec:       \t207.78\n",
      "kBases/sec:      \t103.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.906 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18182020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18192020.fasta out=dormfiles/USAWI-UW-18192020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18192020.fasta, out=dormfiles/USAWI-UW-18192020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.604 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.213 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.257 seconds.\n",
      "Reads/sec:       \t233.12\n",
      "kBases/sec:      \t115.71\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.912 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18192020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18202020.fasta out=dormfiles/USAWI-UW-18202020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18202020.fasta, out=dormfiles/USAWI-UW-18202020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.616 seconds.\n",
      "Analyzed Index:   \t2.570 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.145 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.557 seconds.\n",
      "Reads/sec:       \t107.68\n",
      "kBases/sec:      \t53.45\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "unambiguous:     \t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  3.3333% \t        2 \t  3.3577% \t        1000\n",
      "\n",
      "perfect best site:\t 61.6667% \t       37 \t 61.3861% \t       18282\n",
      "semiperfect site:\t 61.6667% \t       37 \t 61.3861% \t       18282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 97.4647% \t       26103\n",
      "Error Rate:      \t 12.0690% \t        7 \t  0.0261% \t           7\n",
      "Sub Rate:        \t 12.0690% \t        7 \t  0.0261% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 17.2414% \t       10 \t  2.5091% \t         672\n",
      "\n",
      "Total time:     \t4.317 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18202020.fasta.pileup\n",
      "26782 bases in pileup file\n",
      "26782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18212020.fasta out=dormfiles/USAWI-UW-18212020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18212020.fasta, out=dormfiles/USAWI-UW-18212020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.581 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.231 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.261 seconds.\n",
      "Reads/sec:       \t229.49\n",
      "kBases/sec:      \t113.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.909 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18212020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18222020.fasta out=dormfiles/USAWI-UW-18222020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18222020.fasta, out=dormfiles/USAWI-UW-18222020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.614 seconds.\n",
      "Analyzed Index:   \t2.577 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.274 seconds.\n",
      "Reads/sec:       \t219.19\n",
      "kBases/sec:      \t108.80\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6789% \t         500\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29047\n",
      "Error Rate:      \t 22.0339% \t       13 \t  0.0478% \t          14\n",
      "Sub Rate:        \t 22.0339% \t       13 \t  0.0478% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0847% \t        3 \t  0.7547% \t         221\n",
      "\n",
      "Total time:     \t3.781 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18222020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18232020.fasta out=dormfiles/USAWI-UW-18232020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18232020.fasta, out=dormfiles/USAWI-UW-18232020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.587 seconds.\n",
      "Analyzed Index:   \t2.552 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.50\n",
      "kBases/sec:      \t106.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2009% \t       29544\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.728 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18232020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18242020.fasta out=dormfiles/USAWI-UW-18242020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18242020.fasta, out=dormfiles/USAWI-UW-18242020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.588 seconds.\n",
      "Analyzed Index:   \t2.545 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.259 seconds.\n",
      "Reads/sec:       \t231.73\n",
      "kBases/sec:      \t115.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.829 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18242020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18252020.fasta out=dormfiles/USAWI-UW-18252020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18252020.fasta, out=dormfiles/USAWI-UW-18252020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.614 seconds.\n",
      "Analyzed Index:   \t2.552 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.318 seconds.\n",
      "Reads/sec:       \t188.62\n",
      "kBases/sec:      \t93.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.796 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18252020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18262020.fasta out=dormfiles/USAWI-UW-18262020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18262020.fasta, out=dormfiles/USAWI-UW-18262020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.594 seconds.\n",
      "Analyzed Index:   \t2.517 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.295 seconds.\n",
      "Reads/sec:       \t203.49\n",
      "kBases/sec:      \t101.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.714 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18262020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18272020.fasta out=dormfiles/USAWI-UW-18272020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18272020.fasta, out=dormfiles/USAWI-UW-18272020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.615 seconds.\n",
      "Analyzed Index:   \t2.508 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.18\n",
      "kBases/sec:      \t106.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.712 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18272020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18282020.fasta out=dormfiles/USAWI-UW-18282020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18282020.fasta, out=dormfiles/USAWI-UW-18282020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.494 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.287 seconds.\n",
      "Reads/sec:       \t208.85\n",
      "kBases/sec:      \t103.67\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.692 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18282020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18292020.fasta out=dormfiles/USAWI-UW-18292020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18292020.fasta, out=dormfiles/USAWI-UW-18292020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.595 seconds.\n",
      "Analyzed Index:   \t2.541 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.273 seconds.\n",
      "Reads/sec:       \t220.09\n",
      "kBases/sec:      \t109.24\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.717 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18292020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18302020.fasta out=dormfiles/USAWI-UW-18302020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18302020.fasta, out=dormfiles/USAWI-UW-18302020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.604 seconds.\n",
      "Analyzed Index:   \t2.515 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t226.86\n",
      "kBases/sec:      \t112.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "semiperfect site:\t 76.6667% \t       46 \t 77.2279% \t       23000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.699 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18302020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18312020.fasta out=dormfiles/USAWI-UW-18312020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18312020.fasta, out=dormfiles/USAWI-UW-18312020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.540 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.249 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.332 seconds.\n",
      "Reads/sec:       \t180.56\n",
      "kBases/sec:      \t89.63\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2277% \t       29552\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.961 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18312020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18322020.fasta out=dormfiles/USAWI-UW-18322020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18322020.fasta, out=dormfiles/USAWI-UW-18322020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.608 seconds.\n",
      "Analyzed Index:   \t2.608 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.317 seconds.\n",
      "Reads/sec:       \t189.07\n",
      "kBases/sec:      \t93.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9194% \t       29758\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.862 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18322020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18332020.fasta out=dormfiles/USAWI-UW-18332020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18332020.fasta, out=dormfiles/USAWI-UW-18332020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.961 seconds.\n",
      "Analyzed Index:   \t2.668 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.124 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.409 seconds.\n",
      "Reads/sec:       \t146.81\n",
      "kBases/sec:      \t72.87\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.522 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18332020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18342020.fasta out=dormfiles/USAWI-UW-18342020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18342020.fasta, out=dormfiles/USAWI-UW-18342020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.669 seconds.\n",
      "Analyzed Index:   \t2.650 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.266 seconds.\n",
      "Reads/sec:       \t225.46\n",
      "kBases/sec:      \t111.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1872% \t       29044\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0546% \t          16\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0546% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7581% \t         222\n",
      "\n",
      "Total time:     \t3.906 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18342020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18352020.fasta out=dormfiles/USAWI-UW-18352020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18352020.fasta, out=dormfiles/USAWI-UW-18352020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.622 seconds.\n",
      "Analyzed Index:   \t2.679 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.536 seconds.\n",
      "Reads/sec:       \t111.91\n",
      "kBases/sec:      \t55.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.193 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18352020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18362020.fasta out=dormfiles/USAWI-UW-18362020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18362020.fasta, out=dormfiles/USAWI-UW-18362020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.649 seconds.\n",
      "Analyzed Index:   \t2.645 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.311 seconds.\n",
      "Reads/sec:       \t193.22\n",
      "kBases/sec:      \t95.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.042 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18362020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18372020.fasta out=dormfiles/USAWI-UW-18372020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18372020.fasta, out=dormfiles/USAWI-UW-18372020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.616 seconds.\n",
      "Analyzed Index:   \t2.626 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.066 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.445 seconds.\n",
      "Reads/sec:       \t134.84\n",
      "kBases/sec:      \t66.93\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.1015% \t       20282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9295% \t       29761\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.002 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18372020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18382020.fasta out=dormfiles/USAWI-UW-18382020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18382020.fasta, out=dormfiles/USAWI-UW-18382020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t2.883 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.308 seconds.\n",
      "Reads/sec:       \t194.77\n",
      "kBases/sec:      \t96.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1841% \t       29539\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.269 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18382020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18392020.fasta out=dormfiles/USAWI-UW-18392020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18392020.fasta, out=dormfiles/USAWI-UW-18392020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.668 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.531 seconds.\n",
      "Reads/sec:       \t112.94\n",
      "kBases/sec:      \t56.06\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1874% \t       29540\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.109 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18392020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18402020.fasta out=dormfiles/USAWI-UW-18402020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18402020.fasta, out=dormfiles/USAWI-UW-18402020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.672 seconds.\n",
      "Analyzed Index:   \t2.859 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.096 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.321 seconds.\n",
      "Reads/sec:       \t186.65\n",
      "kBases/sec:      \t92.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.279 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18402020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18412020.fasta out=dormfiles/USAWI-UW-18412020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18412020.fasta, out=dormfiles/USAWI-UW-18412020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.604 seconds.\n",
      "Analyzed Index:   \t2.845 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.156 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.539 seconds.\n",
      "Reads/sec:       \t111.26\n",
      "kBases/sec:      \t55.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.403 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18412020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18422020.fasta out=dormfiles/USAWI-UW-18422020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18422020.fasta, out=dormfiles/USAWI-UW-18422020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.617 seconds.\n",
      "Analyzed Index:   \t2.525 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.388 seconds.\n",
      "Reads/sec:       \t154.58\n",
      "kBases/sec:      \t76.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.952 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18422020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18432020.fasta out=dormfiles/USAWI-UW-18432020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18432020.fasta, out=dormfiles/USAWI-UW-18432020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.623 seconds.\n",
      "Analyzed Index:   \t2.617 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.201 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.366 seconds.\n",
      "Reads/sec:       \t164.05\n",
      "kBases/sec:      \t81.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9093% \t       29755\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.052 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18432020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18442020.fasta out=dormfiles/USAWI-UW-18442020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18442020.fasta, out=dormfiles/USAWI-UW-18442020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.608 seconds.\n",
      "Analyzed Index:   \t2.576 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.734 seconds.\n",
      "Reads/sec:       \t81.70\n",
      "kBases/sec:      \t40.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9463% \t       29766\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.233 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18442020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18452020.fasta out=dormfiles/USAWI-UW-18452020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18452020.fasta, out=dormfiles/USAWI-UW-18452020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.632 seconds.\n",
      "Analyzed Index:   \t2.682 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.155 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.371 seconds.\n",
      "Reads/sec:       \t161.64\n",
      "kBases/sec:      \t80.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 66.4227% \t       19782\n",
      "semiperfect site:\t 66.6667% \t       40 \t 66.4227% \t       19782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9261% \t       29760\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.096 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18452020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18462020.fasta out=dormfiles/USAWI-UW-18462020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18462020.fasta, out=dormfiles/USAWI-UW-18462020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.608 seconds.\n",
      "Analyzed Index:   \t2.733 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.297 seconds.\n",
      "Reads/sec:       \t202.08\n",
      "kBases/sec:      \t100.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.968 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18462020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18472020.fasta out=dormfiles/USAWI-UW-18472020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18472020.fasta, out=dormfiles/USAWI-UW-18472020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.663 seconds.\n",
      "Analyzed Index:   \t3.065 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.363 seconds.\n",
      "Reads/sec:       \t165.43\n",
      "kBases/sec:      \t82.11\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.8926% \t       29750\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0504% \t          15\n",
      "\n",
      "Total time:     \t4.417 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18472020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18482020.fasta out=dormfiles/USAWI-UW-18482020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18482020.fasta, out=dormfiles/USAWI-UW-18482020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.701 seconds.\n",
      "Analyzed Index:   \t3.057 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.388 seconds.\n",
      "Reads/sec:       \t154.45\n",
      "kBases/sec:      \t76.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.480 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18482020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18492020.fasta out=dormfiles/USAWI-UW-18492020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18492020.fasta, out=dormfiles/USAWI-UW-18492020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.682 seconds.\n",
      "Analyzed Index:   \t2.788 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.351 seconds.\n",
      "Reads/sec:       \t170.83\n",
      "kBases/sec:      \t84.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.166 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18492020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18502020.fasta out=dormfiles/USAWI-UW-18502020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18502020.fasta, out=dormfiles/USAWI-UW-18502020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.609 seconds.\n",
      "Analyzed Index:   \t2.833 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.349 seconds.\n",
      "Reads/sec:       \t171.86\n",
      "kBases/sec:      \t85.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.122 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18502020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18512020.fasta out=dormfiles/USAWI-UW-18512020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18512020.fasta, out=dormfiles/USAWI-UW-18512020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.633 seconds.\n",
      "Analyzed Index:   \t2.775 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.119 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.390 seconds.\n",
      "Reads/sec:       \t153.86\n",
      "kBases/sec:      \t76.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2176% \t       29549\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.159 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18512020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18522020.fasta out=dormfiles/USAWI-UW-18522020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18522020.fasta, out=dormfiles/USAWI-UW-18522020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.647 seconds.\n",
      "Analyzed Index:   \t2.688 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.336 seconds.\n",
      "Reads/sec:       \t178.51\n",
      "kBases/sec:      \t88.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.980 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18522020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18532020.fasta out=dormfiles/USAWI-UW-18532020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18532020.fasta, out=dormfiles/USAWI-UW-18532020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t3.043 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.255 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.979 seconds.\n",
      "Reads/sec:       \t61.27\n",
      "kBases/sec:      \t30.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t5.234 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18532020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18542020.fasta out=dormfiles/USAWI-UW-18542020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18542020.fasta, out=dormfiles/USAWI-UW-18542020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.949 seconds.\n",
      "Analyzed Index:   \t3.094 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.133 seconds.\n",
      "Cleared Memory:    \t0.125 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.422 seconds.\n",
      "Reads/sec:       \t142.02\n",
      "kBases/sec:      \t70.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.1381% \t       21782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9429% \t       29765\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.930 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18542020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18552020.fasta out=dormfiles/USAWI-UW-18552020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18552020.fasta, out=dormfiles/USAWI-UW-18552020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.601 seconds.\n",
      "Analyzed Index:   \t2.988 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.115 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.454 seconds.\n",
      "Reads/sec:       \t132.14\n",
      "kBases/sec:      \t65.59\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "semiperfect site:\t 65.0000% \t       39 \t 65.4758% \t       19500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 10.0000% \t        6 \t  0.7488% \t         223\n",
      "\n",
      "Total time:     \t4.521 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18552020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18562020.fasta out=dormfiles/USAWI-UW-18562020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18562020.fasta, out=dormfiles/USAWI-UW-18562020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.807 seconds.\n",
      "Analyzed Index:   \t3.336 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.546 seconds.\n",
      "Reads/sec:       \t109.86\n",
      "kBases/sec:      \t54.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t5.338 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18562020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18572020.fasta out=dormfiles/USAWI-UW-18572020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18572020.fasta, out=dormfiles/USAWI-UW-18572020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.600 seconds.\n",
      "Analyzed Index:   \t3.190 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.439 seconds.\n",
      "Reads/sec:       \t136.56\n",
      "kBases/sec:      \t67.78\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.4959% \t       22782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2210% \t       29550\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.542 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18572020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18582020.fasta out=dormfiles/USAWI-UW-18582020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18582020.fasta, out=dormfiles/USAWI-UW-18582020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.660 seconds.\n",
      "Analyzed Index:   \t2.759 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.348 seconds.\n",
      "Reads/sec:       \t172.20\n",
      "kBases/sec:      \t85.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0504% \t          15\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t4.135 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18582020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "15 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18592020.fasta out=dormfiles/USAWI-UW-18592020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18592020.fasta, out=dormfiles/USAWI-UW-18592020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.681 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.067 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.375 seconds.\n",
      "Reads/sec:       \t160.11\n",
      "kBases/sec:      \t79.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9127% \t       29756\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0336% \t          10\n",
      "\n",
      "Total time:     \t3.968 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18592020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18602020.fasta out=dormfiles/USAWI-UW-18602020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18602020.fasta, out=dormfiles/USAWI-UW-18602020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.610 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.230 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.399 seconds.\n",
      "Reads/sec:       \t150.20\n",
      "kBases/sec:      \t74.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1874% \t       29540\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.098 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18602020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18612020.fasta out=dormfiles/USAWI-UW-18612020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18612020.fasta, out=dormfiles/USAWI-UW-18612020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.847 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.656 seconds.\n",
      "Reads/sec:       \t91.45\n",
      "kBases/sec:      \t45.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.429 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18612020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18622020.fasta out=dormfiles/USAWI-UW-18622020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18622020.fasta, out=dormfiles/USAWI-UW-18622020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.629 seconds.\n",
      "Analyzed Index:   \t3.147 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.274 seconds.\n",
      "Reads/sec:       \t218.70\n",
      "kBases/sec:      \t108.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.373 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18622020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18632020.fasta out=dormfiles/USAWI-UW-18632020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18632020.fasta, out=dormfiles/USAWI-UW-18632020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.597 seconds.\n",
      "Analyzed Index:   \t2.673 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.296 seconds.\n",
      "Reads/sec:       \t202.83\n",
      "kBases/sec:      \t100.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9396% \t       29764\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.891 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18632020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18642020.fasta out=dormfiles/USAWI-UW-18642020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18642020.fasta, out=dormfiles/USAWI-UW-18642020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.613 seconds.\n",
      "Analyzed Index:   \t2.719 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.366 seconds.\n",
      "Reads/sec:       \t163.71\n",
      "kBases/sec:      \t81.26\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 69.7804% \t       20782\n",
      "semiperfect site:\t 70.0000% \t       42 \t 69.7804% \t       20782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9328% \t       29762\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.026 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18642020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18652020.fasta out=dormfiles/USAWI-UW-18652020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18652020.fasta, out=dormfiles/USAWI-UW-18652020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.615 seconds.\n",
      "Analyzed Index:   \t2.624 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.266 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.455 seconds.\n",
      "Reads/sec:       \t131.75\n",
      "kBases/sec:      \t65.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "semiperfect site:\t 66.6667% \t       40 \t 67.1547% \t       20000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1941% \t       29542\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t4.204 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18652020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18662020.fasta out=dormfiles/USAWI-UW-18662020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18662020.fasta, out=dormfiles/USAWI-UW-18662020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.622 seconds.\n",
      "Analyzed Index:   \t2.648 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.385 seconds.\n",
      "Reads/sec:       \t155.81\n",
      "kBases/sec:      \t77.34\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7387% \t         220\n",
      "\n",
      "Total time:     \t3.979 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18662020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18672020.fasta out=dormfiles/USAWI-UW-18672020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18672020.fasta, out=dormfiles/USAWI-UW-18672020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.599 seconds.\n",
      "Analyzed Index:   \t2.730 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.101 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.500 seconds.\n",
      "Reads/sec:       \t120.10\n",
      "kBases/sec:      \t59.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2042% \t       29545\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.175 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18672020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18682020.fasta out=dormfiles/USAWI-UW-18682020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18682020.fasta, out=dormfiles/USAWI-UW-18682020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.656 seconds.\n",
      "Analyzed Index:   \t2.568 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.360 seconds.\n",
      "Reads/sec:       \t166.85\n",
      "kBases/sec:      \t82.82\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "semiperfect site:\t 71.6667% \t       43 \t 72.1913% \t       21500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Sub Rate:        \t 25.0000% \t       15 \t  0.0571% \t          17\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.123 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18682020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "17 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18692020.fasta out=dormfiles/USAWI-UW-18692020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18692020.fasta, out=dormfiles/USAWI-UW-18692020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.590 seconds.\n",
      "Analyzed Index:   \t2.708 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.352 seconds.\n",
      "Reads/sec:       \t170.26\n",
      "kBases/sec:      \t84.51\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 33.3333% \t       20 \t  0.0739% \t          22\n",
      "Sub Rate:        \t 33.3333% \t       20 \t  0.0739% \t          22\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.981 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18692020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "22 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18702020.fasta out=dormfiles/USAWI-UW-18702020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18702020.fasta, out=dormfiles/USAWI-UW-18702020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.575 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.23\n",
      "kBases/sec:      \t107.83\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.7969% \t       19000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1774% \t       29537\n",
      "Error Rate:      \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Sub Rate:        \t 31.6667% \t       19 \t  0.0806% \t          24\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  5.0000% \t        3 \t  0.7421% \t         221\n",
      "\n",
      "Total time:     \t3.769 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18702020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "24 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-18712020.fasta out=dormfiles/USAWI-UW-18712020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-18712020.fasta, out=dormfiles/USAWI-UW-18712020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.591 seconds.\n",
      "Analyzed Index:   \t3.059 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.503 seconds.\n",
      "Reads/sec:       \t119.28\n",
      "kBases/sec:      \t59.20\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "semiperfect site:\t 73.3333% \t       44 \t 73.8701% \t       22000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2109% \t       29547\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.457 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-18712020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-23232020.fasta out=dormfiles/USAWI-UW-23232020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-23232020.fasta, out=dormfiles/USAWI-UW-23232020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.622 seconds.\n",
      "Analyzed Index:   \t2.870 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.125 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.337 seconds.\n",
      "Reads/sec:       \t177.94\n",
      "kBases/sec:      \t88.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.647 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-23232020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-24842020.fasta out=dormfiles/USAWI-UW-24842020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-24842020.fasta, out=dormfiles/USAWI-UW-24842020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.615 seconds.\n",
      "Analyzed Index:   \t3.248 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.154 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.369 seconds.\n",
      "Reads/sec:       \t162.69\n",
      "kBases/sec:      \t80.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "semiperfect site:\t 65.0000% \t       39 \t 64.7438% \t       19282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9228% \t       29759\n",
      "Error Rate:      \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.626 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-24842020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-24852020.fasta out=dormfiles/USAWI-UW-24852020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-24852020.fasta, out=dormfiles/USAWI-UW-24852020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.045 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.854 seconds.\n",
      "Analyzed Index:   \t2.853 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.406 seconds.\n",
      "Reads/sec:       \t147.72\n",
      "kBases/sec:      \t73.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "semiperfect site:\t 63.3333% \t       38 \t 63.0649% \t       18782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9161% \t       29757\n",
      "Error Rate:      \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 35.0000% \t       21 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.541 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-24852020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-UW-24862020.fasta out=dormfiles/USAWI-UW-24862020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-UW-24862020.fasta, out=dormfiles/USAWI-UW-24862020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.619 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.068 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29781 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t211.37\n",
      "kBases/sec:      \t104.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 91.6667% \t       55 \t 91.6054% \t       27281\n",
      "unambiguous:     \t 91.6667% \t       55 \t 91.6054% \t       27281\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  5.0000% \t        3 \t  5.0368% \t        1500\n",
      "\n",
      "perfect best site:\t 61.6667% \t       37 \t 61.3848% \t       18281\n",
      "semiperfect site:\t 61.6667% \t       37 \t 61.3848% \t       18281\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 98.7794% \t       26948\n",
      "Error Rate:      \t 21.0526% \t       12 \t  0.0440% \t          12\n",
      "Sub Rate:        \t 21.0526% \t       12 \t  0.0440% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 14.0351% \t        8 \t  1.1766% \t         321\n",
      "\n",
      "Total time:     \t3.830 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-UW-24862020.fasta.pileup\n",
      "27281 bases in pileup file\n",
      "27281 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007242020.fasta out=dormfiles/USAWI-WSLH-2007242020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007242020.fasta, out=dormfiles/USAWI-WSLH-2007242020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.685 seconds.\n",
      "Analyzed Index:   \t2.688 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29779 bases)\n",
      "\n",
      "Mapping:          \t0.416 seconds.\n",
      "Reads/sec:       \t144.19\n",
      "kBases/sec:      \t71.56\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29779\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29779\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 79.8516% \t       23779\n",
      "semiperfect site:\t 80.0000% \t       48 \t 79.8516% \t       23779\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0369% \t          11\n",
      "Del Rate:        \t  1.6667% \t        1 \t  0.0101% \t           3\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.121 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007242020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007262020.fasta out=dormfiles/USAWI-WSLH-2007262020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007262020.fasta, out=dormfiles/USAWI-WSLH-2007262020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.636 seconds.\n",
      "Analyzed Index:   \t3.013 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.395 seconds.\n",
      "Reads/sec:       \t151.86\n",
      "kBases/sec:      \t75.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t4.401 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007262020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007272020.fasta out=dormfiles/USAWI-WSLH-2007272020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007272020.fasta, out=dormfiles/USAWI-WSLH-2007272020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.620 seconds.\n",
      "Analyzed Index:   \t2.610 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.073 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.255 seconds.\n",
      "Reads/sec:       \t235.29\n",
      "kBases/sec:      \t116.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 79.8536% \t       23782\n",
      "semiperfect site:\t 80.0000% \t       48 \t 79.8536% \t       23782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.798 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007272020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007282020.fasta out=dormfiles/USAWI-WSLH-2007282020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007282020.fasta, out=dormfiles/USAWI-WSLH-2007282020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.608 seconds.\n",
      "Analyzed Index:   \t2.830 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t227.47\n",
      "kBases/sec:      \t112.91\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.036 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007282020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007292020.fasta out=dormfiles/USAWI-WSLH-2007292020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007292020.fasta, out=dormfiles/USAWI-WSLH-2007292020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t2.512 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.257 seconds.\n",
      "Reads/sec:       \t233.65\n",
      "kBases/sec:      \t115.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.795 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007292020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007302020.fasta out=dormfiles/USAWI-WSLH-2007302020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007302020.fasta, out=dormfiles/USAWI-WSLH-2007302020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.596 seconds.\n",
      "Analyzed Index:   \t2.598 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t238.37\n",
      "kBases/sec:      \t118.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.769 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007302020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007312020.fasta out=dormfiles/USAWI-WSLH-2007312020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007312020.fasta, out=dormfiles/USAWI-WSLH-2007312020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.608 seconds.\n",
      "Analyzed Index:   \t2.835 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.408 seconds.\n",
      "Reads/sec:       \t146.98\n",
      "kBases/sec:      \t72.96\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.192 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007312020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007322020.fasta out=dormfiles/USAWI-WSLH-2007322020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007322020.fasta, out=dormfiles/USAWI-WSLH-2007322020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.647 seconds.\n",
      "Analyzed Index:   \t2.985 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.287 seconds.\n",
      "Reads/sec:       \t209.37\n",
      "kBases/sec:      \t103.92\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "semiperfect site:\t 78.3333% \t       47 \t 78.1747% \t       23282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2244% \t       29551\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.270 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007322020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007332020.fasta out=dormfiles/USAWI-WSLH-2007332020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007332020.fasta, out=dormfiles/USAWI-WSLH-2007332020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.624 seconds.\n",
      "Analyzed Index:   \t2.674 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.071 seconds.\n",
      "Cleared Memory:    \t0.105 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.285 seconds.\n",
      "Reads/sec:       \t210.71\n",
      "kBases/sec:      \t104.59\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "semiperfect site:\t 75.0000% \t       45 \t 74.8170% \t       22282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2176% \t       29549\n",
      "Error Rate:      \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Sub Rate:        \t 21.6667% \t       13 \t  0.0470% \t          14\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7353% \t         219\n",
      "\n",
      "Total time:     \t3.903 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007332020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "14 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007342020.fasta out=dormfiles/USAWI-WSLH-2007342020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007342020.fasta, out=dormfiles/USAWI-WSLH-2007342020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.779 seconds.\n",
      "Analyzed Index:   \t2.554 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.267 seconds.\n",
      "Reads/sec:       \t224.93\n",
      "kBases/sec:      \t111.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "semiperfect site:\t 71.6667% \t       43 \t 71.4593% \t       21282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 26.6667% \t       16 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.911 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007342020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007352020.fasta out=dormfiles/USAWI-WSLH-2007352020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007352020.fasta, out=dormfiles/USAWI-WSLH-2007352020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.018 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.588 seconds.\n",
      "Analyzed Index:   \t2.572 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.072 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.330 seconds.\n",
      "Reads/sec:       \t181.98\n",
      "kBases/sec:      \t90.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.800 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007352020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007362020.fasta out=dormfiles/USAWI-WSLH-2007362020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007362020.fasta, out=dormfiles/USAWI-WSLH-2007362020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.607 seconds.\n",
      "Analyzed Index:   \t2.588 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.444 seconds.\n",
      "Reads/sec:       \t135.26\n",
      "kBases/sec:      \t67.14\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "semiperfect site:\t 70.0000% \t       42 \t 70.5124% \t       21000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1975% \t       29543\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0705% \t          21\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.953 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007362020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "21 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007372020.fasta out=dormfiles/USAWI-WSLH-2007372020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007372020.fasta, out=dormfiles/USAWI-WSLH-2007372020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.021 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.620 seconds.\n",
      "Analyzed Index:   \t2.623 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.237 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.340 seconds.\n",
      "Reads/sec:       \t176.49\n",
      "kBases/sec:      \t87.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 69.7804% \t       20782\n",
      "semiperfect site:\t 70.0000% \t       42 \t 69.7804% \t       20782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2076% \t       29546\n",
      "Error Rate:      \t 28.3333% \t       17 \t  0.0604% \t          18\n",
      "Sub Rate:        \t 28.3333% \t       17 \t  0.0604% \t          18\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.069 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007372020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "18 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007382020.fasta out=dormfiles/USAWI-WSLH-2007382020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007382020.fasta, out=dormfiles/USAWI-WSLH-2007382020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.635 seconds.\n",
      "Analyzed Index:   \t2.734 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.320 seconds.\n",
      "Reads/sec:       \t187.45\n",
      "kBases/sec:      \t93.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "semiperfect site:\t 68.3333% \t       41 \t 68.8335% \t       20500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1908% \t       29541\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0772% \t          23\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t4.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007382020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "23 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007392020.fasta out=dormfiles/USAWI-WSLH-2007392020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007392020.fasta, out=dormfiles/USAWI-WSLH-2007392020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.598 seconds.\n",
      "Analyzed Index:   \t2.603 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.069 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.253 seconds.\n",
      "Reads/sec:       \t237.45\n",
      "kBases/sec:      \t117.86\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.778 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007392020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007412020.fasta out=dormfiles/USAWI-WSLH-2007412020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007412020.fasta, out=dormfiles/USAWI-WSLH-2007412020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.020 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.603 seconds.\n",
      "Analyzed Index:   \t2.523 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.345 seconds.\n",
      "Reads/sec:       \t173.92\n",
      "kBases/sec:      \t86.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 66.6667% \t       40 \t 66.4227% \t       19782\n",
      "semiperfect site:\t 66.6667% \t       40 \t 66.4227% \t       19782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.1035% \t       29515\n",
      "Error Rate:      \t 30.0000% \t       18 \t  0.0638% \t          19\n",
      "Sub Rate:        \t 30.0000% \t       18 \t  0.0638% \t          19\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.8327% \t         248\n",
      "\n",
      "Total time:     \t3.807 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007412020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "19 SNPs predicted\n",
      "/Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0//calcmem.sh: line 75: [: -v: unary operator expected\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -ea -Xmx3200m -Xms3200m -cp /Users/gagemoreno/opt/anaconda3/opt/bbmap-38.84-0/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=dormfiles/USAWI-WSLH-2007422020.fasta out=dormfiles/USAWI-WSLH-2007422020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=dormfiles/USAWI-WSLH-2007422020.fasta, out=dormfiles/USAWI-WSLH-2007422020.fasta.bam, ref=ref.fa]\n",
      "Version 38.84\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.019 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.602 seconds.\n",
      "Analyzed Index:   \t2.635 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.170 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 8 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3, 4, 5, 6, 7\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t227.66\n",
      "kBases/sec:      \t113.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "semiperfect site:\t 75.0000% \t       45 \t 75.5490% \t       22500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2143% \t       29548\n",
      "Error Rate:      \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Sub Rate:        \t 23.3333% \t       14 \t  0.0537% \t          16\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.7320% \t         218\n",
      "\n",
      "Total time:     \t3.920 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from dormfiles/USAWI-WSLH-2007422020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "16 SNPs predicted\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# UW Madison Dorm samples, this takes ~10 mins to run \n",
    "\n",
    "for file in $(ls dormfiles/*.fasta)\n",
    "do \n",
    "    bbmap.sh in=$file out=$file.bam ref=ref.fa #map to Wuhan reference\n",
    "    samtools sort $file.bam -o $file.bam # sort the bam \n",
    "    samtools mpileup -f ref.fa $file.bam > $file.pileup # generate a pileup file\n",
    "    java -jar VarScan.jar pileup2snp $file.pileup --min-coverage 1 --min-reads2 1 > $file.vcf\n",
    "done\n",
    "\n",
    "##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "# move *.vcf files from dormfiles to data_derived (in consensus_SNVs)\n",
    "# Dorms\n",
    "\n",
    "root_path = \"dormfiles/\"\n",
    "dest_path = \"VCFs/\"\n",
    "\n",
    "for dirpath, dnames, fnames in os.walk(root_path):    \n",
    "    for f in fnames:\n",
    "        if f.endswith(\".vcf\"):\n",
    "            source_file_path =  os.path.join(dirpath, f)\n",
    "            dest_file_path   =  os.path.join(dest_path, f)\n",
    "            sh.copyfile(source_file_path, dest_file_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "# dormfiles cleanup \n",
    "\n",
    "dir_name = \"dormfiles/\"\n",
    "test = os.listdir(dir_name)\n",
    "\n",
    "for item in test:\n",
    "    if item.endswith(\".bam\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".pileup\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".vcf\"):\n",
    "        os.remove(os.path.join(dir_name, item))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['USAWI-UW-14182020.fasta', 'USAWI-UW-14192020.fasta', 'USAWI-UW-14202020.fasta', 'USAWI-UW-14212020.fasta', 'USAWI-UW-14222020.fasta', 'USAWI-UW-14232020.fasta', 'USAWI-UW-14242020.fasta', 'USAWI-UW-14252020.fasta', 'USAWI-UW-14262020.fasta', 'USAWI-UW-14272020.fasta', 'USAWI-UW-14282020.fasta', 'USAWI-UW-14292020.fasta', 'USAWI-UW-14302020.fasta', 'USAWI-UW-14312020.fasta', 'USAWI-UW-14322020.fasta', 'USAWI-UW-16302020.fasta', 'USAWI-UW-16312020.fasta', 'USAWI-UW-16322020.fasta', 'USAWI-UW-16332020.fasta', 'USAWI-UW-16342020.fasta', 'USAWI-UW-16352020.fasta', 'USAWI-UW-16362020.fasta', 'USAWI-UW-16372020.fasta', 'USAWI-UW-16382020.fasta', 'USAWI-UW-16392020.fasta', 'USAWI-UW-16402020.fasta', 'USAWI-UW-16412020.fasta', 'USAWI-UW-16422020.fasta', 'USAWI-UW-16432020.fasta', 'USAWI-UW-16442020.fasta', 'USAWI-UW-16452020.fasta', 'USAWI-UW-16462020.fasta', 'USAWI-UW-16472020.fasta', 'USAWI-UW-16482020.fasta', 'USAWI-UW-16492020.fasta', 'USAWI-UW-16502020.fasta', 'USAWI-UW-16512020.fasta', 'USAWI-UW-16522020.fasta', 'USAWI-UW-16532020.fasta', 'USAWI-UW-16542020.fasta', 'USAWI-UW-16552020.fasta', 'USAWI-UW-16562020.fasta', 'USAWI-UW-16572020.fasta', 'USAWI-UW-16582020.fasta', 'USAWI-UW-16592020.fasta', 'USAWI-UW-16602020.fasta', 'USAWI-UW-16612020.fasta', 'USAWI-UW-16622020.fasta', 'USAWI-UW-16632020.fasta', 'USAWI-UW-16642020.fasta', 'USAWI-UW-16652020.fasta', 'USAWI-UW-16662020.fasta', 'USAWI-UW-16672020.fasta', 'USAWI-UW-16682020.fasta', 'USAWI-UW-16692020.fasta', 'USAWI-UW-16702020.fasta', 'USAWI-UW-16712020.fasta', 'USAWI-UW-16722020.fasta', 'USAWI-UW-16732020.fasta', 'USAWI-UW-16742020.fasta', 'USAWI-UW-16752020.fasta', 'USAWI-UW-16762020.fasta', 'USAWI-UW-16772020.fasta', 'USAWI-UW-16782020.fasta', 'USAWI-UW-16792020.fasta', 'USAWI-UW-16802020.fasta', 'USAWI-UW-16812020.fasta', 'USAWI-UW-16822020.fasta', 'USAWI-UW-16832020.fasta', 'USAWI-UW-16842020.fasta', 'USAWI-UW-16852020.fasta', 'USAWI-UW-16862020.fasta', 'USAWI-UW-16872020.fasta', 'USAWI-UW-16882020.fasta', 'USAWI-UW-16892020.fasta', 'USAWI-UW-16902020.fasta', 'USAWI-UW-16912020.fasta', 'USAWI-UW-16922020.fasta', 'USAWI-UW-16932020.fasta', 'USAWI-UW-16942020.fasta', 'USAWI-UW-16952020.fasta', 'USAWI-UW-16962020.fasta', 'USAWI-UW-16972020.fasta', 'USAWI-UW-16982020.fasta', 'USAWI-UW-16992020.fasta', 'USAWI-UW-17002020.fasta', 'USAWI-UW-17012020.fasta', 'USAWI-UW-17022020.fasta', 'USAWI-UW-17032020.fasta', 'USAWI-UW-17042020.fasta', 'USAWI-UW-17052020.fasta', 'USAWI-UW-17062020.fasta', 'USAWI-UW-17072020.fasta', 'USAWI-UW-17082020.fasta', 'USAWI-UW-17092020.fasta', 'USAWI-UW-17102020.fasta', 'USAWI-UW-17112020.fasta', 'USAWI-UW-17122020.fasta', 'USAWI-UW-17132020.fasta', 'USAWI-UW-17142020.fasta', 'USAWI-UW-17152020.fasta', 'USAWI-UW-17162020.fasta', 'USAWI-UW-17172020.fasta', 'USAWI-UW-17182020.fasta', 'USAWI-UW-17192020.fasta', 'USAWI-UW-17202020.fasta', 'USAWI-UW-17212020.fasta', 'USAWI-UW-17222020.fasta', 'USAWI-UW-17232020.fasta', 'USAWI-UW-17242020.fasta', 'USAWI-UW-17252020.fasta', 'USAWI-UW-17262020.fasta', 'USAWI-UW-17272020.fasta', 'USAWI-UW-17282020.fasta', 'USAWI-UW-17292020.fasta', 'USAWI-UW-17302020.fasta', 'USAWI-UW-17312020.fasta', 'USAWI-UW-17322020.fasta', 'USAWI-UW-17332020.fasta', 'USAWI-UW-17342020.fasta', 'USAWI-UW-17352020.fasta', 'USAWI-UW-17362020.fasta', 'USAWI-UW-17372020.fasta', 'USAWI-UW-17382020.fasta', 'USAWI-UW-17392020.fasta', 'USAWI-UW-17402020.fasta', 'USAWI-UW-17412020.fasta', 'USAWI-UW-17422020.fasta', 'USAWI-UW-17432020.fasta', 'USAWI-UW-17442020.fasta', 'USAWI-UW-17452020.fasta', 'USAWI-UW-17462020.fasta', 'USAWI-UW-17472020.fasta', 'USAWI-UW-17482020.fasta', 'USAWI-UW-17492020.fasta', 'USAWI-UW-17502020.fasta', 'USAWI-UW-17512020.fasta', 'USAWI-UW-17522020.fasta', 'USAWI-UW-17532020.fasta', 'USAWI-UW-17542020.fasta', 'USAWI-UW-17552020.fasta', 'USAWI-UW-17562020.fasta', 'USAWI-UW-17572020.fasta', 'USAWI-UW-17582020.fasta', 'USAWI-UW-17592020.fasta', 'USAWI-UW-17602020.fasta', 'USAWI-UW-17612020.fasta', 'USAWI-UW-17622020.fasta', 'USAWI-UW-17632020.fasta', 'USAWI-UW-17642020.fasta', 'USAWI-UW-17652020.fasta', 'USAWI-UW-17662020.fasta', 'USAWI-UW-17672020.fasta', 'USAWI-UW-17682020.fasta', 'USAWI-UW-17692020.fasta', 'USAWI-UW-17702020.fasta', 'USAWI-UW-17712020.fasta', 'USAWI-UW-17722020.fasta', 'USAWI-UW-17732020.fasta', 'USAWI-UW-17742020.fasta', 'USAWI-UW-17752020.fasta', 'USAWI-UW-17762020.fasta', 'USAWI-UW-17772020.fasta', 'USAWI-UW-17782020.fasta', 'USAWI-UW-17792020.fasta', 'USAWI-UW-17802020.fasta', 'USAWI-UW-17812020.fasta', 'USAWI-UW-17822020.fasta', 'USAWI-UW-17832020.fasta', 'USAWI-UW-17842020.fasta', 'USAWI-UW-17852020.fasta', 'USAWI-UW-17862020.fasta', 'USAWI-UW-17872020.fasta', 'USAWI-UW-17882020.fasta', 'USAWI-UW-17892020.fasta', 'USAWI-UW-17902020.fasta', 'USAWI-UW-17912020.fasta', 'USAWI-UW-17922020.fasta', 'USAWI-UW-17932020.fasta', 'USAWI-UW-17942020.fasta', 'USAWI-UW-17952020.fasta', 'USAWI-UW-17962020.fasta', 'USAWI-UW-17972020.fasta', 'USAWI-UW-17982020.fasta', 'USAWI-UW-17992020.fasta', 'USAWI-UW-18002020.fasta', 'USAWI-UW-18012020.fasta', 'USAWI-UW-18022020.fasta', 'USAWI-UW-18032020.fasta', 'USAWI-UW-18042020.fasta', 'USAWI-UW-18052020.fasta', 'USAWI-UW-18062020.fasta', 'USAWI-UW-18072020.fasta', 'USAWI-UW-18082020.fasta', 'USAWI-UW-18092020.fasta', 'USAWI-UW-18102020.fasta', 'USAWI-UW-18112020.fasta', 'USAWI-UW-18122020.fasta', 'USAWI-UW-18132020.fasta', 'USAWI-UW-18142020.fasta', 'USAWI-UW-18152020.fasta', 'USAWI-UW-18162020.fasta', 'USAWI-UW-18172020.fasta', 'USAWI-UW-18182020.fasta', 'USAWI-UW-18192020.fasta', 'USAWI-UW-18202020.fasta', 'USAWI-UW-18212020.fasta', 'USAWI-UW-18222020.fasta', 'USAWI-UW-18232020.fasta', 'USAWI-UW-18242020.fasta', 'USAWI-UW-18252020.fasta', 'USAWI-UW-18262020.fasta', 'USAWI-UW-18272020.fasta', 'USAWI-UW-18282020.fasta', 'USAWI-UW-18292020.fasta', 'USAWI-UW-18302020.fasta', 'USAWI-UW-18312020.fasta', 'USAWI-UW-18322020.fasta', 'USAWI-UW-18332020.fasta', 'USAWI-UW-18342020.fasta', 'USAWI-UW-18352020.fasta', 'USAWI-UW-18362020.fasta', 'USAWI-UW-18372020.fasta', 'USAWI-UW-18382020.fasta', 'USAWI-UW-18392020.fasta', 'USAWI-UW-18402020.fasta', 'USAWI-UW-18412020.fasta', 'USAWI-UW-18422020.fasta', 'USAWI-UW-18432020.fasta', 'USAWI-UW-18442020.fasta', 'USAWI-UW-18452020.fasta', 'USAWI-UW-18462020.fasta', 'USAWI-UW-18472020.fasta', 'USAWI-UW-18482020.fasta', 'USAWI-UW-18492020.fasta', 'USAWI-UW-18502020.fasta', 'USAWI-UW-18512020.fasta', 'USAWI-UW-18522020.fasta', 'USAWI-UW-18532020.fasta', 'USAWI-UW-18542020.fasta', 'USAWI-UW-18552020.fasta', 'USAWI-UW-18562020.fasta', 'USAWI-UW-18572020.fasta', 'USAWI-UW-18582020.fasta', 'USAWI-UW-18592020.fasta', 'USAWI-UW-18602020.fasta', 'USAWI-UW-18612020.fasta', 'USAWI-UW-18622020.fasta', 'USAWI-UW-18632020.fasta', 'USAWI-UW-18642020.fasta', 'USAWI-UW-18652020.fasta', 'USAWI-UW-18662020.fasta', 'USAWI-UW-18672020.fasta', 'USAWI-UW-18682020.fasta', 'USAWI-UW-18692020.fasta', 'USAWI-UW-18702020.fasta', 'USAWI-UW-18712020.fasta', 'USAWI-UW-23232020.fasta', 'USAWI-UW-24842020.fasta', 'USAWI-UW-24852020.fasta', 'USAWI-UW-24862020.fasta']\n"
     ]
    }
   ],
   "source": [
    "list_of_DFs=load_dataframes('VCFs/')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:20: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:21: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:22: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:24: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "/Users/gagemoreno/opt/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:25: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n"
     ]
    }
   ],
   "source": [
    "clean_dataframes('VCFS/', list_of_DFs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['USAWI-UW-14182020-cleaned', 'USAWI-UW-14192020-cleaned', 'USAWI-UW-14202020-cleaned', 'USAWI-UW-14212020-cleaned', 'USAWI-UW-14222020-cleaned', 'USAWI-UW-14232020-cleaned', 'USAWI-UW-14242020-cleaned', 'USAWI-UW-14252020-cleaned', 'USAWI-UW-14262020-cleaned', 'USAWI-UW-14272020-cleaned', 'USAWI-UW-14282020-cleaned', 'USAWI-UW-14292020-cleaned', 'USAWI-UW-14302020-cleaned', 'USAWI-UW-14312020-cleaned', 'USAWI-UW-14322020-cleaned', 'USAWI-UW-16302020-cleaned', 'USAWI-UW-16312020-cleaned', 'USAWI-UW-16322020-cleaned', 'USAWI-UW-16332020-cleaned', 'USAWI-UW-16342020-cleaned', 'USAWI-UW-16352020-cleaned', 'USAWI-UW-16362020-cleaned', 'USAWI-UW-16372020-cleaned', 'USAWI-UW-16382020-cleaned', 'USAWI-UW-16392020-cleaned', 'USAWI-UW-16402020-cleaned', 'USAWI-UW-16412020-cleaned', 'USAWI-UW-16422020-cleaned', 'USAWI-UW-16432020-cleaned', 'USAWI-UW-16442020-cleaned', 'USAWI-UW-16452020-cleaned', 'USAWI-UW-16462020-cleaned', 'USAWI-UW-16472020-cleaned', 'USAWI-UW-16482020-cleaned', 'USAWI-UW-16492020-cleaned', 'USAWI-UW-16502020-cleaned', 'USAWI-UW-16512020-cleaned', 'USAWI-UW-16522020-cleaned', 'USAWI-UW-16532020-cleaned', 'USAWI-UW-16542020-cleaned', 'USAWI-UW-16552020-cleaned', 'USAWI-UW-16562020-cleaned', 'USAWI-UW-16572020-cleaned', 'USAWI-UW-16582020-cleaned', 'USAWI-UW-16592020-cleaned', 'USAWI-UW-16602020-cleaned', 'USAWI-UW-16612020-cleaned', 'USAWI-UW-16622020-cleaned', 'USAWI-UW-16632020-cleaned', 'USAWI-UW-16642020-cleaned', 'USAWI-UW-16652020-cleaned', 'USAWI-UW-16662020-cleaned', 'USAWI-UW-16672020-cleaned', 'USAWI-UW-16682020-cleaned', 'USAWI-UW-16692020-cleaned', 'USAWI-UW-16702020-cleaned', 'USAWI-UW-16712020-cleaned', 'USAWI-UW-16722020-cleaned', 'USAWI-UW-16732020-cleaned', 'USAWI-UW-16742020-cleaned', 'USAWI-UW-16752020-cleaned', 'USAWI-UW-16762020-cleaned', 'USAWI-UW-16772020-cleaned', 'USAWI-UW-16782020-cleaned', 'USAWI-UW-16792020-cleaned', 'USAWI-UW-16802020-cleaned', 'USAWI-UW-16812020-cleaned', 'USAWI-UW-16822020-cleaned', 'USAWI-UW-16832020-cleaned', 'USAWI-UW-16842020-cleaned', 'USAWI-UW-16852020-cleaned', 'USAWI-UW-16862020-cleaned', 'USAWI-UW-16872020-cleaned', 'USAWI-UW-16882020-cleaned', 'USAWI-UW-16892020-cleaned', 'USAWI-UW-16902020-cleaned', 'USAWI-UW-16912020-cleaned', 'USAWI-UW-16922020-cleaned', 'USAWI-UW-16932020-cleaned', 'USAWI-UW-16942020-cleaned', 'USAWI-UW-16952020-cleaned', 'USAWI-UW-16962020-cleaned', 'USAWI-UW-16972020-cleaned', 'USAWI-UW-16982020-cleaned', 'USAWI-UW-16992020-cleaned', 'USAWI-UW-17002020-cleaned', 'USAWI-UW-17012020-cleaned', 'USAWI-UW-17022020-cleaned', 'USAWI-UW-17032020-cleaned', 'USAWI-UW-17042020-cleaned', 'USAWI-UW-17052020-cleaned', 'USAWI-UW-17062020-cleaned', 'USAWI-UW-17072020-cleaned', 'USAWI-UW-17082020-cleaned', 'USAWI-UW-17092020-cleaned', 'USAWI-UW-17102020-cleaned', 'USAWI-UW-17112020-cleaned', 'USAWI-UW-17122020-cleaned', 'USAWI-UW-17132020-cleaned', 'USAWI-UW-17142020-cleaned', 'USAWI-UW-17152020-cleaned', 'USAWI-UW-17162020-cleaned', 'USAWI-UW-17172020-cleaned', 'USAWI-UW-17182020-cleaned', 'USAWI-UW-17192020-cleaned', 'USAWI-UW-17202020-cleaned', 'USAWI-UW-17212020-cleaned', 'USAWI-UW-17222020-cleaned', 'USAWI-UW-17232020-cleaned', 'USAWI-UW-17242020-cleaned', 'USAWI-UW-17252020-cleaned', 'USAWI-UW-17262020-cleaned', 'USAWI-UW-17272020-cleaned', 'USAWI-UW-17282020-cleaned', 'USAWI-UW-17292020-cleaned', 'USAWI-UW-17302020-cleaned', 'USAWI-UW-17312020-cleaned', 'USAWI-UW-17322020-cleaned', 'USAWI-UW-17332020-cleaned', 'USAWI-UW-17342020-cleaned', 'USAWI-UW-17352020-cleaned', 'USAWI-UW-17362020-cleaned', 'USAWI-UW-17372020-cleaned', 'USAWI-UW-17382020-cleaned', 'USAWI-UW-17392020-cleaned', 'USAWI-UW-17402020-cleaned', 'USAWI-UW-17412020-cleaned', 'USAWI-UW-17422020-cleaned', 'USAWI-UW-17432020-cleaned', 'USAWI-UW-17442020-cleaned', 'USAWI-UW-17452020-cleaned', 'USAWI-UW-17462020-cleaned', 'USAWI-UW-17472020-cleaned', 'USAWI-UW-17482020-cleaned', 'USAWI-UW-17492020-cleaned', 'USAWI-UW-17502020-cleaned', 'USAWI-UW-17512020-cleaned', 'USAWI-UW-17522020-cleaned', 'USAWI-UW-17532020-cleaned', 'USAWI-UW-17542020-cleaned', 'USAWI-UW-17552020-cleaned', 'USAWI-UW-17562020-cleaned', 'USAWI-UW-17572020-cleaned', 'USAWI-UW-17582020-cleaned', 'USAWI-UW-17592020-cleaned', 'USAWI-UW-17602020-cleaned', 'USAWI-UW-17612020-cleaned', 'USAWI-UW-17622020-cleaned', 'USAWI-UW-17632020-cleaned', 'USAWI-UW-17642020-cleaned', 'USAWI-UW-17652020-cleaned', 'USAWI-UW-17662020-cleaned', 'USAWI-UW-17672020-cleaned', 'USAWI-UW-17682020-cleaned', 'USAWI-UW-17692020-cleaned', 'USAWI-UW-17702020-cleaned', 'USAWI-UW-17712020-cleaned', 'USAWI-UW-17722020-cleaned', 'USAWI-UW-17732020-cleaned', 'USAWI-UW-17742020-cleaned', 'USAWI-UW-17752020-cleaned', 'USAWI-UW-17762020-cleaned', 'USAWI-UW-17772020-cleaned', 'USAWI-UW-17782020-cleaned', 'USAWI-UW-17792020-cleaned', 'USAWI-UW-17802020-cleaned', 'USAWI-UW-17812020-cleaned', 'USAWI-UW-17822020-cleaned', 'USAWI-UW-17832020-cleaned', 'USAWI-UW-17842020-cleaned', 'USAWI-UW-17852020-cleaned', 'USAWI-UW-17862020-cleaned', 'USAWI-UW-17872020-cleaned', 'USAWI-UW-17882020-cleaned', 'USAWI-UW-17892020-cleaned', 'USAWI-UW-17902020-cleaned', 'USAWI-UW-17912020-cleaned', 'USAWI-UW-17922020-cleaned', 'USAWI-UW-17932020-cleaned', 'USAWI-UW-17942020-cleaned', 'USAWI-UW-17952020-cleaned', 'USAWI-UW-17962020-cleaned', 'USAWI-UW-17972020-cleaned', 'USAWI-UW-17982020-cleaned', 'USAWI-UW-17992020-cleaned', 'USAWI-UW-18002020-cleaned', 'USAWI-UW-18012020-cleaned', 'USAWI-UW-18022020-cleaned', 'USAWI-UW-18032020-cleaned', 'USAWI-UW-18042020-cleaned', 'USAWI-UW-18052020-cleaned', 'USAWI-UW-18062020-cleaned', 'USAWI-UW-18072020-cleaned', 'USAWI-UW-18082020-cleaned', 'USAWI-UW-18092020-cleaned', 'USAWI-UW-18102020-cleaned', 'USAWI-UW-18112020-cleaned', 'USAWI-UW-18122020-cleaned', 'USAWI-UW-18132020-cleaned', 'USAWI-UW-18142020-cleaned', 'USAWI-UW-18152020-cleaned', 'USAWI-UW-18162020-cleaned', 'USAWI-UW-18172020-cleaned', 'USAWI-UW-18182020-cleaned', 'USAWI-UW-18192020-cleaned', 'USAWI-UW-18202020-cleaned', 'USAWI-UW-18212020-cleaned', 'USAWI-UW-18222020-cleaned', 'USAWI-UW-18232020-cleaned', 'USAWI-UW-18242020-cleaned', 'USAWI-UW-18252020-cleaned', 'USAWI-UW-18262020-cleaned', 'USAWI-UW-18272020-cleaned', 'USAWI-UW-18282020-cleaned', 'USAWI-UW-18292020-cleaned', 'USAWI-UW-18302020-cleaned', 'USAWI-UW-18312020-cleaned', 'USAWI-UW-18322020-cleaned', 'USAWI-UW-18332020-cleaned', 'USAWI-UW-18342020-cleaned', 'USAWI-UW-18352020-cleaned', 'USAWI-UW-18362020-cleaned', 'USAWI-UW-18372020-cleaned', 'USAWI-UW-18382020-cleaned', 'USAWI-UW-18392020-cleaned', 'USAWI-UW-18402020-cleaned', 'USAWI-UW-18412020-cleaned', 'USAWI-UW-18422020-cleaned', 'USAWI-UW-18432020-cleaned', 'USAWI-UW-18442020-cleaned', 'USAWI-UW-18452020-cleaned', 'USAWI-UW-18462020-cleaned', 'USAWI-UW-18472020-cleaned', 'USAWI-UW-18482020-cleaned', 'USAWI-UW-18492020-cleaned', 'USAWI-UW-18502020-cleaned', 'USAWI-UW-18512020-cleaned', 'USAWI-UW-18522020-cleaned', 'USAWI-UW-18532020-cleaned', 'USAWI-UW-18542020-cleaned', 'USAWI-UW-18552020-cleaned', 'USAWI-UW-18562020-cleaned', 'USAWI-UW-18572020-cleaned', 'USAWI-UW-18582020-cleaned', 'USAWI-UW-18592020-cleaned', 'USAWI-UW-18602020-cleaned', 'USAWI-UW-18612020-cleaned', 'USAWI-UW-18622020-cleaned', 'USAWI-UW-18632020-cleaned', 'USAWI-UW-18642020-cleaned', 'USAWI-UW-18652020-cleaned', 'USAWI-UW-18662020-cleaned', 'USAWI-UW-18672020-cleaned', 'USAWI-UW-18682020-cleaned', 'USAWI-UW-18692020-cleaned', 'USAWI-UW-18702020-cleaned', 'USAWI-UW-18712020-cleaned', 'USAWI-UW-23232020-cleaned', 'USAWI-UW-24842020-cleaned', 'USAWI-UW-24852020-cleaned', 'USAWI-UW-24862020-cleaned']\n"
     ]
    }
   ],
   "source": [
    "cleaned = load_cleaned_dataframes('VCFs/cleaned/')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "list_of_snps = make_snp_list('VCFs/Cleaned/', cleaned)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "snps_in_dorms = make_dictionary('VCFs/Cleaned/', cleaned)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'USAWI-UW-14182020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6793_G>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14192020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-14202020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14212020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14222020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-14232020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14242020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '5672_C>T',\n",
       "  '14408_C>T',\n",
       "  '16260_C>T',\n",
       "  '23126_G>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-14252020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16260_C>T',\n",
       "  '23126_G>T',\n",
       "  '23403_A>G',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '27403_A>G',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14262020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-14272020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14282020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14292020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-14302020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-14312020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-14322020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '25609_C>A',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16302020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16312020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16322020': ['241_C>T',\n",
       "  '681_A>G',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3821_C>A',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5055_C>T',\n",
       "  '5672_C>T',\n",
       "  '7837_A>C',\n",
       "  '8658_A>G',\n",
       "  '14408_C>T',\n",
       "  '14700_T>C',\n",
       "  '16647_G>C',\n",
       "  '23120_G>T',\n",
       "  '23403_A>G',\n",
       "  '25854_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16332020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16342020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '12065_A>G',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16352020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16362020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16372020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16382020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17977_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16392020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '10277_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16402020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-16412020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16422020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16432020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13045_G>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16442020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-16452020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16462020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16472020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-16482020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16492020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16502020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16512020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16522020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16532020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16542020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16552020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16562020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16572020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16582020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29362_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16592020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28468_A>C',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16602020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '7420_C>T',\n",
       "  '11029_G>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29005_A>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16612020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16622020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '22015_T>C',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-16632020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16642020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16652020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16662020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16672020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6166_G>A',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16682020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '4113_C>T',\n",
       "  '5907_C>T',\n",
       "  '14408_C>T',\n",
       "  '23191_C>T',\n",
       "  '23403_A>G'],\n",
       " 'USAWI-UW-16692020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16702020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16712020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '2037_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6419_G>A',\n",
       "  '7837_A>C',\n",
       "  '11541_T>G',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '25947_G>T',\n",
       "  '28977_C>T',\n",
       "  '29742_G>A'],\n",
       " 'USAWI-UW-16722020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16732020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-16742020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16752020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16762020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-16772020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16782020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '6793_G>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16792020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6793_G>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16802020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '14408_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-16812020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '18369_T>C',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16822020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16832020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-16842020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '8577_A>G',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16852020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16862020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '28952_T>G'],\n",
       " 'USAWI-UW-16872020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16882020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16892020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16902020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-16912020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6793_G>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16922020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16932020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16942020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16952020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16962020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-16972020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6793_G>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16982020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-16992020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17002020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17012020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3821_C>A',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5055_C>T',\n",
       "  '5672_C>T',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '14700_T>C',\n",
       "  '16117_G>T',\n",
       "  '16647_G>C',\n",
       "  '23120_G>T',\n",
       "  '23403_A>G',\n",
       "  '25854_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17022020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17032020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17042020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17052020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17062020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17072020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '9969_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17082020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17092020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17102020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17112020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '27798_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17122020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23191_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17132020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17142020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17152020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17162020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '17135_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29715_G>T'],\n",
       " 'USAWI-UW-17172020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13664_A>C',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17182020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17192020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-17202020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-17212020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17222020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17232020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17242020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-17252020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17262020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17272020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17282020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17292020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-17302020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4960_G>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17312020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17322020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19547_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17332020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17342020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17352020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17362020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17372020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17382020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '16741_G>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '27403_A>G',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17392020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17402020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28717_C>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17412020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4891_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '28854_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17422020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17432020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14021_A>G',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29260_G>A',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-17442020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '7360_G>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17452020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17462020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17472020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17482020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17492020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17502020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '23403_A>G'],\n",
       " 'USAWI-UW-17512020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17522020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17532020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-17542020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17552020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1737_T>C',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '21920_G>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17562020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17572020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17582020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17592020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17602020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17612020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17622020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-17632020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '16741_G>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '27403_A>G',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17642020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17652020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '7420_C>T',\n",
       "  '11029_G>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29005_A>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17662020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17672020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17682020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6781_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17692020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17702020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-17712020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17722020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-17732020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T'],\n",
       " 'USAWI-UW-17742020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-17752020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6781_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17762020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25290_G>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17772020': ['241_C>T',\n",
       "  '1562_T>C',\n",
       "  '2035_G>T',\n",
       "  '3037_C>T',\n",
       "  '6187_C>T',\n",
       "  '11674_C>T',\n",
       "  '11956_C>T',\n",
       "  '12025_C>T',\n",
       "  '14408_C>T',\n",
       "  '19162_G>T',\n",
       "  '19839_T>C',\n",
       "  '23403_A>G',\n",
       "  '23638_C>T',\n",
       "  '23707_C>T',\n",
       "  '28310_C>T',\n",
       "  '28881_G>A',\n",
       "  '28882_G>A',\n",
       "  '28883_G>C',\n",
       "  '29769_C>T'],\n",
       " 'USAWI-UW-17782020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20104_C>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '27925_C>A',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17792020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17802020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17812020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '6781_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17822020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17832020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28468_A>C',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17842020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '27078_C>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17852020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '11595_A>G',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17862020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17872020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17882020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-17892020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '22015_T>C',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T'],\n",
       " 'USAWI-UW-17902020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16158_G>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17912020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17922020': ['241_C>T',\n",
       "  '338_G>A',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17932020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17942020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '22015_T>C',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T'],\n",
       " 'USAWI-UW-17952020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17962020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17972020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17982020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17550_C>T',\n",
       "  '17706_G>A',\n",
       "  '18432_A>G',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-17992020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18002020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T'],\n",
       " 'USAWI-UW-18012020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-18022020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18032020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-18042020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18052020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18062020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18072020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18082020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18092020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18102020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23584_T>C',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18112020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18122020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18132020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-18142020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18152020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18162020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18172020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18182020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18192020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18202020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '4226_C>T',\n",
       "  '7837_A>C',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18212020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18222020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18232020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19328_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29614_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18242020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18252020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18262020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '2037_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '7837_A>C',\n",
       "  '11541_T>G',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '25947_G>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18272020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18282020': ['241_C>T',\n",
       "  '1562_T>C',\n",
       "  '2035_G>T',\n",
       "  '3037_C>T',\n",
       "  '6187_C>T',\n",
       "  '11674_C>T',\n",
       "  '11956_C>T',\n",
       "  '12025_C>T',\n",
       "  '14408_C>T',\n",
       "  '19162_G>T',\n",
       "  '19839_T>C',\n",
       "  '23403_A>G',\n",
       "  '23638_C>T',\n",
       "  '23707_C>T',\n",
       "  '28310_C>T',\n",
       "  '28881_G>A',\n",
       "  '28882_G>A',\n",
       "  '28883_G>C',\n",
       "  '29769_C>T'],\n",
       " 'USAWI-UW-18292020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18302020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18312020': ['203_C>T',\n",
       "  '241_C>T',\n",
       "  '370_G>T',\n",
       "  '1059_C>T',\n",
       "  '2607_C>T',\n",
       "  '3037_C>T',\n",
       "  '14408_C>T',\n",
       "  '14634_A>G',\n",
       "  '18555_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '28892_C>T'],\n",
       " 'USAWI-UW-18322020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18332020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18342020': ['241_C>T',\n",
       "  '3037_C>T',\n",
       "  '3393_C>T',\n",
       "  '4113_C>T',\n",
       "  '5907_C>T',\n",
       "  '8114_G>A',\n",
       "  '14408_C>T',\n",
       "  '21974_G>C',\n",
       "  '23191_C>T',\n",
       "  '23403_A>G',\n",
       "  '25571_C>A',\n",
       "  '26456_C>T',\n",
       "  '27534_T>C',\n",
       "  '28881_G>A',\n",
       "  '28882_G>A',\n",
       "  '28883_G>C'],\n",
       " 'USAWI-UW-18352020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18362020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18372020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-18382020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18392020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18402020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18412020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '13339_T>A',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '18687_C>T',\n",
       "  '19059_T>C',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-18422020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-18432020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18442020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-18452020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-18462020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18472020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18482020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1063_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '5477_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23191_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18492020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '19118_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18502020': ['241_C>T',\n",
       "  '664_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18512020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18522020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18532020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-18542020': ['241_C>T',\n",
       "  '264_A>G',\n",
       "  '1059_C>T',\n",
       "  '2407_G>A',\n",
       "  '3037_C>T',\n",
       "  '3885_C>T',\n",
       "  '10323_A>G',\n",
       "  '13957_C>T',\n",
       "  '14408_C>T',\n",
       "  '15276_T>A',\n",
       "  '21575_C>T',\n",
       "  '22162_T>C',\n",
       "  '23403_A>G',\n",
       "  '24010_A>G',\n",
       "  '25563_G>T',\n",
       "  '28842_G>T'],\n",
       " 'USAWI-UW-18552020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '10030_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28681_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18562020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '1427_C>T',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '6285_C>T',\n",
       "  '14408_C>T',\n",
       "  '19524_C>T',\n",
       "  '20429_C>T',\n",
       "  '23403_A>G',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18572020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18582020': ['241_C>T',\n",
       "  '696_A>C',\n",
       "  '3037_C>T',\n",
       "  '3871_G>T',\n",
       "  '3931_T>C',\n",
       "  '4226_C>T',\n",
       "  '5672_C>T',\n",
       "  '6166_G>A',\n",
       "  '6352_G>A',\n",
       "  '7837_A>C',\n",
       "  '14408_C>T',\n",
       "  '19677_G>T',\n",
       "  '23403_A>G',\n",
       "  '27276_C>T',\n",
       "  '28977_C>T'],\n",
       " 'USAWI-UW-18592020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T',\n",
       "  '29402_G>T'],\n",
       " 'USAWI-UW-18602020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18612020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18622020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '20321_A>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18632020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '27078_C>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18642020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '13339_T>A',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '18687_C>T',\n",
       "  '19059_T>C',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T'],\n",
       " 'USAWI-UW-18652020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '7745_G>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '18424_A>G',\n",
       "  '19947_G>T',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27225_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '29402_G>T',\n",
       "  '29724_C>T'],\n",
       " 'USAWI-UW-18662020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '22015_T>C',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T'],\n",
       " 'USAWI-UW-18672020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '28963_T>C',\n",
       "  '29627_C>T',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18682020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '13176_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18692020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '1358_G>A',\n",
       "  '3037_C>T',\n",
       "  '4633_C>T',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '10728_C>T',\n",
       "  '12473_C>T',\n",
       "  '13201_G>T',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16748_A>G',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '22015_T>C',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-18702020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '4596_C>A',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15672_G>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '17721_G>T',\n",
       "  '18985_G>T',\n",
       "  '23126_G>T',\n",
       "  '23185_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25460_C>T',\n",
       "  '25563_G>T',\n",
       "  '26801_C>T',\n",
       "  '28727_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-18712020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '3773_C>T',\n",
       "  '7086_C>T',\n",
       "  '14191_T>C',\n",
       "  '14408_C>T',\n",
       "  '15654_C>T',\n",
       "  '16092_C>T',\n",
       "  '16260_C>T',\n",
       "  '23403_A>G',\n",
       "  '23900_G>C',\n",
       "  '24253_A>T',\n",
       "  '25563_G>T',\n",
       "  '28821_C>A',\n",
       "  '29808_G>T'],\n",
       " 'USAWI-UW-23232020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '7736_G>A',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '12820_A>G',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16616_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '21638_C>T',\n",
       "  '21639_C>A',\n",
       "  '22205_G>T',\n",
       "  '22687_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '26748_A>G',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-24842020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '7736_G>A',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '12820_A>G',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16616_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '21638_C>T',\n",
       "  '21639_C>A',\n",
       "  '22205_G>T',\n",
       "  '22687_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '26748_A>G',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-24852020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '7736_G>A',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '12820_A>G',\n",
       "  '14408_C>T',\n",
       "  '14805_C>T',\n",
       "  '16616_C>T',\n",
       "  '18424_A>G',\n",
       "  '21304_C>T',\n",
       "  '21638_C>T',\n",
       "  '21639_C>A',\n",
       "  '22205_G>T',\n",
       "  '22687_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '25907_G>T',\n",
       "  '26748_A>G',\n",
       "  '27964_C>T',\n",
       "  '28472_C>T',\n",
       "  '28869_C>T'],\n",
       " 'USAWI-UW-24862020': ['241_C>T',\n",
       "  '1059_C>T',\n",
       "  '3037_C>T',\n",
       "  '7736_G>A',\n",
       "  '8083_G>A',\n",
       "  '10319_C>T',\n",
       "  '14408_C>T',\n",
       "  '16616_C>T',\n",
       "  '21304_C>T',\n",
       "  '23403_A>G',\n",
       "  '25563_G>T',\n",
       "  '26748_A>G']}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "snps_in_dorms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# this function rnadomly chooses and compares two of the above lists and returns % overlap in snp identities \n",
    "\n",
    "def diff(lists_of_lists, repeat_times):\n",
    "    percent_overlap_distribution = []\n",
    "    for i in range(repeat_times):\n",
    "        li1 = random.choices(lists_of_lists, k=1)\n",
    "        li2 = random.choices(lists_of_lists, k=1)\n",
    "        li1 = li1[0]\n",
    "        li2 = li2[0]\n",
    "        len1 = len(li1)\n",
    "        len2 = len(li2)\n",
    "        total_len = len1 + len2\n",
    "        overlap = (1 - (len((list(list(set(li1)-set(li2)) + list(set(li2)-set(li1))))) / total_len)) * 100\n",
    "        percent_overlap_distribution.append(overlap)\n",
    "    return(percent_overlap_distribution)\n",
    "\n",
    "# compare two particular lists \n",
    "\n",
    "def compare(li1, li2):\n",
    "    li1 = li1[0]\n",
    "    li2 = li2[0]\n",
    "    len1 = len(li1)\n",
    "    len2 = len(li2)\n",
    "    total_len = len1 + len2\n",
    "    overlap = (1 - (len((list(list(set(li1)-set(li2)) + list(set(li2)-set(li1))))) / total_len)) * 100\n",
    "#     print(1 - overlap)\n",
    "    return(overlap)\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "71.42857142857143\n",
      "100.0\n",
      "100.0\n",
      "100.0\n",
      "100.0\n"
     ]
    }
   ],
   "source": [
    "# compare roommates in dorms\n",
    "\n",
    "roommates1 = compare(snps_in_dorms['USAWI-UW-17932020'],snps_in_dorms['USAWI-UW-18632020'])\n",
    "print(roommates1)\n",
    "roommates2 = compare(snps_in_dorms['USAWI-UW-18302020'],snps_in_dorms['USAWI-UW-18532020'])\n",
    "print(roommates2)\n",
    "roommates3 = compare(snps_in_dorms['USAWI-UW-18052020'],snps_in_dorms['USAWI-UW-16492020'])\n",
    "print(roommates3)\n",
    "roommates4 = compare(snps_in_dorms['USAWI-UW-18402020'],snps_in_dorms['USAWI-UW-18352020'])\n",
    "print(roommates4)\n",
    "roommates5 = compare(snps_in_dorms['USAWI-UW-16302020'],snps_in_dorms['USAWI-UW-18342020'])\n",
    "print(roommates5)\n",
    "roommates6 = compare(snps_in_dorms['USAWI-UW-16352020'],snps_in_dorms['USAWI-UW-18622020'])\n",
    "print(roommates6)\n",
    "roommates7 = compare(snps_in_dorms['USAWI-UW-16592020'],snps_in_dorms['USAWI-UW-18022020'])\n",
    "print(roommates7)\n",
    "roommates8 = compare(snps_in_dorms['USAWI-UW-18082020'],snps_in_dorms['USAWI-UW-18322020'])\n",
    "print(roommates8)\n",
    "roommates9 = compare(snps_in_dorms['USAWI-UW-16342020'],snps_in_dorms['USAWI-UW-16422020'])\n",
    "print(roommates9)\n",
    "roommates10 = compare(snps_in_dorms['USAWI-UW-16462020'],snps_in_dorms['USAWI-UW-16632020'])\n",
    "print(roommates10)\n",
    "roommates11 = compare(snps_in_dorms['USAWI-UW-18472020'],snps_in_dorms['USAWI-UW-18592020'])\n",
    "print(roommates11)\n",
    "roommates12 = compare(snps_in_dorms['USAWI-UW-17952020'],snps_in_dorms['USAWI-UW-18552020'])\n",
    "print(roommates12)\n",
    "roommates13 = compare(snps_in_dorms['USAWI-UW-18392020'],snps_in_dorms['USAWI-UW-18542020'])\n",
    "print(roommates13)\n",
    "roommates14 = compare(snps_in_dorms['USAWI-UW-17942020'],snps_in_dorms['USAWI-UW-16702020'])\n",
    "print(roommates14)\n",
    "roommates15 = compare(snps_in_dorms['USAWI-UW-16832020'],snps_in_dorms['USAWI-UW-17662020'])\n",
    "print(roommates15)\n",
    "roommates16 = compare(snps_in_dorms['USAWI-UW-17262020'],snps_in_dorms['USAWI-UW-16822020'])\n",
    "print(roommates16)\n",
    "roommates17 = compare(snps_in_dorms['USAWI-UW-18192020'],snps_in_dorms['USAWI-UW-18252020'])\n",
    "print(roommates17)\n",
    "roommates18 = compare(snps_in_dorms['USAWI-UW-17392020'],snps_in_dorms['USAWI-UW-17762020'])\n",
    "print(roommates18)\n",
    "roommates19 = compare(snps_in_dorms['USAWI-UW-17542020'],snps_in_dorms['USAWI-UW-17572020'])\n",
    "print(roommates19)\n",
    "roommates20 = compare(snps_in_dorms['USAWI-UW-17512020'],snps_in_dorms['USAWI-UW-17442020'])\n",
    "print(roommates20)\n",
    "roommates21 = compare(snps_in_dorms['USAWI-UW-16852020'],snps_in_dorms['USAWI-UW-17252020'])\n",
    "print(roommates21)\n",
    "roommates22 = compare(snps_in_dorms['USAWI-UW-17162020'],snps_in_dorms['USAWI-UW-17602020'])\n",
    "print(roommates22)\n",
    "roommates23 = compare(snps_in_dorms['USAWI-UW-17752020'],snps_in_dorms['USAWI-UW-17812020'])\n",
    "print(roommates23)\n",
    "roommates24 = compare(snps_in_dorms['USAWI-UW-17362020'],snps_in_dorms['USAWI-UW-18152020'])\n",
    "print(roommates24)\n",
    "roommates25 = compare(snps_in_dorms['USAWI-UW-17452020'],snps_in_dorms['USAWI-UW-17422020'])\n",
    "print(roommates25)\n",
    "roommates26 = compare(snps_in_dorms['USAWI-UW-17032020'],snps_in_dorms['USAWI-UW-17702020'])\n",
    "print(roommates26)\n",
    "roommates27 = compare(snps_in_dorms['USAWI-UW-17382020'],snps_in_dorms['USAWI-UW-17632020'])\n",
    "print(roommates27)\n",
    "roommates28 = compare(snps_in_dorms['USAWI-UW-16862020'],snps_in_dorms['USAWI-UW-18062020'])\n",
    "print(roommates28)\n",
    "roommates29 = compare(snps_in_dorms['USAWI-UW-17502020'],snps_in_dorms['USAWI-UW-17532020'])\n",
    "print(roommates29)\n",
    "roommates30 = compare(snps_in_dorms['USAWI-UW-18232020'],snps_in_dorms['USAWI-UW-17352020'])\n",
    "print(roommates30)\n",
    "roommates31 = compare(snps_in_dorms['USAWI-UW-17072020'],snps_in_dorms['USAWI-UW-17212020'])\n",
    "print(roommates31)\n",
    "roommates32 = compare(snps_in_dorms['USAWI-UW-16742020'],snps_in_dorms['USAWI-UW-16752020'])\n",
    "print(roommates32)\n",
    "roommates33 = compare(snps_in_dorms['USAWI-UW-16972020'],snps_in_dorms['USAWI-UW-16782020'])\n",
    "print(roommates33)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "71.42857142857143\n"
     ]
    }
   ],
   "source": [
    "print(roommates29)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAysAAAE4CAYAAABbimxAAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAPYQAAD2EBqD+naQAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzdeXjM5/7/8eckyCIbQVP7HlLUriiJkiAURWuv9LSHWFq1tUhCErXWegit1tLiS1C6WEsrdtWq9aB2tdYSWWwhyfz+8DOn0wQZJhKZ1+O65rpm7rnnk/cs2nnNvXwMRqPRiIiIiIiISDZjl9UFiIiIiIiIpEdhRUREREREsiWFFRERERERyZYUVkREREREJFtSWBERERERkWxJYUVERERERLIlhRUREREREcmWFFZERERERCRbUlgREREREZFsSWFFRERs1r1796hWrRpBQUFZXYrYoLt37/Lpp5/y0ksv4ezsTMmSJfn444+5ceNGmr5nzpwhKCiIwoULkzt3bgoXLsy7777LhQsXsqBykWdHYUVERGxSSkoK3bp1Y+/evVlditio7t2789FHH1G5cmUmT57M66+/zqRJk/D19eXOnTumfleuXKFu3bpER0fz1ltvMX36dNq1a8eCBQuoXbs2Fy9ezMJnIZK5cmV1ASIiIs/an3/+Sbdu3di8eXNWlyI2auPGjSxevJhevXoxY8YMU3upUqUYOHAgCxYs4L333gMgNDSUixcvsnbtWpo2bWrq6+/vT+vWrfnkk0+Iiop65s9B5FnQyIqIiNiU+fPn4+3tza5duxg2bFhWlyM26tKlS1SvXp3g4GCz9oCAAAB+//13U9v69espXbq0WVABaNWqFe7u7mzatCnzCxbJIgorIiKSLfn5+VGpUiX2799PYGAg7u7uuLi4EBAQwK+//mrqZzAYHnuJiYkx9d+7dy8tW7bkwIED/Pvf/86CZybPm8z4LHbq1Indu3dTpUoVs7/1IKSULFnS1LZp0yZWrlyZpq6bN29y8+ZN7O3trf+kRbIJTQMTEZFs69KlSzRs2JAWLVowfvx4Tp06xeTJk2nSpAl//vkn7u7uzJ8//7HHqVixoun66NGjcXBwAOD06dOZVbrkMJnxWXzg3r17/Pnnn/z4448MGTKE0qVLm6aAARQrVizdY02ZMoXk5GQaNWr05E9MJJtTWBERkWzr2rVrjBs3jo8++sjU5uLiQlhYGEuWLOHf//43Xbt2teiYD4KKiCUy47P4wJo1a2jdujUAzs7OzJgxg/z58z/yMT///DORkZG4ubkxcODAJ/q7Is8DhRUREcnW/vkFsFatWsD9X7oBrl69+thjuLu7kzt3busXJzYlsz6LZcuWZfny5Vy7do0pU6bQvHlzZs6cSc+ePdM9xo8//sgbb7xBSkoK8+fPf+jIi0hOoLAiIiLZmpeXl9ntByMjKSkpABQsWPCxx9i4cSN+fn5Wr01sS2Z9Fn18fPDx8QGgQ4cOVK5cmcGDB9OlSxdcXFzM+s6ZM4fg4GAMBgOLFy+mVatWT/p0RJ4LCisiIpKt2dk9ei+Y9evXP/YYL7/8srXKERv2LD6Lrq6utGrVimnTpvHHH39Qo0YN032hoaGMGjUKd3d3li9fzmuvvZaxwkWeYworIiLyXGvSpElWlyACWPZZDA4OZvny5Rw6dIgCBQqY3ZeYmAiAk5OTqe3DDz9k6tSpFC9enNWrV/PSSy9Zp2iRbE5bF4uIiIg8Y2XKlOHKlStMmjTJrP3YsWMsXboUb29v085hs2bNYurUqZQpU4bt27crqIhN0ciKiIiIyDP2/vvvs2TJEsaOHcuZM2do2LAhp0+fZubMmRgMBubNm4fBYODmzZsMHToUgDfeeIONGzemOZaLiwtt2rR56pqS/jzLvWvXnvo4TyO3pycOxbVhgDUYjUYMBkNWl/HUFFZEREREnjFHR0diYmIYOXIk0dHRLFmyhHz58hEYGMiIESPw9vYGYPfu3cTGxgIwYcKEdI9VokSJpw4rSX+e5fdK1TDeSXqq4zwtg6MD1Q/ueaLA4ufnx6ZNm8yPZzDg6upKhQoV6N+/Px07drRWqdnaqFGjyJMnD4MHD87qUp6awWg0GrO6CBERERHJOjf27GV/nQZZXQYAVX7Zgku1qhY/zs/Pj4SEBGbMmGFqS0lJ4ezZs0yePJldu3axZs0amjVrZs1ysyWDwcCIESMIDw/P6lKemkZWRERERCRHcHNz45VXXknTHhgYSKFChZgzZ45NhJWcRAvsRURERCRHc3R0JE+ePKbbKSkpzJgxg8qVK+Pk5ETx4sUZMmQId+7cMXvc+vXradCgAe7u7nh6etK5c2fOnj1run/evHk4Ojqybds2atWqhaOjI97e3vzwww/88ccfNG7cGGdnZ8qWLcvixYuf+nEAmzdvpmnTpuTLl488efJQqlQpwsPDSU1NBTCtU4mIiDBbs3Lw4EFatmyJm5sbbm5uvPHGG5w8edLs2NOmTaNChQo4OjpSpEgRevfubdqdLqsorIiIiIhIjmA0GklOTjZd7ty5w/Hjx3nvvfdITEykW7duAPTs2ZN+/frRunVrvv/+e/r27cu0adNo3bo1D1ZILFiwgICAAIoUKcKiRYuYPHkyO3bsoG7duly+fNn0N+/du0fHjh3p2bMn3333HY6OjnTp0oWWLVvSokULli5dyosvvkj37t05d+7cUz1u3759NG7cGE9PT6Kjo/nhhx+oX78+ERERplCzY8cOAN59913T9aNHj1KvXj0uX77MvHnzmD17NidPnqR+/fqm57J48WIGDRpEnz59WLduHcOHD2f+/Pm8//77mfyuPZqmgYmIiIhIjrB582Zy585t1mYwGKhcuTJLly7l9ddf59ChQ8yePZtPPvmEkJAQAPz9/SlcuDDdunVj7dq1NG3alMGDB9OkSROzkY369evj4+PDxIkTGTduHACpqamEhITw3nvvAXD9+nU6derEhx9+yIABAwDw8vKiZs2a/PbbbxQtWvSJH7d//378/f1ZsGCB6SSl/v7+rFy5kk2bNtG5c2fTNLiiRYuarkdERODk5MSGDRtwc3MDoHHjxpQuXZpPP/2UTz/9lJiYGEqWLEmfPn2ws7PD19cXFxcXrl69auV3yTIKKxYyGo0kJibi6uqaI7aDExEREckpqlevzueffw7AuXPnCAsL4969eyxevNh03poHO4Z16dLF7LEdO3YkKCiIjRs3UrJkSS5dusSYMWPM+pQpU4a6deum2UK6Xr16puteXl4AZmtnPD09AYiLi3uqx3Xr1o1u3bpx584dTpw4wbFjx/j9999JTk7m7t27D31dfvrpJxo1aoSzszPJycnA/fU9DRo0YP369QA0atSIzz//nBo1atCuXTtatGhB586ds/z7rqaBWSgxMRF3d/csn78nIiLpS7l5k+15XNmex5WUmzezuhwRfSafIVdXV2rWrEnNmjVp06YNP/30E/Hx8QQEBHDlyhUA01bQD8LBA7ly5aJAgQLExcU9tM+Dtn+GjgejFX/n7Oz82Hotfdzt27d57733cHd3p3LlygwcOJBTp06RO3duHrXB77Vr14iOjiZ37txml5UrV3LhwgUAOnTowP/93//h4uJCeHg41atXp3Tp0mnWzDxrCisiIiIikiMVKlSIqKgozp07R79+/QDInz8/AJcuXTLre+/ePa5evUqBAgUe2gfg4sWLFChQIJMrT1+/fv1YtmwZ0dHRJCYmcuLECebPn59m6ts/eXh40LFjR3799dc0lx9//NHUr1OnTmzZsoVr166xZMkSPD096dq1qynQZAWFFRERERHJsdq2bUuzZs1YtGgRMTEx+Pr6ArBw4UKzfosXLyYlJYVXX30Vb29vvLy80vQ5efIkO3bs4NVXX31m9f/d1q1badSoEW3atCFv3rzA/ROHXrlyxbQbGGBaz/KAr68vhw4domrVqqaRpxo1ajB58mRWrFgB3B9Zadu2LQDu7u68+eabhIWFkZKSkqVhRWtWRERERCRHmzJlCpUrV+b9999nz549dO/enfDwcG7fvo2fnx979+4lPDycRo0a0axZM+zs7BgzZgzvvPMOHTt2pHv37ly9epXw8HDy589vWgD/rNWuXZslS5bw2WefUbFiRfbt28cnn3yCwWDg5t+mGHp4eLB9+3Y2b95MgwYNGD58OHXr1qVly5b06tULR0dHPv/8c7799luWLVsGwGuvvUZwcDCDBg0iMDCQ69evEx4eTrly5Xj55Zez5PmCwoqIiIiI5HDe3t7069ePCRMmMH36dGbPnk25cuWYM2cO48ePp0iRInzwwQcMHz7cNCoRFBSEq6srY8aMoU2bNri5udGsWTNGjx6d7lqWZ2HSpEncu3eP0NBQkpKSKFWqFKGhofz3v//lhx9+ICUlBXt7e0JCQhg5ciTNmzfn8OHDVKlShS1bthASEkK3bt0wGo1UqlSJb7/9llatWgH3t3O+e/cun332GTNmzMDJyYkmTZowfvz4x04zy0wG46NW40gaCQkJuLu7Ex8fn+6iKBERyVopN2/yS777XyTqXL+E/f+fKiGSVZ6Hz2TSn2f5vVI1jHeSsrQOg6MD1Q/uwaF4sSytQ7IPjayIiIiI2DiH4sWofnAP965dy9I6cnt6KqiIGYUVEREREcGheDEFBcl2tBuYiIiIiIhkSworIiIiIiKSLSmsiIiIiIhItqQ1KyIiIlYSERGRpm3EiBFZUImISM6gkRUREREREcmWFFZERERERCRbUlgREREREZFsSWtWRERERIT4+Hhu3bqVpTU4Ozvj7u6epTVI9qKwIiIiImLj4uPjmT59OsnJyVlaR65cuejbt6/FgcXPz49NmzaZtRkMBlxdXalQoQL9+/enY8eO1iz1kUqWLImfnx/z5s17Zn8zMwQFBRETE8Pp06ezrAaFFREREREbd+vWrSwPKgDJycncunXriUZXqlWrxowZM0y3U1JSOHv2LJMnT6ZTp054eHjQrFkza5ab44WFhdGvX78srUFhRURERESee25ubrzyyitp2gMDAylUqBBz5sxRWLFQmTJlsrqEJ1tgv3nzZrZv3w7A6dOnadGiBVWqVGHkyJFWLU5ERERE5Gk4OjqSJ08es7bbt28zdOhQypUrh4ODA25ubvj7+7N3715Tn6CgIJo0acLcuXMpX748Dg4OvPzyy6xevdrsWPv378ff3x8XFxdKlCjBwoUL09Rw584dRo4cSYUKFXB0dKRcuXKMGzeO1NRUUx8/Pz969uzJqFGjKFKkCM7OzgQGBvLXX38xd+5cypYti4uLC02aNHnktKzTp09jMBhYvHgxr7/+Os7OzhQrVozw8HCzv5fR16BkyZKm2yVLlqR///40btwYNzc3goODAZg2bZrpuRUpUoTevXuTmJj46DcmgyweWVmwYAHdu3dn4MCB1KtXj169erF161b8/f0ZNWoUefLk4eOPP7ZKcSIiIiIiGWE0Gs2msiUnJ3Pu3DkiIyNJTEykW7dupvvefvttNm3axNixYylTpgxHjx4lLCyMjh07cvjwYQwGAwC//fYbFy5cIDIyEnd3d8LCwmjfvj3nz58nX758nD9/noYNG1KmTBkWLlxIQkICH3/8MX/99ZdZXa+//jo7duxgxIgRVK1alY0bNxISEsKJEyeYNWuWqe/ixYupVq0as2fP5syZM/Tt2xdfX1+cnJyYMGECsbGxfPDBB/Tp04dVq1Y98vXo1asXLVq0YPny5WzdupWRI0eSmJjIxIkTLXoN/mn69On07duXIUOG4OzszOLFixk0aBATJkygSpUqHDlyhEGDBnHr1i2rrNmxOKxMnDiRoKAgxo8fz+XLl1m/fj1jx45l0KBBTJw4kc8//1xhRURERESeqc2bN5M7d26zNoPBQOXKlVm6dCmvv/46AHfv3iUxMZFp06bRoUMHAHx9fUlMTGTgwIFcunSJF198Ebi/8cDu3btN06Hy5s2Lr68vP//8M+3atWPKlCncu3ePNWvWUKhQIQDKly9vNh1t7dq1bNiwgQULFtClSxcA/P39cXZ2JiwsjA8//BAfHx9TbStWrCBfvnwArFixgnXr1nHixAlKly4NwJ49e5g/f/5jX4/q1auzYMECAJo1a8aNGzeYNm0aYWFhODs7Z/g1+KfChQszceJE7OzuT9AKDg6mZMmS9OnTBzs7O3x9fXFxceHq1auPrTEjLA4rR44cYfLkyQCsWbMGo9FI69atAahVqxahoaFWKUxEREREJKOqV6/O559/DsC5c+cICwvj3r17LF68mIoVK5r65cmTh7Vr1wJw8eJFjh07xpEjR1i5ciVwPzA8ULBgQbN1G0WLFgXg5s2bAGzZsoW6deuaggpAnTp1KF68uOl2TEwM9vb2plDwQNeuXQkLCyMmJsYUVipWrGgKKgBeXl4ULFjQFFQAPD09iY+Pf+zr0bVrV7Pb7dq1Y+rUqezcuZNmzZpl+DX4Jx8fH1NQAWjUqBGff/45NWrUoF27drRo0YLOnTs/dGTGUhavWfHw8DDNQVu9ejUlSpSgXLlyAJw4cYICBQpYpTARERERkYxydXWlZs2a1KxZkzZt2vDTTz8RHx9PQEAAV65cMeu7bt06KlasSOHChWnZsiVff/01Dg4OwP1pWw84OzubPe7Bl/QHaz9iY2MpWLBgmlr+PioRGxtLgQIFyJXLfIzAy8sLgLi4OFObm5tbmmP9s4aMKly4sNntB4Hq+vXrQMZfg3964YUXzG536NCB//u//8PFxYXw8HCqV69O6dKlWbx48RPV/U8Wh5XGjRsTHh7OmDFjWL58uSklfvPNN4SFhREQEPDExZw9exYPDw9iYmLM2jt27IjBYEhz+fuLkJiYSHBwMF5eXuTNmxd/f38OHTqU5m9MmjSJMmXK4OjoSNWqVfnuu++euF4RERERyZ4KFSpEVFQU586dM9t+98SJE7Rp04aXX36Z48ePk5CQwNatW03TxCxRoEABs/UpD1y7ds10PX/+/Fy9ejXN1tAXL140HSMz/L0GwFRnoUKFrPoaAHTq1IktW7Zw7do1lixZgqenJ127duXChQtP/TwsDitTp06lQIECREZG4u/vz7BhwwDo378/xYsXZ8yYMU9UyJkzZ/D39093WGvv3r107dqVHTt2mF38/f1NfTp16sTy5csZO3Ys8+fP5/Lly7z22mvExsaa+nz66ad89NFHBAUFsWLFCsqWLUu7du3YvHnzE9UsIiIiItlX27ZtadasGYsWLTL9GL57927u3LnD0KFDzaZ4rVmzBsBsx6zHady4Mdu3b+f8+fOmtkOHDnHy5EnTbV9fX1JSUoiOjjZ77IP1JK+++qrFzysjvv/+e7Pby5Ytw9nZmVdeecWqr0GHDh1o27YtAO7u7rz55puEhYWRkpJilbBi8ZoVT09P1q1bl6Z969atZvPzMio1NZWvvvqKQYMGpXv/rVu3OHbsGEOHDk1372yAHTt2sGrVKlatWkVgYCAADRo0oFSpUsyYMYPQ0FBu377N6NGjGTBgAGFhYcD9xUb16tUjMjKSDRs2WFy7iIiI5EwRERHpto8YMeIZVyJPa8qUKVSuXJn333+fPXv2UL16dXLlysXHH3/MwIEDSUpKYu7cuabdtR6sR8mIDz/8kNmzZ9O0aVMiIiJISUkhJCTEbKvk5s2b06hRI3r27MmFCxeoVq2aaReu7t27m9arWNvSpUvx8vIiMDCQmJgYoqKiGDVqFHnz5rXqa/Daa68RHBzMoEGDCAwM5Pr164SHh1OuXDlefvnlp34eT3SeFbg/3+3cuXP8+eef/PnnnwBm1zNq//799OrVi+7du6e7s8H+/ftJTU2latWqDz3GunXryJs3r9kUtIIFC+Lr62vaC/uXX34hLi7OlPzg/g4Rbdu2JSYmhtu3b1tUt4iIiIhkf97e3vTr14+DBw8yffp0ypYty6JFizh37hytWrWiZ8+ewP2F8AaDgS1btmT42J6enmzdupXSpUsTFBREv3796N27t9mXdIPBwMqVKwkODmbq1Km0aNGCpUuXMnr0aGbPnm315/vAyJEjOXToEK1bt+abb74hKiqKIUOGAFj1NejZsyf/+c9/WLNmDS1btqRHjx74+Piwfv36NLuzPQmD8VEraNJx7NgxgoKC2Llz50P7pKSkZPh4sbGx3Lp1i6JFixITE0OjRo3YuHEjfn5+AHz22Wf06tWL4OBgVqxYQWxsLHXq1GHChAnUqVMHuD/8dPjwYfbv32927H79+rFw4UKuXr3KzJkz6d27N9euXSN//vymPitWrKBt27YcOHCASpUqPbbehIQE3N3diY+PT3cRlIiIZK2Umzf5Jd/9hat1rl/CPm/eZ/a30/s1Xr/EP5+sObKSlZ/JjIqPj2f69Olp1lU8a7ly5aJv3764u7tnaR3Ps9OnT1OqVCnmzp1LUFBQVpfz1CyeBvb+++9z9OhRwsPDKVq0qNnWZU8if/78ZuHhnx6cRfP27dssXryYa9euMXbsWBo1asTOnTupUqUKcXFx6QYHV1dXEhISgP/ttPDPfq6urgCmfv+UlJREUlKS6fbD+omIiIg8r9zd3enbty+3bt3K0jqcnZ0VVMSMxWFl8+bNzJ49m06dOmVGPWn079+fN998k8aNG5vaGjduTLly5Rg1ahTR0dGkpqamu5ez0WhMs8Vcen2Ah4auMWPGPPTXFREREZGcwt3dXUFBsh2Lh0Xc3NweORJibd7e3mZBBe6f66V+/frs27fPdDu9EY8bN26Y/tF5eHgAmM4R8/c+wEP/cQ4dOpT4+HjT5ezZs0/3hEREREREMknJkiUxGo05YgoYPEFYefvtt4mKinrkyWKsafHixaxfvz5N++3bt037Unt7e3Pq1Kk0oyfHjx837bDg7e1tavtnHwcHB7Mzg/6dg4MDbm5uZhcREREREcl8Fk8Dc3Z2ZsuWLZQtW5ZatWqlOaumwWCw6s4GM2bM4MKFCxw6dMi0Ddz58+fZtm0bH374IQABAQGMGjWKdevW0bx5cwCuXLnCpk2bCAkJAaBevXrkzZuXZcuWUatWLeD+FLDly5fj6+trOmOniIiIiIhkDxaHla+++goPDw9SU1P55Zdf0tyf3tqRpzF8+HCaNm1K27Zt6du3L7GxsYSHh5MvXz7TuVkaNmyIn58fXbp0Yfz48Xh6ehIeHo6HhwfBwcHA/ZA1aNAgIiMjyZMnD/Xq1WPOnDns3r2bjRs3WrVmERERERF5ehaHlVOnTmVGHQ/VpEkT1q5dS2RkJB06dMDOzo6mTZsyfvx40zoUgOXLlzNgwAAGDx5Mamoq9evXZ8mSJeTLl8/UZ8SIEeTKlYtZs2YxYcIEfHx8+P7776lfv/4zfU4iIiIiIvJ4FoeVB+Li4ti5cydxcXEULFiQWrVqPfV6Dj8/v3TXwvj7++Pv7//Ix+bLl4+5c+cyd+7ch/YxGAyEhoYSGhr6VHWKiIiIiEjme6KwMnbsWEaOHGl21vc8efIQEhJCWFiY1YoTERERERHbZfFuYHPnzmXYsGF07tyZjRs3cvjwYX7++We6dOlCeHg4X331VWbUKSIiIiIiNsbikZVJkybRq1cvoqKiTG3e3t74+fnh5OTE1KlT6d69u1WLFBERERER22PxyMrx48dp06ZNuve1bt2aI0eOPHVRIiIiIiIiFoeVIkWKPHRHsJMnT+qkiSIiIiIiYhUWh5VWrVoxfPhwdu7cada+Y8cORowYQatWraxWnIiIiIiI2C6L16yEh4ezfv166tevT4kSJfDy8uLSpUucOXOGihUrMnbs2MyoU0REREREbIzFYcXNzY1ff/2VuXPnsmnTJmJjY6lduzaDBw8mKCgIJyenzKhTREQkx4iIiEjTNmLEiCyoREQke3ui86w4OjrSq1cvevXqBUBycjIJCQkKKiIiIiIiYjUWr1lJTk4mMjKShQsXAvDzzz9TqFAhChYsSOPGjbl+/brVixQREREREdtjcVgZMWIEI0eOJD4+HoAPP/yQAgUKMHnyZI4fP87QoUOtXqSIiIiIiNgei6eBLVq0iDFjxtC7d2/++OMPDh48yLx583j77bfx9PRk0KBBfPbZZ5lRq4iIiIiI2BCLR1YuXLhAnTp1AFi9ejV2dnYEBgYCULRoUdOIi4iIiIiIyNOwOKwULlzYdFLIFStWUK1aNQoUKADA9u3bKVq0qHUrFBERERERm2RxWOnatSsDBgygadOmbN26lX/961/A/bUr4eHhdO3a1epFioiIiIiI7bF4zUpERAROTk5s3ryZsWPHmrYv/vXXXxk4cCAhISFWL1JERERERGyPxWHFYDAwdOjQNLt+bdu2zWpFiYiIiIiIZCisfP3117Ro0QJPT0++/vrrx/Z/++23n7owERERERGxbRkKK0FBQezcuRNPT0+CgoIe2ddgMCisiIiIiIjIU8tQWDl16hQvvvii6bqIiIiIiEhmy1BYKVGihOl637596devH02aNMm0okRERERERCzeunjz5s3kymXxunwRERERERGLWJw6AgIC+PLLL3nllVdwdHTMjJpEREREnlpERESathEjRmRBJSLypCwOK46OjkRHR7N8+XJKlSrFCy+8YHa/wWDgp59+slqBIiIiIiJimywOK+fOnaN+/fqm20aj0ez+f94WERERERF5EhaHlY0bN2ZGHSIiIiIiImYsXmD/KDdv3mTt2rXWPKSIiIiIiNgoi0dWzpw5Q8+ePdm0aRN3795Nt09KSspTFyYiIiIiIrbN4rDSv39/tm/fTo8ePdi2bRvOzs7UrVuXH3/8kQMHDrB8+fLMqFNERERERGyMxdPANm3axCeffMLUqVN55513cHBwYNy4cfz222/4+vry3XffZUadIiIiIiJiYywOKzdu3KBq1aoA+Pj4sHfvXgDs7e3p06cPP//8s3UrFBERERERm2RxWHnxxRe5dOkSAGXLliU2NpaLFy8CkD9/fv766y/rVigiIiIiIjbJ4rDSokULwsLC2L59O8WKFaNo0aJMmDCBxMRE5syZQ5EiRTKjThERERERsTEWh5XIyEg8PDwYPnw4AKNHj2bq1Kl4eHiwcKq5JxAAACAASURBVOFCBg4caPUiRURERETE9li8G5inpye//PKLaepXly5dKFGiBDt27KB27dr4+vpavUgREREREbE9FoeVB1588UWOHj3K9evXKVKkCIMHD7ZmXSIiIiIiYuOe6Az2UVFRvPjii1SsWJF69epRtmxZSpcuzZIlS6xdn4iIiIiI2CiLR1aioqJ4//33ad26Ne3ataNQoUL89ddfLF26lE6dOpEnTx7atGmTGbWKiIiIiIgNsTisTJkyhd69ezN9+nSz9m7duhEcHExkZKTCioiIiIiIPDWLp4GdP3+e1q1bp3tfu3btOHLkyFMXJSIiIiIiYvHISq1atVi7di3+/v5p7tuxYwdVqlSxSmEiIiLZRURERJq2ESNGZEElIiK2xeKwEhoaSseOHUlISKBr164ULlyYa9eu8f333zN58mSmTJnC5s2bTf0bNmxo1YJFRERERMQ2WBxWmjZtCsDs2bOZM2eOqd1oNALQu3dv022DwUBKSoo16hQRERERERtjcVjZuHFjZtQhIiIiIiJixuKwojPUi4iIiIjIs/BEZ7D/7bff2L59O3FxcWnuMxgMhIWFPXVhIiIiIs+rv2/KYHf3Lo2zsBaR55nFYWXq1KkMGDDAtEblnxRWRERERETEGiw+z8rEiRNp3bo1V69eJTU1Nc1FC+pFRERERMQaLA4r169fp1evXuTPnz8z6hEREREREQGeIKwEBASwffv2zKhFRERERETExOI1K1FRUTRq1IjTp09Tp04dnJ2d0/R5++23rVKciIiIiIjYLovDyqpVqzh+/Dh//PEHX331VZr7DQaDwoqIiIiIiDw1i8NKZGQkjRo1YuTIkXh5eWVGTSIiIiIiIpavWbly5QpDhgyhTp06lChRIt3Lkzp79iweHh7ExMSYtV+8eJFOnTpRoEAB3NzcaN++PefPnzfrk5yczLBhwyhWrBhOTk7UrVuXbdu2pfkbCxYs4KWXXsLJyQlvb2++/PLLJ65XREREREQyj8UjK6+88goHDhzgtddes2ohZ86coWnTpsTHx5u1Jycn07x5c27cuMHMmTO5d+8eQ4YMISAggL1795I7d24A+vXrx7x58xg3bhwlSpRg0qRJNG3alN9//53y5csDsHTpUt5++2369etHs2bN+Pbbb/n3v/+Nk5MTXbp0serzkezp7yfpemDEiBFZUImIiIiIPI7FYSUsLIxOnTpx+fJl6tati5ubW5o+DRs2zPDxUlNT+eqrrxg0aFC69y9dupR9+/Zx8OBBXnrpJQCqVq1KpUqViI6OpmvXrpw9e5ZZs2YxdepUevfuDdzftax8+fKMHz/eNHoSGhpK+/btmTx5MgBNmzYlNjaWESNGKKyIiIiIiGQzFoeVxo0bAzBmzBjg/oL6B4xGIwaDwaITQ+7fv59evXrRu3dvmjRpQosWLczuX7duHd7e3qagAuDj40PFihVZvXo1Xbt25aeffiI5OZm2bdua+jg4ONCyZUtWrFgBwOnTpzl69GiaX9bbt2/PkiVLOHr0qGkERkREREREsp7FYWXjxo1WLaB48eIcP36cokWLplmrAnD48OF0Q0TZsmU5evSoqY+Li0uaBf9ly5bl4sWL3Lhxg8OHDwOkOVbZsmUBFFZERERERLIZi8OKr6+vVQvInz8/+fPnf+j9cXFxlCtXLk27q6srCQkJpj7u7u7p9gFISEggLi4OIM20tb/3SU9SUhJJSUmm2w/rJyIiIiIi1mXxbmBwfxSiU6dOeHl54ejoSNGiRenUqRNHjhyxdn2kpqaaTTV7wGg0Ymdn99g+AHZ2dqSmpgKk6ff3PukZM2YM7u7upkuxYsWe/MmIiIiIiEiGWTyycujQIerVq0fu3Llp2bIlXl5eXLx4kZUrV7Jq1Sp++eUXKlasaLUCPTw80h3NuHHjhmk05VF9ANzd3fHw8ADSjoz8vU96hg4dyoABA0y3ExISFFhERERERJ4Bi8PKkCFDKFWqFDExMWZf8OPj43nttdcICQlh+fLlVivQ29ubPXv2pGk/fvw4tWvXNvVJSEjgypUrFCxY0KxPyZIlTedUedBWrVo1sz5wf9F+ehwcHHBwcLDa8xERERERkYyxeBrYpk2bCAkJSTMS4e7uztChQ9m0aZPVioP7WxAfPnyYQ4cOmdoOHTrE4cOHCQgIAMDf3x+AZcuWmfokJSWxcuVKU5+yZctSunRpsz4PHlO+fPmnOpmliIiIiIhYn8UjK7lz537oSIODg4PZYnRr6NChA6NHj6Z58+aMHTsWuD+6U7lyZd58800ASpQoQffu3enfvz+3b9+mfPnyTJo0ibi4OAYPHmw6VlhYGO+88w6enp60atWK77//niVLlhAdHW3VmsV6dBJHEREREdtlcVipVasWUVFRtGzZMs05VqZNm0bNmjWtWqCDgwPr16+nX79+9OjRg9y5cxMQEMDkyZPJlet/5X/++efky5ePcePGcePGDWrUqMH69etNWxMDBAUFkZSUxIQJE5gzZw6lS5fm66+/5q233rJqzfLsKdSIiIiI5DwWh5WRI0dSv359KlWqRIcOHfDy8uLSpUtER0dz7NgxNmzY8MTF+Pn5mXbn+rtixYo9dh2Mg4MDkydPNp2d/mF69uxJz549n7hGERERERF5NiwOKzVr1mTt2rUMGTKEiIgI01nrH7Q3bNgwM+oUEREREREbY3FYAWjUqBFbt27l3r17xMXF4eLiQnJy8iNP7igiIiIiImIJi8PK3bt36dOnD7t37+b333/H2dmZDRs2EBgYSO/evZk4cSL29vaZUas8p7SeRERERESehMVhZfjw4URHRzNy5EhTW40aNZg4cSJhYWEUKlSIYcOGWbVIkexKQUxEREQk81gcVhYtWsSECRPo0aOHqS1fvny8//772NvbM2nSJIUVEStSIBIRERFbZfFJIa9evUqpUqXSva98+fKcP3/+qYsSERERERGxeGTFx8eHZcuWmc4a/3crVqygXLlyVilM5GmlNyIhIiIiIs8Pi8PKwIED6dy5M7GxsbRp04ZChQpx5coVvv32W7755hvmzZuXCWWKyONoupiIiIjkNBaHlY4dOxIfH094eDjffPONqb1AgQJMnz6dbt26WbVAERERERGxTU90npWePXvSo0cPjh49yrVr1/Dw8KBChQrY2Vm8BEZERERERCRdTxRWAAwGA97e3tasRURERMSMpriK2DYNhYiIiIiISLaksCIiIiIiItmSwoqIiIiIiGRLCisiIiIiIpItPVFYSUpKYubMmbRt25a6dety5MgRPvvsM3bt2mXt+kRERERExEZZHFauXr1KrVq1+OCDDzh+/Di7du3i1q1brFq1Cj8/P3bs2JEZdYqIiIiIiI2xOKwMGjSIhIQEDh8+zO+//47RaARg6dKl1KpVi+HDh1u9SBERERERsT0Wn2flhx9+YOrUqZQtW5aUlBRTu6OjI4MGDaJ79+5WLVAku0hvr38RERERyTwWj6zcuXOH/Pnzp3tfrly5uHv37lMXJSIiIiIiYvHISq1atZgxYwaBgYFp7lu4cCE1a9a0SmEiOYnOwCwiIiJiOYvDysiRI2nSpAlVq1YlMDAQg8HAokWLGDFiBOvWrWPdunWZUafIc0PTxURERESsw+JpYA0aNGD9+vXkzZuX8ePHYzQamTRpEpcuXWLVqlU0atQoM+oUEREREREbY/HICkDDhg3Ztm0bt2/f5vr167i5ueHi4mLt2kRERERExIY9UVhJTEwkISGBIkWKYG9vz9SpUzl79izt27enYcOG1q5RREQkU2g9mYhI9mZxWNm1axfNmjWjR48ejB07lg8++IBZs2bh4eHBjBkzWL58Oa1atcqMWkUArQkRERERsRUWr1kJCQmhQoUK9OzZk9u3b7NgwQJ69+5NbGws7777LqNGjcqMOkVERERExMZYHFZ++eUXwsLCKFWqFD/99BO3b9+mW7duAHTs2JGDBw9avUgREREREbE9FocVOzs7HBwcAFi9ejUeHh7Url0bgISEBJydna1boYiIiIiI2CSL16zUrFmTL7/8EicnJ6Kjo2nZsiUGg4HLly8zduxYnRRSRERERESswuKw8umnn9KsWTMWL15MwYIFCQ0NBaBSpUqkpqbqpJAiIiIiImIVFoeVatWqcfz4cQ4dOkSlSpXImzcvADNnzqR+/fp4eXlZvUgREREREbE9T3SeFVdXV+rUqWPW1q5dO6sUJCIiIiIiAhkMK6VLl2bFihW8/PLLlCpVCoPB8NC+BoOBEydOWK1AERERERGxTRkKK76+vri5uZmuPyqsiIiIiIiIWEOGwsrcuXNN1z/77DMcHR0zrSARERF5uIiIiDRtI0aMyIJKREQyn8XnWcmfPz+tW7fmyy+/5NKlS5lRk4iIiIiIiOVhZeLEiQD079+fIkWKUKtWLSIjI9mzZ4/VixMREREREdtlcVjp1asX3333HbGxsaxdu5aGDRuyePFiatasSbFixejVq1dm1CkiIiIiIjbmibYuBsidOzf+/v6ULl0aHx8fvvrqK7Zu3cqsWbOYOXOmNWsUkSekue0iIiLyPLM4rBw7doxNmzYRExPDpk2buHDhAq6urjRs2JCJEyfSqFGjzKhTRERERERsjMVhxdvbG4PBQI0aNejXrx++vr7UqFEDOzuLZ5SJDdMv/iIiIiLyOBaHlffee4+NGzfy22+/ER8fz/Hjxzl58iS+vr54eXllRo0iNiO9ECciIiJiqyweDpk1axbHjh3j9OnTDBkyhBs3bjBgwACKFCmCj48Pffr0yYw6RURERETExjzxAvvixYvzzjvv0LlzZ7Zs2cLMmTNZsWIFf/zxB1FRUdasUeSZ0wiHiIiISNZ7orCye/duNmzYwIYNG9i2bRt3796lWrVqDB8+nBYtWli7RhERERERsUEWhxVPT0/i4uJwcXHB39+fqKgoAgMDeeGFFzKjPhEREZHH0oi4SM5kcVgpUaIE8+bNo3nz5uTK9cSzyERERERERB7J4gX2Fy5cIDExUUFFREREREQylcVhxdHREUdHx8yoRURERERExMTi4ZFhw4bRs2dP9u3bR6VKldJdq9KwYUOrFCciz4ZO0ikiIiLZkcVhJTg4GICRI0cCYDAYTPcZjUYMBgMpKSlWKk9ERERERGyVxWFl48aNmVHHY926dQtXV1dSU1PN2h0cHLhz5w4Au3btYvDgwezevRsXFxe6du3KqFGjcHBwMPW/ePEiAwYMYP369dy9e5eAgACmTp1KkSJFnunzERERERGRR7M4rPj6+mZGHY+1f/9+UlNTWbRoESVLljS129ndX3Zz4sQJ/P39qVevHkuWLOHw4cOEhIQQHx/PF198AUBycjLNmzfnxo0bzJw5k3v37jFkyBACAgLYu3cvuXPnzoqnJv+ftp0UEZHsTtNmRZ6tDIWVyMhI3nvvPQoXLkxkZOQj+xoMBsLCwqxS3N/t3buXPHny0K5du3RDxfjx43F1deW7774jT548BAYG4uzsTN++fQkNDaVEiRIsXbqUffv2cfDgQV566SUAqlatSqVKlYiOjqZr165Wr1vkeaX/IYuIiEhWy1BYCQ8Pp1mzZhQuXJjw8PBH9s3MsOLj4/PQ0Y9169bRsmVL8uTJY2pr3749vXv3Zt26dfTo0YN169bh7e1tCioAPj4+VKxYkdWrVyusiIiIiIhkIxnaujg1NZXatWubrj/qklmL6/fu3YudnR3+/v7kzZuX/Pnz07NnTxITE7l9+zZnzpyhfPnyZo8pWLAgbm5uHD16FIDDhw+n6QNQtmxZUx8REREREckeLD7PyuPEx8db+5CkpqZy4MABjh07Rtu2bVmzZg0hISEsWrSIwMBAYmNjAXBzc0vzWFdXVxISEgCIi4t7bJ9/SkpKIiEhwewiIiIiIiKZz+IF9klJSUyePJlNmzZx9+5djEYjcD9Q3Lx5k//+97/cunXLqkUajUZWrVqFl5cXFSpUAO6fy8XLy4uuXbsSExMDmG+j/PfHPliEn5qa+tg+/zRmzBgt/BYRERERyQIWh5WPPvqIadOmUblyZS5fvoyTkxMFCxbkwIED3L1797FrWp6Evb09fn5+adpbtGgBwOnTpwHSHfW4ceMG7u7uAHh4eDy2zz8NHTqUAQMGmG4nJCRQrFgxS5+CSI6lhfgiIiKSWSyeBvbNN9/Qv39/9u3bxwcffEDNmjX55ZdfOHbsGCVLlkxzHhRrOH/+PF988QXnzp0za799+zYAL7zwAkWKFOH48eNm91+5coWEhAR8fHwA8Pb2TtMH4Pjx46Y+/+Tg4ICbm5vZRUREREREMp/FYeXy5cumEY0qVaqwa9cuAIoUKcLQoUNZvHixdSvk/tSzHj16MGvWLLP26Oho7OzsaNCgAQEBAaxcuZKkpCTT/cuWLcPe3p7XXnsNgICAAA4fPsyhQ4dMfQ4dOsThw4cJCAiwet0ijxIREZHmIiIiIiL/Y/E0MA8PD1MgKF++PGfPniUxMRFXV1fKlSvHn3/+afUiS5cuTbdu3Rg3bhwODg688sorbN26ldGjR9O7d2+8vb356KOPWLRoEc2bN2fAgAEcPXqUYcOG0bNnT9O0rQ4dOjB69GiaN2/O2LFjARgyZAiVK1fmzTfftHrdIiIiIiLy5CweWWnQoAH/+c9/uHnzJqVKlSJv3rwsX74cgB07djx07cfTmjVrFqGhoXz11Ve0aNGCr7/+moiICKZMmQJAhQoV+PHHH7l16xbt27dn0qRJ9O/fn6lTp5qO4eDgwPr166lRowY9evSgT58+1K1bl7Vr15Irl8W5TUREREREMpHF39DDw8Np0KABLVu2ZOPGjfTu3ZuePXvyn//8h/3799OrV6/MqBNHR0fCwsIeecLJBg0asHPnzkcep1ixYqZwJSIiIiIi2ZfFYaVy5cocOXKEAwcOAPe39nVzc2Pbtm20atWKoUOHWr1IERERERGxPU8098nLywsvLy/g/rlNhg0bZtWiREREREREnugM9nv27KFDhw4ULlwYJycnSpQowbvvvsupU6esXZ+IiIiIiNgoi0dWfv75Z5o3b46npyeBgYEUKlSIv/76i9WrV7Ns2TK2bdtGpUqVMqNWERERERGxIRaHlWHDhtGgQQNWrlyJo6Ojqf327ds0a9aMwYMHs2bNGqsWKSIiIiIitsfisHLgwAGWLVtmFlQAnJycGDx4MJ06dbJacSIiIiIiYrssXrNSvHhxTp48me59V65coWDBgk9dlIiIiIiIiMUjKxMmTKB79+54eHjQsWNH7O3tAfjxxx8JDQ1l5syZVi9SRERERERsj8VhpW/fvty5c4e3336bd955h4IFCxIbG8vdu3cxGo288cYbpr4Gg4Hk5GSrFiwiIiIiIrbB4rASFBSUCWWIiIiIiIiYszisjBgxIjPqEBERERERMfNEJ4UUERERERHJbAorIiIiIiKSLSmsiIiIiIhItpShNSsnTpygVKlS2Nkp24iIiIhlIiIi0rRpDayIZESG0kft2rXZtm0bAP/61784depUphYlIiIiIiKSobBy584dDh06BMC8efO4evVqphYlIiIiIiKSobDSpEkTevXqZTpb/SuvvIK9vX26l1y5LN4NWUREREREJI0MJYsFCxbw9ddfc/XqVSIiIvjXv/5F0aJFM7s2EREREavS+hmR50uGwoqrqyt9+vQBICYmhoEDB1KhQoVMLUxEchZ9QRARERFLWTxna+PGjQAcOXKEmJgY4uLiKFCgAK+++qoCjIiIiIiIWM0TLTAJDg7miy++wGg0mtoMBgNBQUF8+eWXGAwGqxUoIiLypEaPHk1qnjyARvJERJ5HFp84Zfz48cyePZvIyEhOnTrF7du3OXnyJOHh4cyfP5/JkydnRp0iIiIiImJjLB5Z+fLLL/n4448JCQkxtZUsWZKwsDDu3r3Ll19+yYABA6xapIiIiIiI2B6Lw8rZs2dp1KhRuvf5+fkxYcKEpy5KROR5os0DREREMofF08BKlizJvn370r1vz549FCxY8KmLEhERERERsXhkpXPnzoSHh1O4cGHeeust7OzsSE1NJTo6msjISIKDgzOjTsmG9GuyiIiIiGQmi0dWPvroI+rUqUPnzp1xdHSkcOHCODo60rVrV2rVqkVkZGRm1CkiIiIiIjbG4pEVBwcH1q9fz5o1a9i0aROxsbHkz58fX19fmjdvnhk1ioiIiIiIDXqi86wANG/eXOFEREQeSdNFRUTkaVg8DUxERERERORZUFgREREREZFs6YmngYmIiEj2pSl4IpITKKyIiEiW0xdryUrpff5EJHvIUFj573//y0svvQSAvb09O3bsoHbt2qSkpJAnTx5+/fVXqlevnqmFimQW/U9KHkZfoEX+R/8eRCQrZCisVK1aFTc3N+rVq4fRaOT333+nYsWKODs7YzQaM7tGeY7oi7+IiIiIWEuGFthfv36dRYsWmUZPBg4cSL58+ahcuTIGg4ElS5YQExPD9evXM7VYERERERGxHRkKKy4uLgQEBJh+Nf/55585ePAggwYNwmg08t1339GyZUs8PT0pUaJEphYsIiIiIiK2IUPTwGbPnk2DBg0oX748AHZ2dlSoUIFy5crx3nvvsXDhQqpVq8bRo0c5cOBAphYsIjmbtefFa569yPNDU4lF5J8yFFamTp1KcHAw+fLlw2AwMG/ePK5fv061atUAMBgMGAwGvL298fb2ztSCRURERETENmQorOzfv5/4+Hi2bdtGy5Yt2bVrF9HR0cTGxmIwGAgNDaVBgwZUrVqVl19+mRdffDGz65ZnTL92iYiIiMizluEz2Lu7uxMYGAhAVFQUV69e5dChQxiNRlxdXVm/fj3dunWjaNGimVasiIiIiIjYDotPClmiRAkcHBwAKF++PCVKlCAsLMx0Hpbz589bt0IRERGxCq3hEpHnjcVh5dSpU6brdnZ2ZrcBihQp8vRViYiIpON5/LL9NNNoNQVXRGydxWFFRESeP8/jl3zJ/vS5EpHMprAiIiIW05dUERF5FhRWRESeIX3JFxERyTiFFRERsQnWXv+h9SQiIplPYUVERMSGZdVon8KeiGSEwoqIZJmMflnR1KmsZWtfKm3t+Vrbs3j99B6J2A6FFRF5LinAiPyPpriJSE6lsPKc0Rc0kZwnO/271pdUERHJThRWJA19WZGHycmfjefhuWWnUJOd/PN1sbt7l8ZZVIuIiFiXTYaVtWvXEhoayqFDhyhYsCDBwcEMGTIEg8GQ1aVlKn3REREREZHnic2Fle3bt9OqVSs6dOjAJ598wtatWwkJCSE1NZWQkJCsLs+MFinKw2T39y2715eT6LUWEZGczObCSkREBFWrVmX+/PkANGvWjHv37jF27FgGDBiAk5NTFlcoIjmBQoSIiMjTs6mwkpSURExMTJovEe3bt2f8+PFs2bKFgICALKrOuvRFSURERESedzYVVk6ePMndu3cpX768WXvZsmUBOHr06HMZVhRMRJ6dZ/XvTdNAs399IiKS+WwqrMTFxQHg5uZm1u7q6gpAQkJCmsckJSWRlJRkuh0fH//QvtZ2586dTP8bYnvS++zmlM9aRp9bTn4NslJ2ef3t7t3jptF4/7hJSaSmplrt2JlJn8vn1+Peu79/JhMSErBPScnUelxdXXP8pkFiOwxG4///12MDtm3bxquvvsqGDRto3Ph/G1smJyeTO3duxowZw5AhQ8weEx4erl/3RERE5LkRHx+f5odZkeeVTY2seHh4AGl/AUlMTATA3d09zWOGDh3KgAEDTLdTU1OJjY3F09PTpn+1SEhIoFixYpw9e1b/QcwB9H7mHHovcw69lznLs3w/H8wYEckJbCqslClTBnt7e44fP27W/uC2j49Pmsc4ODjg4OBg1vYg9Mj9KXX6n2jOofcz59B7mXPovcxZ9H6KWMYuqwt4lhwdHWnYsCHLly/n77Pfli1bhoeHB7Vr187C6kRERERE5O9samQFIDQ0lCZNmvDWW2/xr3/9i+3bt/Ppp58ybtw4nWNFRERERCQbsQ8PDw/P6iKepVKlSlGlShWWLVtGVFQUp0+fJiQkhEGDBmV1ac8de3t7/Pz8yJXL5jJvjqT3M+fQe5lz6L3MWfR+iljOpnYDExERERGR54dNrVkREREREZHnh8KKiIiIiIhkSworIiIiIiKSLSmsyCMZjUZmzZpFlSpVcHFxoXTp0nz44YdmJ9b8448/aNGiBe7u7nh6evLuu+8SFxeXhVVLRrRt25aSJUuatem9fH7s3LmTRo0akTdvXl544QW6d+/O5cuXTffrvXy+fPHFF7z00kvkzZuXihUrEhUVZbbFvt7P7O3s2bN4eHgQExNj1n7x4kU6depEgQIFcHNzo3379pw/f96sT3JyMsOGDaNYsWI4OTlRt25dtm3b9gyrF8nmjCKPMG7cOKO9vb1xyJAhxvXr1xtnzpxpLFCggLFx48bG1NRU4/Xr141FihQx1qpVy/jdd98ZZ82aZfTw8DD6+/+/9u49LIrr/AP4d9Fl2XWBBRHQIqBCNkLUVJSHS4CgSFHBGxWiYjDEx1tMGq3iLYjGSry0moqkKhAgRlBjMYqSLAFXRINWUxtUjEJUYhTFoqCQcn9/f1CmDAuoSdTl5/t5nn2e7JkzZ87MC5F35pwzo55111kndu7cSQDIzs5OKONYdh1nzpwhIyMjGjt2LGk0GkpKSiJra2tyd3cnIo5lVxMfH08A6O2336bs7GyKiooiiURCGzduJCKOp767du0aqdVqAkBarVYor6+vpyFDhtCAAQNo7969tGvXLurbty85OTlRXV2dUG/evHmkUCgoNjaWDh48SK+++ir16NGDLl269AzOhjH9w8kK61BjYyOpVCqaN2+eqHzv3r0EgE6fPk0xMTGkUCiorKxM2J6ZmUkAKC8v72l3mT2CTMiJhQAAFb9JREFUGzdukJmZGdnY2IiSFY5l1+Hr60tubm7U0NAglP39738nGxsbunLlCseyi3F3dydPT09RWWhoKNnb2xMR/27qq8bGRvr444/J3NyczM3NdZKV1NRUAkDnz58Xyi5cuEASiYR27txJREQ//PADde/eneLi4oQ6NTU1ZGtrS2+++eZTOxfG9BkPA2Mdun//PsLCwjB16lRR+QsvvAAA+P7776HRaODl5YVevXoJ23/3u9/B2NgYmZmZT7W/7NHMnDkT/v7+GDlypKicY9k1lJeX4+jRo5g3bx66desmlE+aNAnXr19Hv379OJZdTG1tLUxNTUVlFhYWKC8vB8C/m/qqoKAAc+fORXh4OHbu3KmzXaPRQK1Ww9nZWShzcnLCwIEDhbjl5OSgoaEBkyZNEurIZDIEBgZybBn7L05WWIdUKhViY2Ph6ekpKk9PTwcAvPTSS7h48aKQvLQwMDBAv379cPny5afWV/ZoEhIS8M0332Dr1q062ziWXUNBQQGICJaWlpg2bRqMjY2hVCoRFhaGe/fuAeBYdjULFixAVlYWPv30U1RWVkKj0SAlJQXTp08HwPHUV7a2tiguLsamTZugUCh0trcXNwBwcHAQ4nbx4kUolUpYW1vr1CktLUVVVdWT6TxjXQi/QpU9lq+//hrr16/HhAkT4OzsjIqKCpiYmOjUMzY2Fk3CZ89eSUkJFi5ciKSkJFhYWOhs51h2DXfu3AEAREREYPTo0fj8889RVFSEZcuW4fvvv8eJEyc4ll3M5MmTceTIESE5AZqfnHz44YcA+HdTX5mbm8Pc3LzD7RUVFXB0dNQpbx23iooKnadqLXWA5hEOSqXyV+oxY10TJyvskeXl5SEoKAgDBgxAYmIigObVwiQSiU5dIoKBAT+40xdEhIiICIwZMwbBwcEd1uFY6r+6ujoAgIuLCxISEgAAI0eOhEqlwpQpU/DVV19xLLuY8ePH48SJE9iwYQNcXV1RUFCAVatWYfLkydi/fz/Hs4tqamp6aNw6qwOA48sYOFlhj2j37t2YMWMG1Go1NBqNcDfJ1NS03Tt7VVVVsLGxedrdZB2Ii4tDQUEBzp07h4aGBgD/+8ewoaEBBgYGHMsuouWOa2BgoKg8ICAAAPCvf/2LY9mFfP3119BoNIiPj8fMmTMBAD4+Pujfvz8CAwNx+PBhjmcXpVKpOoxby9OUzuoAaPepC2PPG07Z2UNt3LgRU6dOhZubG44dOyYaW6tWq1FcXCyq39TUhKtXr8LJyelpd5V1YN++ffj3v/+N3r17QyqVQiqV4pNPPkFJSQmkUinef/99jmUX0TKspLa2VlReX18PAJDL5RzLLqSkpAQAdOYG+vj4AAAuXLjA8eyi2osbABQXFwtxU6vVuH//vjC8s3Ude3t7yOXyp9JXxvQZJyusU9u3b0dkZCQmT56MrKwsnbs8/v7+yM3NFf2PVqPR4MGDB/D393/a3WUd2L59O06fPi36BAYGonfv3jh9+jRmzZrFsewiBg4cCHt7e+zevVtUfvDgQQCAl5cXx7ILefHFFwE0D7NtreWlgP369eN4dlH+/v64ePEiCgsLhbLCwkJcvHhRiNuoUaMANN9QalFbW4tDhw5xbBlr8UwWTGZdQmlpKcnlcrKzs6O8vDzKz88XfcrKyujOnTtkYWFBQ4YMofT0dIqPjyczMzMaPXr0s+4+e4jw8HDRe1Y4ll3HZ599RhKJhEJCQigrK4u2bNlCSqWSgoODiYhj2dUEBwdTjx49aN26daTVamnr1q1kYWFBQ4cOpbq6Oo5nF6DVanXes1JTU0NqtZpsbW0pNTWVUlNTydbWlgYPHkz19fVCvfDwcJLJZPSXv/yFMjIyyNfXl0xMTKioqOgZnAlj+oeTFdahxMREAtDhJykpiYiIzp07RyNHjiS5XE6WlpY0a9Ysun///rPtPHuotskKEceyK8nIyKDhw4eTTCaj3r1706JFi6impkbYzrHsOmpraykqKors7e3J0NCQHBwcaPHixfTgwQOhDsdTv7WXrBA1v/Rx4sSJpFQqyczMjEJDQ+nmzZuiOjU1NfTuu++SpaUlKRQK8vLyolOnTj3F3jOm3yRE/51lyxhjjDHGGGN6hOesMMYYY4wxxvQSJyuMMcYYY4wxvcTJCmOMMcYYY0wvcbLCGGOMMcYY00ucrDDGGGOMMcb0EicrjDHGGGOMMb3EyQpjjDG98rAV9XnFfcYYe35wssLYc+jmzZsYNWoUjI2N4e3tjaKiItH2b7/9FpaWlrh///4z6uHjkUgkWLVq1RM/zrVr1yCRSJCcnPyL26qursaqVavg5OQEuVwOU1NTeHh4ID4+Hk1NTUK9o0ePQiKRYMKECe22k5ycDIlEgmvXronqt/4YGBjAxMQEnp6eyMjI6LRfM2bMgL29fad12h7z15SYmIhFixZ1eKzCwkJ4enqK9nla8WeMMfb0cbLC2HPoD3/4Ax48eID09HSoVCq8/vrrou2RkZFYtmwZTExMnlEP9VPv3r2Rn5+PsWPH/qJ2iAhBQUHYtm0bZs2ahcOHDyM1NRXDhg3D7NmzsXDhQp19Dhw4gF27dj3yMeLi4pCfn4/8/HycOHECaWlpkMlkGD9+PL744osO94uKisL+/ft/1nn9GtasWYPy8nLh+9ixY5Gfn4/evXsDAPbu3Yv8/HzRPvn5+Zg5c+ZT7SdjjLGno/uz7gBj7OnLzs5GQkICRo0aBQsLCwwdOhRVVVVQKpXIzs7GpUuXcPDgwWfdTb0jk8ng5ub2i9s5fvw4tFotNBoN/P39hfKxY8eiW7duiI2NxdKlS2FtbS1sU6lUeOedd+Dn5wcrK6uHHsPJyUmnr97e3ujbty/++te/YvTo0e3uN2DAgJ95Vk9Gr1690KtXr07r/BoxYYwxpp/4yQpjzyGJRAK5XA4AMDQ0BAA0NjaCiBAZGYn3338fMpnskds7c+YMAgIC0LNnT5iYmCAoKAgXLlwAANTU1EClUmHBggWifZqamtCnTx+89dZbQllCQgKcnZ0hk8lga2uLVatWoaGhQdg+Y8YMjBw5EnPnzoVKpcLQoUNF21sUFBRg0qRJ6NWrF6RSKX7zm9/gnXfewX/+8x/RNdi6dSvCw8OhVCphZWWlU6ettsPAkpOT0b17d5w6dQru7u4wMjKCra0tNmzY0On1unXrFoD2517MmzcPMTExkEgkovKYmBhUV1djzpw5nbbdGWNjY6jVapSUlHRYp+0wsKamJvzpT3+Cra0tFAoFJkyYgLt37+rsd/78eQQGBsLExAQmJiaYOHEirly5ImxvGZ6Wk5MDf39/KBQKWFlZYfHixUIM7e3tUVJSgpSUFGHoV+thYKtWrcLq1asBiId+tR0GdvfuXcyePRtWVlYwMjKCm5sbcnJyRP3Nzs6Gu7s7lEolzMzMMGHCBFy6dOlxLyljjLEnjJMVxp5D7u7u+PTTT1FRUYHk5GQMGjQIpqamSE1NRX19PcLCwh65La1WCw8PDzQ2NuLjjz9GQkICrl+/Dg8PD3z33XcwMjLC73//e+zdu1c0F0Or1aK0tBTTp08HAHzwwQeYNWsW/Pz8kJGRgfnz52P9+vWYPXu26HjHjh1DUVER0tPTsXLlSnTvLn5AXFpaCi8vL1RXVyM5ORlffPEFQkJCEBsbi82bN4vqRkVFoaysDHv37kVkZCTi4+Mf69yB5j/mQ0JC8NprryEzMxNeXl5YsmQJNBpNh/v4+PhAqVTitddew5IlS3D06FEhSXJ0dERkZKTO05OBAwdi9erV+Pzzz5GWlvZYfWxRV1eHq1evPtbTk8jISKxevRoRERHYv38/LCwssHTpUlGdy5cvw8PDA2VlZUhOTkZiYiKuXLkCT09PlJWViepOmzYNXl5eOHToEKZNm4Y///nPSExMBADs378f1tbWGDNmjGjoV4uZM2fizTffBNDx0K+amhqMGDECBw4cwNq1a5Geng4bGxsEBATgyJEjAIArV65g3LhxcHFxQUZGBhISEvDdd99hzJgxop9RxhhjeoAYY8+doqIiGjx4MAEgR0dH+uc//0k1NTVkb29PGRkZVFxcTH5+fuTs7EybNm3qtC1XV1d68cUXqaGhQSi7d+8e9ezZk0JCQoiI6OjRowSAjh49KtSJiIggBwcHIiKqqKgghUJBc+bMEbWdkJBAAOj8+fNERBQeHk4AqKioSFQPAEVHRxMRkUajIW9vb6qsrBTVGTRoEPn7+4v2cXR0pPr6eqFs8+bNouO1dfXqVQJASUlJRESUlJREACghIUGoU1NTQ0ZGRjR//vyOLxoRHTt2jPr3708ACABJpVLy9vambdu2ifqk1WoJAGm1WmpoaCBXV1fq2bMn3bp1S9SHq1eviupnZ2dTfX091dfX008//USFhYU0ZcoUAkCZmZkd9is8PJzs7OyIqDmOUqmUFi1aJKoTEBAgOubUqVPJ0tJSdM3Ly8vJ1NRU2LelX++9956orX79+lFgYKDw3c7OjsLDw4Xvbc8vOjqa2v7T1Tr+O3bsIAB08uRJYXtTUxN5e3vTsGHDiIgoLS2NANCPP/4o1Dl16hQtX75c5+eGMcbYs8VPVhh7Djk4OODbb79FVVUVLl++jN/+9reIi4tD3759ERgYiMmTJ8PFxQUpKSlYv349Dh8+3G471dXVOH36NEJDQ9GtWzehXKVSITAwEFqtFkDzXAk7Ozvs3r0bQPMd/vT0dOGpSn5+Pn766SeMGzcODQ0NwicoKAgA8NVXXwlty+XyTp8M+Pv7Izc3F3K5HJcvX8ahQ4cQExODsrIy1NXViepOmTJF9GQmODgYAJCXl/fI1xJoflLVQiaToVevXqiuru50Hy8vLxQVFSE3NxcrVqyAq6sr8vPzMWfOHLz66qvtDkfr1q0bkpKSUFVVhblz53bavp+fH6RSKaRSKRQKBZycnJCTk4MtW7Z0OF+lrZMnT6K+vh7jx48XlYeEhIi+5+TkwNfXFwqFQoidiYkJvLy8RLEDxNcKAGxsbB56rR5HTk4OrK2t4eLiIvSlsbERQUFBOHPmDO7duwc3NzcYGRnB1dUVCxcuRHZ2Nl5++WWsXbuWF5VgjDE9wxPsGXuO9ejRAwBQWVmJmJgYHDx4EFeuXMHZs2fx5ZdfwtLSEpMmTcK+ffvaXQGroqICRCSaCN7C2toaFRUVAJrnFISFhWHHjh2IjY1FZmYmKioqhCFXLas/jRkzpt1+3rx5U/hvS0tLnfkcrTU1NWH58uWIi4tDVVUV+vbtC1dXV8jlcp05In369BF9t7S0BADcu3evw/bbo1AoRN8NDAweaTiRgYEBvL294e3tLRz3vffew0cffYTExETMnz9fZx8nJydER0dj+fLlQvLXnm3btsHFxQVAc5Jjbm4OW1vbTq9dWy1zU9pOcG87PKu8vBx79uzBnj17dNpou+/PvVaPqry8HLdu3YJUKm13e2lpKZycnJCbm4t169Zhx44d2Lx5M1QqFebNm4c1a9bAwIDv4zHGmL7gZIUxhpiYGLzyyivw8PDAyZMnAQDm5uYAADMzM1y9erXd/VQqFSQSiTBhvLXS0lJYWFgI36dPn461a9dCq9UiLS0Nnp6e6N+/v9AOAOzatQsvvPCCTluPsvpVi3Xr1mHTpk3Ytm0bgoODYWpqCgBwdXXVqdt6iVwAuH37NoD/JS1PSmhoKMrLy5GdnS0qNzMzQ2xsLNLS0lBYWNjh/pGRkUhPT8fbb7+NxYsXt1tHrVZj2LBhv6ifLfG7ffs21Gq1UN72uqlUKvj5+eGPf/yjThtt5xQ9aSqVCo6OjkhNTW13e79+/QA0/zykp6ejrq4Ox48fx/bt2xETE4PBgwcjNDT0aXaZMcZYJ/j2EWPPuR9//BFxcXGIiYkB8L8/1FsSkNLS0g7/eO/RoweGDRuGPXv2oLGxUSivrKzEoUOH8MorrwhlarUaw4cPx+7du3Ho0CFhCBjQvPSsoaEhbty4gWHDhgkfQ0NDLF26tMNkqT3Hjx+Hs7MzIiIihETlxo0bOHfunM4d/LbLM+/btw8SiQS+vr6PfLyfw8HBAUeOHBESw9Zu3ryJqqoqDBo0qMP9u3XrhuTkZNy/f1+I25Pg4eEBuVyOzz77TFTe9sWSPj4+KCwsxMsvvyzEzsXFBZs3b37sd7a0Hk74c7b7+Pjg+vXrsLS0FP0sZWdnY8OGDejevTs+/PBD2Nvbo7a2FoaGhhgxYgR27NgBALh+/fpj9ZcxxtiTxckKY8+5qKgohIaGwsnJCUDznWdnZ2csW7YM6enpSE9P15mz0NoHH3yAoqIiBAQE4ODBg9i3bx9GjBiB2tpaREdHi+q+/vrrSElJQUNDg2jeQ8+ePREZGYmoqChERUUhJycHn3zyCYKCglBcXIwhQ4Y88vm4urqioKAA69atQ25uLhITE+Ht7Y3a2lqduRH/+Mc/EBYWBo1Gg40bN2LFihWYNWuW8MTnSVm0aBEGDhwIPz8/LFmyBF9++SWOHTuGuLg4eHt746WXXsKMGTM6bcPZ2RkrV65EZWXlE+unUqlEVFQUPvroIyxduhRZWVl49913dZKVlStXori4GIGBgThw4AA0Gg2Cg4ORlpb2WLEDmp+MnD17Frm5ue3O22l5CpeWltZuEvvGG2/Azs4Oo0aNQkpKCrRaLZYvX44VK1agT58+kEqlGDFiBEpLSzFx4kRkZmYiKysLb7zxBmQymTBPijHGmJ541jP8GWPPzrlz56hHjx70ww8/iMrPnj1LgwcPJnNzc1q2bBk1NTV12o5WqyUvLy+Sy+WkUqlo3Lhx7a6odefOHZJKpTRx4sR224mLiyMnJycyNDQkKysrmjZtGpWUlAjbW69U1RparQZVU1NDb731FllbW5NcLie1Wk3R0dG0evVqkslkdPfuXWGfBQsWUEhICMnlcurbty+tWbNGtKpZWx2tBtayUlWLtitataeyspJWrFhBgwYNImNjY5LJZKRWq2nFihX04MEDoV7r1cDaqq+vJxcXl3ZXA2uv/qNo7xpv2bKF+vfvTzKZjHx9felvf/ubznl/8803FBAQQMbGxqRUKsnNzY0OHDjw0PPw8fEhHx8f4XtqaipZWlqSTCajvLw8nWt848YNGj58OEmlUpo7dy4RieNPRHT79m2KiIgQ2lGr1bRhwwZqbGwU6mg0GvL09CQTExNSKBTk7e1Nubm5P+uaMcYYe3IkRO28lYwxxv6fk0gkiI6OFr1MkDHGGGP6hYeBMcYYY4wxxvQSJyuMMcYYY4wxvcRLFzPGnks8ApYxxhjTf/xkhTHGGGOMMaaXOFlhjDHGGGOM6SVOVhhjjDHGGGN6iZMVxhhjjDHGmF7iZIUxxhhjjDGmlzhZYYwxxhhjjOklTlYYY4wxxhhjeomTFcYYY4wxxphe4mSFMcYYY4wxppf+D02TpOWRoHE/AAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 1600x1000 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "percent_overlap_10000 = diff(list_of_snps, 33930)\n",
    "\n",
    "# generate a figure with sub-plots (one for each gene segment)\n",
    "fig = plt.figure(figsize=(16,10), facecolor='w')\n",
    "plt.style.use('default')\n",
    "mpl.rc('font', family='Arial')\n",
    "gs = gridspec.GridSpec(3, 20)\n",
    "\n",
    "ax1 = fig.add_subplot(gs[0,0:9]) \n",
    "\n",
    "# get right of top and right frame of each subplot\n",
    "ax1.spines['top'].set_visible(False)\n",
    "ax1.spines['right'].set_visible(False)\n",
    "ax1.spines['bottom'].set_visible(True)\n",
    "ax1.spines['left'].set_visible(True)  \n",
    "\n",
    "# plot distribution\n",
    "plt.hist(percent_overlap_10000, 100, color='grey')\n",
    "\n",
    "# plot household pairs\n",
    "plt.axvline(x=roommates1, color='#c5050c')\n",
    "plt.axvline(x=roommates29, color='#c5050c')\n",
    "\n",
    "textstr1 = 'n=32'\n",
    "textstr2 = 'n=1'\n",
    "\n",
    "\n",
    "ax1.set_ylabel('# of pairwise comparisons',fontsize=12)\n",
    "ax1.set_xlabel('% overlap in SNP identities',fontsize=12)\n",
    "\n",
    "ax1.tick_params(axis='both', which='major', labelsize=12)\n",
    "ax1.tick_params(axis='both', which='minor', labelsize=12)\n",
    "\n",
    "#legend\n",
    "A = mpatches.Patch(color='#c5050c', label='Roommates')\n",
    "B = mpatches.Patch(color='grey', label='Random pairs')\n",
    "\n",
    "ax1.text(0.92, 1.09, textstr1, transform=ax1.transAxes, fontsize=14,\n",
    "        verticalalignment='top')\n",
    "ax1.text(0.625, 1.09, textstr2, transform=ax1.transAxes, fontsize=14,\n",
    "        verticalalignment='top')\n",
    "\n",
    "\n",
    "plt.legend(handles=(A, B), frameon=False, bbox_to_anchor=[1, 1.1], fontsize=12)\n",
    "\n",
    "\n",
    "plt.savefig(\"figures/within-vs-between-roomies.svg\", bbox_inches='tight', dpi=300)\n",
    "plt.savefig(\"figures/roommates_vs_pairs.png\", bbox_inches='tight', dpi=300)\n",
    "\n",
    "\n",
    "# plt.savefig(\"figures/within-vs-between-household.png\", bbox_inches='tight', dpi=300)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SNP</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>241_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>1059_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>3037_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>3773_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>6793_G&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>176</td>\n",
       "      <td>21638_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>177</td>\n",
       "      <td>21639_C&gt;A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>178</td>\n",
       "      <td>22205_G&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>179</td>\n",
       "      <td>22687_C&gt;T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>180</td>\n",
       "      <td>26748_A&gt;G</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>181 rows  1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           SNP\n",
       "0      241_C>T\n",
       "1     1059_C>T\n",
       "2     3037_C>T\n",
       "3     3773_C>T\n",
       "4     6793_G>T\n",
       "..         ...\n",
       "176  21638_C>T\n",
       "177  21639_C>A\n",
       "178  22205_G>T\n",
       "179  22687_C>T\n",
       "180  26748_A>G\n",
       "\n",
       "[181 rows x 1 columns]"
      ]
     },
     "execution_count": 245,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "unique_snps = get_unique_snps('VCFs/Cleaned/', cleaned)\n",
    "unique_snps"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "roommates = [roommates1,\n",
    "roommates2,\n",
    "roommates3,\n",
    "roommates4,\n",
    "roommates5,\n",
    "roommates6,\n",
    "roommates7,\n",
    "roommates8,\n",
    "roommates9,\n",
    "roommates10,\n",
    "roommates11,\n",
    "roommates12,\n",
    "roommates13,\n",
    "roommates14,\n",
    "roommates15,\n",
    "roommates16,\n",
    "roommates17,\n",
    "roommates18,\n",
    "roommates19,\n",
    "roommates20,\n",
    "roommates21,\n",
    "roommates22,\n",
    "roommates23,\n",
    "roommates24,\n",
    "roommates25,\n",
    "roommates26,\n",
    "roommates27,\n",
    "roommates28,\n",
    "roommates29,\n",
    "roommates30,\n",
    "roommates31,\n",
    "roommates32,\n",
    "roommates33]\n",
    "\n",
    "randompairs = percent_overlap_10000"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "MannwhitneyuResult(statistic=32128.0, pvalue=3.3978656376908585e-21)"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stats.mannwhitneyu(randompairs, roommates)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
